<!DOCTYPE html> <html lang = "en">
 <body>

<h2>Pubmed_ID: 33353957</h2>

<h3>Author Correction: Genome-wide association study of intracranial aneurysms identifies 17 risk loci and genetic overlap with clinical risk factors</h3>

<h3>作者更正：颅内动脉瘤的全基因组关联研究确定了17个风险位点和遗传重叠与临床危险因素 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41588-020-00760-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33353957/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature genetics  {'ISSN': '1061-4036', 'Name': 'nature genetics', '2019': '27.605', '2018': '25.455', '2017': '27.125', '2016': '27.959', '2015': '31.616'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/22/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 没有摘要</p>

<h2>Pubmed_ID: 33347571</h2>

<h3>Comparison of long-read methods for sequencing and assembly of a plant genome</h3>

<h3>植物基因组测序和组装的长效方法比较 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa146</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33347571/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2019': '5.993', '2018': '4.688', '2017': '7.267', '2016': '6.871', '2015': '7.463'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/21/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Background: Sequencing technologies have advanced to the point where it is possible to generate high-accuracy, haplotype-resolved, chromosome-scale assemblies. Several long-read sequencing technologies are available, and a growing number of algorithms have been developed to assemble the reads generated by those technologies. When starting a new genome project, it is therefore challenging to select the most cost-effective sequencing technology, as well as the most appropriate software for assembly and polishing. It is thus important to benchmark different approaches applied to the same sample. Results: Here, we report a comparison of 3 long-read sequencing technologies applied to the de novo assembly of a plant genome, Macadamia jansenii. We have generated sequencing data using Pacific Biosciences (Sequel I), Oxford Nanopore Technologies (PromethION), and BGI (single-tube Long Fragment Read) technologies for the same sample. Several assemblers were benchmarked in the assembly of Pacific Biosciences and Nanopore reads. Results obtained from combining long-read technologies or short-read and long-read technologies are also presented. The assemblies were compared for contiguity, base accuracy, and completeness, as well as sequencing costs and DNA material requirements. Conclusions: The 3 long-read technologies produced highly contiguous and complete genome assemblies of M. jansenii. At the time of sequencing, the cost associated with each method was significantly different, but continuous improvements in technologies have resulted in greater accuracy, increased throughput, and reduced costs. We propose updating this comparison regularly with reports on significant iterations of the sequencing technologies.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 背景：测序技术已经发展到可以生成高精度，单倍型解析的染色体级装配的水平。可以使用几种长读测序技术，并且已经开发了越来越多的算法来组合由这些技术生成的读数。因此，在开始一个新的基因组计划时，选择最具成本效益的测序技术以及最适合组装和抛光的软件是一项挑战。因此，重要的是对应用于同一样本的不同方法进行基准测试。结果：在这里，我们报告了三种应用于植物基因组从头澳洲坚果（Macadamiajansenii）的从头装配的长测序技术的比较。我们已经使用PacificBiosciences（SequelI），OxfordNanoporeTechnologies（PromethION）和BGI（单管长片段读取）技术为同一样品生成了测序数据。几个汇编程序在PacificBiosciences和Nanopore阅读程序的汇编中进行了基准测试。还介绍了将长读技术或短读和长读技术相结合所获得的结果。比较了组装件的连续性，基本准确性和完整性，以及测序成本和DNA材料要求。结论：3种长期阅读的技术产生了高度连续且完整的詹氏甲烷球菌基因组组装体。在测序时，每种方法的成本差异很大，但是技术的不断改进已提高了准确性，增加了通量并降低了成本。我们建议定期更新此比较，并提供有关测序技术重大迭代的报告。</p>

<h2>Pubmed_ID: 33347570</h2>

<h3>Parliament2: Accurate structural variant calling at scale</h3>

<h3>国会2：精确的结构变体大规模调用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa145</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33347570/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2019': '5.993', '2018': '4.688', '2017': '7.267', '2016': '6.871', '2015': '7.463'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/21/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Background: Structural variants (SVs) are critical contributors to genetic diversity and genomic disease. To predict the phenotypic impact of SVs, there is a need for better estimates of both the occurrence and frequency of SVs, preferably from large, ethnically diverse cohorts. Thus, the current standard approach requires the use of short paired-end reads, which remain challenging to detect, especially at the scale of hundreds to thousands of samples. Findings: We present Parliament2, a consensus SV framework that leverages multiple best-in-class methods to identify high-quality SVs from short-read DNA sequence data at scale. Parliament2 incorporates pre-installed SV callers that are optimized for efficient execution in parallel to reduce the overall runtime and costs. We demonstrate the accuracy of Parliament2 when applied to data from NovaSeq and HiSeq X platforms with the Genome in a Bottle (GIAB) SV call set across all size classes. The reported quality score per SV is calibrated across different SV types and size classes. Parliament2 has the highest F1 score (74.27%) measured across the independent gold standard from GIAB. We illustrate the compute performance by processing all 1000 Genomes samples (2,691 samples) in <1 day on GRCH38. Parliament2 improves the runtime performance of individual methods and is open source (https://github.com/slzarate/parliament2), and a Docker image, as well as a WDL implementation, is available. Conclusion: Parliament2 provides both a highly accurate single-sample SV call set from short-read DNA sequence data and enables cost-efficient application over cloud or cluster environments, processing thousands of samples.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 背景：结构变异（SV）是遗传多样性和基因组疾病的关键因素。为了预测SV的表型影响，需要更好地估计SV的发生率和发生频率，最好是从大型，不同种族的人群中进行估计。因此，当前的标准方法要求使用短的配对末端读数，这对检测仍然具有挑战性，尤其是在成百上千个样本的规模上。调查结果：我们提出了Parliament2，这是一个共识性SV框架，它利用多种同类最佳的方法从短期读取的DNA序列数据中大规模地识别高质量的SV。Parliament2包含预安装的SV调用程序，这些调用程序已针对并行高效执行进行了优化，以减少总体运行时间和成本。我们证明了应用在所有大小类别的瓶中基因组（GIAB）SV调用集中的Nova2eq和HiSeqX平台数据中，Parliament2的准确性。所报告的每个SV质量得分是在不同SV类型和尺寸类别之间进行校准的。在GIAB的独立黄金标准中，Parliament2的F1得分最高（74.27％）。我们通过在GRCH38上处理少于1天的所有1000个基因组样本（2,691个样本）来说明计算性能。Parliament2提高了各个方法的运行时性能，并且是开源的（https://github.com/slzarate/parliament2），并且可以使用Docker映像以及WDL实现。结论：Parliament2可以从短读的DNA序列数据中提供高度准确的单样本SV呼叫集，并且可以在云或群集环境中以经济高效的方式应用，处理数千个样本。</p>

</body>
</html>

