<!DOCTYPE html> <html lang = "en">
 <body>

<h2>Pubmed_ID: 33307546</h2>

<h3>Genetic mechanisms of critical illness in Covid-19</h3>

<h3>Covid-19中重大疾病的遗传机制 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-020-03065-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33307546/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2015': '38.138', '2016': '40.137', '2017': '41.577', '2018': '43.07', '2019': '42.778'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/11/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Host-mediated lung inflammation is present,1 and drives mortality,2 in critical illness caused by Covid-19. Host genetic variants associated with critical illness may identify mechanistic targets for therapeutic development.3 Here we report the results of the GenOMICC (Genetics Of Mortality In Critical Care) genome-wide association study(GWAS) in 2244 critically ill Covid-19 patients from 208 UK intensive care units (ICUs). We identify and replicate novel genome-wide significant associations, on chr12q24.13 (rs10735079, p=1.65 [Formula: see text] 10-8) in a gene cluster encoding antiviral restriction enzyme activators (OAS1, OAS2, OAS3), on chr19p13.2 (rs2109069, p=2.3 [Formula: see text] 10-12) near the gene encoding tyrosine kinase 2 (TYK2), on chr19p13.3 (rs2109069, p=3.98 [Formula: see text] 10-12) within the gene encoding dipeptidyl peptidase 9 (DPP9), and on chr21q22.1 (rs2236757, p=4.99 [Formula: see text] 10-8) in the interferon receptor gene IFNAR2. We identify potential targets for repurposing of licensed medications: using Mendelian randomisation we found evidence in support of a causal link from low expression of IFNAR2, and high expression of TYK2, to life-threatening disease; transcriptome-wide association in lung tissue revealed that high expression of the monocyte/macrophage chemotactic receptor CCR2 is associated with severe Covid-19. Our results identify robust genetic signals relating to key host antiviral defence mechanisms, and mediators of inflammatory organ damage in Covid-19. Both mechanisms may be amenable to targeted treatment with existing drugs. Large-scale randomised clinical trials will be essential before any change to clinical practice.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在由Covid-19引起的严重疾病中，存在宿主介导的肺部炎症1并导致死亡2。与危重病相关的宿主遗传变异可能会确定治疗发展的机制靶点。3在此，我们报告了GenOMICC（危重病死亡率遗传学）全基因组关联研究（GWAS）在2244名来自208名重症Covid-19患者中的结果英国重症监护病房（ICU）。我们在编码chr19p13的抗病毒限制酶激活子（OAS1，OAS2，OAS3）的基因簇中，在chr12q24.13（rs10735079，p=1.65[公式：参见文本]10-8）上识别并复制新的全基因组重要关联。.2（rs2109069，p=2.3[公式：参见文本]10-12）在编码chr19p13.3的酪氨酸激酶2（TYK2）的基因附近（rs2109069，p=3.98[公式：参见文本]10-12）编码二肽基肽酶9（DPP9）的基因，以及干扰素受体基因IFNAR2中的chr21q22.1（rs2236757，p=4.99[公式：参见文本]10-8）。我们确定了重新使用许可药物的潜在目标：使用孟德尔随机化法，我们发现证据支持从低表达IFNAR2和高表达TYK2到威胁生命的疾病之间的因果关系；肺组织中转录组范围的关联揭示单核细胞/巨噬细胞趋化性受体CCR2的高表达与严重的Covid-19相关。我们的结果确定了与关键宿主抗病毒防御机制以及Covid-19中炎性器官损伤的介质有关的强大遗传信号。两种机制都可能适合使用现有药物进行靶向治疗。在改变临床实践之前，大规模的随机临床试验将是必不可少的。</p>

<h2>Pubmed_ID: 33307246</h2>

<h3>Genetic Architecture and Key Genes Controlling the Diversity of Oil Composition in Rice Grain</h3>

<h3>控制稻米油组成多样性的遗传结构和关键基因 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.molp.2020.12.001</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33307246/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Molecular plant  {'ISSN': '1674-2052', 'Name': 'molecular plant', '2015': '7.142', '2016': '8.827', '2017': '9.326', '2018': '10.812', '2019': '12.084'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Rice grain oil is a valuable nutrient source. However, the genetic basis of oil biosynthesis in rice grains remains unclear. In this study, we performed a genome-wide association study on oil composition and oil concentration in a diverse panel of 533 cultivated rice accessions. High variation for eleven oil-related traits was observed and oil composition was found to differentiate in rice grains among the subpopulations. We identified 46 loci that are significantly associated with grain oil concentration or composition, of which 16 were detected in three RIL populations. Twenty-six candidate genes encoding enzymes involved in oil metabolism were identified from these forty-six loci, of which four genes (OsPAL6, OsLIN6, OsMYR2 and OsFAE6) were found to contribute to natural variation in oil composition and show differentiation among the subpopulations. Interestingly, population genetic analyses revealed that specific haplotypes of OsPAL6 and OsLIN6 were selected in japonica rice. Based on these results, we propose a possible oil biosynthetic pathway in rice grains. Collectively, our study provides new insights into the genetic basis of oil biosynthesis in rice grains and would facilitate marker-based breeding of rice varieties with enhanced oil and grain quality.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 大米籽油是一种宝贵的营养来源。但是，尚不清楚水稻中油生物合成的遗传基础。在这项研究中，我们对533种栽培水稻种质中的油组成和油浓度进行了全基因组关联研究。观察到11个与油有关的性状变异很大，并且发现亚群体中的稻米籽粒中油的成分有所不同。我们确定了与谷物油浓度或组成显着相关的46个基因座，其中三个RIL种群中检测到16个。从这46个基因座中鉴定出二十六个候选基因编码油代谢相关的酶，其中四个基因（OsPAL6，OsLIN6，OsMYR2和OsFAE6）有助于油成分的自然变化并显示亚群之间的差异。有趣的是，群体遗传分析显示，在粳稻中选择了OsPAL6和OsLIN6的特定单倍型。基于这些结果，我们提出了水稻籽粒中可能的油脂生物合成途径。总体而言，我们的研究为水稻籽粒中石油生物合成的遗传基础提供了新见识，并有助于提高基于标记的水稻品种的籽粒育种，从而提高了石油和谷物的品质。</p>

<h2>Pubmed_ID: 33303764</h2>

<h3>Whole-genome sequencing analysis of the cardiometabolic proteome</h3>

<h3>代谢组蛋白质组的全基因组测序分析 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20079-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33303764/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/10/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The human proteome is a crucial intermediate between complex diseases and their genetic and environmental components, and an important source of drug development targets and biomarkers. Here, we comprehensively assess the genetic architecture of 257 circulating protein biomarkers of cardiometabolic relevance through high-depth (22.5×) whole-genome sequencing (WGS) in 1328 individuals. We discover 131 independent sequence variant associations (P < 7.45 × 10-11) across the allele frequency spectrum, all of which replicate in an independent cohort (n = 1605, 18.4x WGS). We identify for the first time replicating evidence for rare-variant cis-acting protein quantitative trait loci for five genes, involving both coding and noncoding variation. We construct and validate polygenic scores that explain up to 45% of protein level variation. We find causal links between protein levels and disease risk, identifying high-value biomarkers and drug development targets.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 人类蛋白质组学是复杂疾病及其遗传和环境成分之间的重要中介，也是药物开发目标和生物标志物的重要来源。在这里，我们通过深度（22.5x）全基因组测序（WGS）对1328位个体的257种代谢代谢相关循环蛋白生物标记物的遗传结构进行了全面评估。我们在等位基因频谱中发现了131个独立的序列变异关联（P<7.45×10-11），所有这些都在一个独立的队列中复制（n=1605，18.4xWGS）。我们首次确定了五个基因的稀有顺式作用蛋白定量性状基因座的复制证据，涉及编码和非编码变异。我们构建并验证了可解释高达45％的蛋白质水平变化的多基因评分。我们发现蛋白质水平与疾病风险之间的因果关系，确定了高价值的生物标志物和药物开发目标。</p>

<h2>Pubmed_ID: 33303762</h2>

<h3>Scalable multiple whole-genome alignment and locally collinear block construction with SibeliaZ</h3>

<h3>利用SibeliaZ进行可扩展的多个全基因组比对和局部共线模块构建 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-19777-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33303762/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/10/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Multiple whole-genome alignment is a challenging problem in bioinformatics. Despite many successes, current methods are not able to keep up with the growing number, length, and complexity of assembled genomes, especially when computational resources are limited. Approaches based on compacted de Bruijn graphs to identify and extend anchors into locally collinear blocks have potential for scalability, but current methods do not scale to mammalian genomes. We present an algorithm, SibeliaZ-LCB, for identifying collinear blocks in closely related genomes based on analysis of the de Bruijn graph. We further incorporate this into a multiple whole-genome alignment pipeline called SibeliaZ. SibeliaZ shows run-time improvements over other methods while maintaining accuracy. On sixteen recently-assembled strains of mice, SibeliaZ runs in under 16 hours on a single machine, while other tools did not run to completion for eight mice within a week. SibeliaZ makes a significant step towards improving scalability of multiple whole-genome alignment and collinear block reconstruction algorithms on a single machine.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 多个全基因组比对是生物信息学中一个具有挑战性的问题。尽管取得了许多成功，但当前的方法无法跟上组装基因组的数量，长度和复杂性的增长，特别是在计算资源有限的情况下。基于压缩的deBruijn图来识别锚并将锚扩展到局部共线块的方法具有可伸缩性的潜力，但当前的方法无法适应哺乳动物的基因组。我们提出了一种算法，即SibeliaZ-LCB，用于根据deBruijn图的分析来识别紧密相关的基因组中的共线区块。我们将其进一步整合到称为SibeliaZ的多个全基因组比对管道中。SibeliaZ显示了在保持准确性的同时，对其他方法的运行时改进。在最近组装的16种小鼠品系上，SibeliaZ在一台机器上运行时间不到16小时，而其他工具却无法在一周内完成对8只小鼠的加工。SibeliaZ朝着提高单个机器上的多个全基因组比对和共线块重构算法的可扩展性迈出了重要的一步。</p>

<h2>Pubmed_ID: 33303589</h2>

<h3>A transmissible cancer shifts from emergence to endemism in Tasmanian devils</h3>

<h3>塔斯马尼亚恶魔的可传播癌症从出现转移为地方病 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1126/science.abb9772</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33303589/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Science (New York, N.Y.)   ||| Not finding IF</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/11/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Emerging infectious diseases pose one of the greatest threats to human health and biodiversity. Phylodynamics is often used to infer epidemiological parameters essential for guiding intervention strategies for human viruses such as severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2). Here, we applied phylodynamics to elucidate the epidemiological dynamics of Tasmanian devil facial tumor disease (DFTD), a fatal, transmissible cancer with a genome thousands of times larger than that of any virus. Despite prior predictions of devil extinction, transmission rates have declined precipitously from ~3.5 secondary infections per infected individual to ~1 at present. Thus, DFTD appears to be transitioning from emergence to endemism, lending hope for the continued survival of the endangered Tasmanian devil. More generally, our study demonstrates a new phylodynamic analytical framework that can be applied to virtually any pathogen.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 新兴传染病是对人类健康和生物多样性的最大威胁之一。系统动力学经常被用来推断流行病学参数，这些参数对于指导人类病毒（如严重急性呼吸系统综合症冠状病毒2（SARS-Cov-2））的干预策略至关重要。在这里，我们应用系统动力学方法阐明了塔斯马尼亚恶魔面部肿瘤病（DFTD）的流行病学动态，该疾病是一种致命的，可传播的癌症，其基因组比任何病毒大数千倍。尽管先前有关于魔鬼灭绝的预测，但传播速度已从每名受感染个体约3.5次继发感染急剧下降至目前的1个。因此，DFTD似乎正从出现过渡到地方特有，为濒临灭绝的塔斯马尼亚恶魔的持续生存提供了希望。更广泛地说，我们的研究表明了一种新的系统动力学分析框架，该框架几乎可以应用于任何病原体。</p>

<h2>Pubmed_ID: 33303588</h2>

<h3>CiBER-seq dissects genetic networks by quantitative CRISPRi profiling of expression phenotypes</h3>

<h3>CiBER-seq通过表达表型的定量CRISPRi分析来解剖遗传网络 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1126/science.abb9662</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33303588/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Science (New York, N.Y.)   ||| Not finding IF</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/11/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: To realize the promise of CRISPR-Cas9-based genetics, approaches are needed to quantify a specific, molecular phenotype across genome-wide libraries of genetic perturbations. We addressed this challenge by profiling transcriptional, translational, and posttranslational reporters using CRISPR interference (CRISPRi) with barcoded expression reporter sequencing (CiBER-seq). Our barcoding approach allowed us to connect an entire library of guides to their individual phenotypic consequences using pooled sequencing. CiBER-seq profiling fully recapitulated the integrated stress response (ISR) pathway in yeast. Genetic perturbations causing uncharged transfer RNA (tRNA) accumulation activated ISR reporter transcription. Notably, tRNA insufficiency also activated the reporter, independent of the uncharged tRNA sensor. By uncovering alternate triggers for ISR activation, we illustrate how precise, comprehensive CiBER-seq profiling provides a powerful and broadly applicable tool for dissecting genetic networks.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 为了实现基于CRISPR-Cas9的遗传学的前景，需要采取方法对整个基因扰动的全基因组库中的特定分子表型进行定量。我们通过使用带有条形码表达报告子测序（CiBER-seq）的CRISPR干扰（CRISPRi）来分析转录，翻译和翻译后报告子来应对这一挑战。我们的条形码方法使我们可以使用合并测序将整个指南库与各自的表型后果联系起来。CiBER-seq分析完全概括了酵母中的整合应激反应（ISR）途径。遗传扰动引起电荷转移RNA（tRNA）积累，激活了ISR报告基因转录。值得注意的是，tRNA功能不全也会激活报告基因，而与不带电的tRNA传感器无关。通过揭示ISR激活的替代触发因素，我们说明了精确，全面的CiBER-seq分析如何为解剖遗传网络提供强大而广泛适用的工具。</p>

<h2>Pubmed_ID: 33303584</h2>

<h3>Developmental clock and mechanism of de novo polarization of the mouse embryo</h3>

<h3>小鼠胚胎的发育时钟和从头极化的机理 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1126/science.abd2703</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33303584/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Science (New York, N.Y.)   ||| Not finding IF</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/11/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Embryo polarization is critical for mouse development; however, neither the regulatory clock nor the molecular trigger that it activates is known. Here, we show that the embryo polarization clock reflects the onset of zygotic genome activation, and we identify three factors required to trigger polarization. Advancing the timing of transcription factor AP-2 gamma (Tfap2c) and TEA domain transcription factor 4 (Tead4) expression in the presence of activated Ras homolog family member A (RhoA) induces precocious polarization as well as subsequent cell fate specification and morphogenesis. Tfap2c and Tead4 induce expression of actin regulators that control the recruitment of apical proteins on the membrane, whereas RhoA regulates their lateral mobility, allowing the emergence of the apical domain. Thus, Tfap2c, Tead4, and RhoA are regulators for the onset of polarization and cell fate segregation in the mouse.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 胚胎极化对于小鼠发育至关重要。但是，调节时钟及其激活的分子触发器均未知。在这里，我们显示了胚胎极化时钟反映了合子基因组激活的开始，并且我们确定了触发极化所需的三个因素。在存在激活的Ras同源家族成员A（RhoA）的情况下提前转录因子AP-2γ（Tfap2c）和TEA域转录因子4（Tead4）的表达诱导早熟极化以及随后的细胞命运规范和形态发生。Tfap2c和Tead4诱导肌动蛋白调节剂的表达，该蛋白控制膜上顶端蛋白的募集，而RhoA调节其横向移动性，从而允许顶端结构域的出现。因此，Tfap2c，Tead4和RhoA是小鼠中极化和细胞命运分离发作的调节剂。</p>

<h2>Pubmed_ID: 33300949</h2>

<h3>Chromosome-level draft genome of a diploid plum (Prunus salicina)</h3>

<h3>二倍体李子（李属）的染色体水平草图基因组 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa130</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33300949/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2015': '7.463', '2016': '6.871', '2017': '7.267', '2018': '4.688', '2019': '5.993'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/10/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Background: Plums are one of the most economically important Rosaceae fruit crops and comprise dozens of species distributed across the world. Until now, only limited genomic information has been available for the genetic studies and breeding programs of plums. Prunus salicina, an important diploid plum species, plays a predominant role in modern commercial plum production. Here we selected P. salicina for whole-genome sequencing and present a chromosome-level genome assembly through the combination of Pacific Biosciences sequencing, Illumina sequencing, and Hi-C technology. Findings: The assembly had a total size of 284.2 Mb, with contig N50 of 1.78 Mb and scaffold N50 of 32.32 Mb. A total of 96.56% of the assembled sequences were anchored onto 8 pseudochromosomes, and 24,448 protein-coding genes were identified. Phylogenetic analysis showed that P. salicina had a close relationship with Prunus mume and Prunus armeniaca, with P. salicina diverging from their common ancestor ∼9.05 million years ago. During P. salicina evolution 146 gene families were expanded, and some cell wall-related GO terms were significantly enriched. It was noteworthy that members of the DUF579 family, a new class involved in xylan biosynthesis, were significantly expanded in P. salicina, which provided new insight into the xylan metabolism in plums. Conclusions: We constructed the first high-quality chromosome-level plum genome using Pacific Biosciences, Illumina, and Hi-C technologies. This work provides a valuable resource for facilitating plum breeding programs and studying the genetic diversity mechanisms of plums and Prunus species.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 背景：李子是经济上最重要的蔷薇科水果作物之一，由分布在世界各地的数十种物种组成。到目前为止，李子的遗传研究和育种程序仅可获得有限的基因组信息。樱桃李是重要的二倍体李子种类，在现代商业李子生产中起主要作用。在这里，我们选择了水杨假单胞菌进行全基因组测序，并结合PacificBiosciences测序，Illumina测序和Hi-C技术介绍了染色体水平的基因组装配。结果：该组件的总大小为284.2Mb，重叠群N50为1.78Mb，支架N50为32.32Mb。总共96.56％的组装序列被锚定在8个假染色体上，并鉴定出24,448个蛋白质编码基因。系统发育分析表明，水杨假单胞菌与梅树和杏李有密切的关系，水杨假单胞菌距其共同祖先约905万年前。在水杨假单胞菌进化期间，扩展了146个基因家族，并且显着丰富了一些与细胞壁相关的GO术语。值得注意的是，参与木聚糖生物合成的新类别DUF579家族的成员在水杨假单胞菌中得到了显着扩展，这为李子木聚糖代谢提供了新的见识。结论：我们使用PacificBiosciences，Illumina和Hi-C技术构建了第一个高质量的染色体级李子基因组。这项工作为促进李子育种计划和研究李子和李属物种的遗传多样性机制提供了宝贵的资源。</p>

<h2>Pubmed_ID: 33300184</h2>

<h3>The genome of Shanputao (Vitis amurensis) provides a new insight into cold tolerance of grapevine</h3>

<h3>Shanputao（Vitis amurensis）的基因组为葡萄的耐寒性提供了新的见解 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15127</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33300184/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology   ||| Not finding IF</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/10/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Vitis amurensis (Shanputao) is the most cold tolerant Vitis species and so is of great interest to grape breeders and producers in areas with low winter temperatures. Here, we report its high-quality, chromosome-level genome assembly based on a combination of sequence data from Illumina and PacBio platforms, BioNano optical mapping and high-throughput chromosome conformation Capture (Hi-C) mapping. The 604.56 Mb genome contains 32,885 protein coding genes. Shanputao was found to share a common ancestor with PN40024 (V. vinifera) approximately 2.17 - 2.91MYA, and gene expansion observed in Shanputao might contribute to the enhancement of cold tolerance. Transcriptome analysis revealed seventeen genes involved in cold signal transduction, suggesting that there were different response mechanism to chilling temperature and freezing conditions. Furthermore, a genome wide association study uncovered a phosphoglycerate kinase gene that may contribute to the freezing resistance of buds in the winter. The Shanputao genome sequence not only represents a valuable resource for grape breeders, but also is important for clarifying the molecular mechanisms involved in cold tolerance.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 紫葡萄（Shanputao）是最耐寒的葡萄品种，因此在冬季气温较低的地区，葡萄育种者和生产者都非常感兴趣。在这里，我们根据Illumina和PacBio平台的序列数据，BioNano光学作图和高通量染色体构象捕获（Hi-C）作图的组合，报告了其高质量的染色体级基因组装配。604.56Mb基因组包含32,885个蛋白质编码基因。已发现Shanputao与PN40024（V.vinifera）共有一个共同祖先，大约为2.17-2.91MYA，并且在Shanputao中观察到的基因扩增可能有助于增强耐寒性。转录组分析揭示了十七个基因参与冷信号转导，表明对低温和冷冻条件的响应机制不同。此外，一项全基因组关联研究发现了磷酸甘油酸激酶基因，该基因可能有助于冬季芽的抗冻性。Shanputao基因组序列不仅为葡萄育种者提供了宝贵的资源，而且对于阐明涉及耐寒性的分子机制也很重要。</p>

<h2>Pubmed_ID: 33299182</h2>

<h3>H1 histones control the epigenetic landscape by local chromatin compaction</h3>

<h3>H1组蛋白通过局部染色质压实控制表观遗传景观 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-020-3032-z</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33299182/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2015': '38.138', '2016': '40.137', '2017': '41.577', '2018': '43.07', '2019': '42.778'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/09/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: H1 linker histones are the most abundant chromatin-binding proteins1. In vitro studies indicate that their association with chromatin determines nucleosome spacing and enables arrays of nucleosomes to fold into more compact chromatin structures. However, the in vivo roles of H1 are poorly understood2. Here we show that the local density of H1 controls the balance of repressive and active chromatin domains by promoting genomic compaction. We generated a conditional triple-H1-knockout mouse strain and depleted H1 in haematopoietic cells. H1 depletion in T cells leads to de-repression of T cell activation genes, a process that mimics normal T cell activation. Comparison of chromatin structure in normal and H1-depleted CD8+ T cells reveals that H1-mediated chromatin compaction occurs primarily in regions of the genome containing higher than average levels of H1: the chromosome conformation capture (Hi-C) B compartment and regions of the Hi-C A compartment marked by PRC2. Reduction of H1 stoichiometry leads to decreased H3K27 methylation, increased H3K36 methylation, B-to-A-compartment shifting and an increase in interaction frequency between compartments. In vitro, H1 promotes PRC2-mediated H3K27 methylation and inhibits NSD2-mediated H3K36 methylation. Mechanistically, H1 mediates these opposite effects by promoting physical compaction of the chromatin substrate. Our results establish H1 as a critical regulator of gene silencing through localized control of chromatin compaction, 3D genome organization and the epigenetic landscape.</p>

<p style="line-height:1.8;margin:20px"> 摘要: H1接头组蛋白是最丰富的染色质结合蛋白1。体外研究表明，它们与染色质的结合决定了核小体的间隔，并使核小体的阵列折叠成更紧凑的染色质结构。然而，人们对H1在体内的作用了解甚少2。在这里，我们显示H1的局部密度通过促进基因组紧缩来控制阻抑和活跃染色质域的平衡。我们生成了条件三联H1基因敲除小鼠品系，并在造血细胞中消耗了H1。T细胞中H1的消耗导致T细胞活化基因的抑制，该过程模仿正常的T细胞活化。正常和H1耗尽的CD8+T细胞中染色质结构的比较表明，H1介导的染色质紧缩主要发生在基因组中的H1水平高于平均水平：染色体构象捕获（Hi-C）B区和Hi-C用PRC2标记的车厢。H1化学计量的降低导致H3K27甲基化减少，H3K36甲基化增加，B至A隔室移位以及隔室之间的相互作用频率增加。在体外，H1促进PRC2介导的H3K27甲基化并抑制NSD2介导的H3K36甲基化。在机理上，H1通过促进染色质底物的物理压实来介导这些相反的作用。我们的结果通过局部控制染色质紧实度，3D基因组组织和表观遗传景观，将H1确立为基因沉默的关键调节因子。</p>

<h2>Pubmed_ID: 33299181</h2>

<h3>Histone H1 loss drives lymphoma by disrupting 3D chromatin architecture</h3>

<h3>组蛋白H1丢失通过破坏3D染色质结构驱动淋巴瘤 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-020-3017-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33299181/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2015': '38.138', '2016': '40.137', '2017': '41.577', '2018': '43.07', '2019': '42.778'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/09/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Linker histone H1 proteins bind to nucleosomes and facilitate chromatin compaction1, although their biological functions are poorly understood. Mutations in the genes that encode H1 isoforms B-E (H1B, H1C, H1D and H1E; also known as H1-5, H1-2, H1-3 and H1-4, respectively) are highly recurrent in B cell lymphomas, but the pathogenic relevance of these mutations to cancer and the mechanisms that are involved are unknown. Here we show that lymphoma-associated H1 alleles are genetic driver mutations in lymphomas. Disruption of H1 function results in a profound architectural remodelling of the genome, which is characterized by large-scale yet focal shifts of chromatin from a compacted to a relaxed state. This decompaction drives distinct changes in epigenetic states, primarily owing to a gain of histone H3 dimethylation at lysine 36 (H3K36me2) and/or loss of repressive H3 trimethylation at lysine 27 (H3K27me3). These changes unlock the expression of stem cell genes that are normally silenced during early development. In mice, loss of H1c and H1e (also known as H1f2 and H1f4, respectively) conferred germinal centre B cells with enhanced fitness and self-renewal properties, ultimately leading to aggressive lymphomas with an increased repopulating potential. Collectively, our data indicate that H1 proteins are normally required to sequester early developmental genes into architecturally inaccessible genomic compartments. We also establish H1 as a bona fide tumour suppressor and show that mutations in H1 drive malignant transformation primarily through three-dimensional genome reorganization, which leads to epigenetic reprogramming and derepression of developmentally silenced genes.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 接头组蛋白H1蛋白与核小体结合并促进染色质紧实1，尽管对其生物学功能了解甚少。编码H1亚型BE的基因中的突变（H1B，H1C，H1D和H1E；分别称为H1-5，H1-2，H1-3和H1-4）在B细胞淋巴瘤中高度复发，但具有致病性这些突变与癌症的相关性及其涉及的机制尚不清楚。在这里，我们显示淋巴瘤相关的H1等位基因是淋巴瘤中的遗传驱动基因突变。H1功能的破坏导致基因组的深刻的结构重塑，其特征在于染色质从紧实状态到松弛状态的大规模但焦点转移。这种分解作用驱动表观遗传状态发生明显变化，这主要是由于赖氨酸36（H3K36me2）处的组蛋白H3二甲基化增加和/或赖氨酸27（H3K27me3）处的抑制性H3三甲基化损失。这些变化解锁了通常在早期发育过程中沉默的干细胞基因的表达。在小鼠中，H1c和H1e（分别也称为H1f2和H1f4）的丧失赋予生发中心B细胞增强的适应性和自我更新特性，最终导致侵袭性淋巴瘤的繁殖潜力增加。总体而言，我们的数据表明，通常需要H1蛋白才能将早期发育基因隔离到结构上难以接近的基因组区室中。我们还将H1建立为真正的肿瘤抑制因子，并证明H1中的突变主要通过三维基因组重组来驱动恶性转化，从而导致表观遗传重编程和发育沉默基因的抑制。</p>

<h2>Pubmed_ID: 33299166</h2>

<h3>CRISPR gene therapy shows promise against blood diseases</h3>

<h3>CRISPR基因疗法显示出对抗血液疾病的希望 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/d41586-020-03476-x</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33299166/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2015': '38.138', '2016': '40.137', '2017': '41.577', '2018': '43.07', '2019': '42.778'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 没有摘要</p>

<h2>Pubmed_ID: 33298935</h2>

<h3>Analytical validity of nanopore sequencing for rapid SARS-CoV-2 genome analysis</h3>

<h3>纳米孔测序对SARS-CoV-2基因组快速分析的分析有效性 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20075-6</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33298935/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/09/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Viral whole-genome sequencing (WGS) provides critical insight into the transmission and evolution of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). Long-read sequencing devices from Oxford Nanopore Technologies (ONT) promise significant improvements in turnaround time, portability and cost, compared to established short-read sequencing platforms for viral WGS (e.g., Illumina). However, adoption of ONT sequencing for SARS-CoV-2 surveillance has been limited due to common concerns around sequencing accuracy. To address this, here we perform viral WGS with ONT and Illumina platforms on 157 matched SARS-CoV-2-positive patient specimens and synthetic RNA controls, enabling rigorous evaluation of analytical performance. We report that, despite the elevated error rates observed in ONT sequencing reads, highly accurate consensus-level sequence determination was achieved, with single nucleotide variants (SNVs) detected at >99% sensitivity and >99% precision above a minimum ~60-fold coverage depth, thereby ensuring suitability for SARS-CoV-2 genome analysis. ONT sequencing also identified a surprising diversity of structural variation within SARS-CoV-2 specimens that were supported by evidence from short-read sequencing on matched samples. However, ONT sequencing failed to accurately detect short indels and variants at low read-count frequencies. This systematic evaluation of analytical performance for SARS-CoV-2 WGS will facilitate widespread adoption of ONT sequencing within local, national and international COVID-19 public health initiatives.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 病毒全基因组测序（WGS）提供了对严重急性呼吸系统综合症冠状病毒2（SARS-CoV-2）的传播和进化的重要见解。与已建立的病毒WGS短读测序平台（例如Illumina）相比，牛津纳米孔技术（ONT）的长读测序设备有望在周转时间，可移植性和成本方面取得显着改善。但是，由于对测序准确性的普遍关注，在SARS-CoV-2监视中采用ONT测序受到了限制。为了解决这个问题，我们在157个匹配的SARS-CoV-2阳性患者标本和合成RNA对照样品上使用ONT和Illumina平台进行了病毒WGS分析，从而可以对分析性能进行严格评估。我们报告，尽管在ONT测序读取中观察到错误率升高，但仍实现了高度准确的共有水平序列测定，检测到的单核苷酸变体（SNV）的灵敏度大于99％，而精密度大于99％，至少约为60倍以上覆盖深度，从而确保适用于SARS-CoV-2基因组分析。ONT测序还发现了SARS-CoV-2标本中令人惊讶的结构变异多样性，这得益于对匹配样品进行短读测序的证据。但是，ONT测序无法在低读取次数频率下准确检测到短插入缺失和变异。对SARS-CoV-2WGS的分析性能的系统评估将促进ONT测序在本地，国家和国际COVID-19公共卫生计划中的广泛采用。</p>

<h2>Pubmed_ID: 33298918</h2>

<h3>Primary effusion lymphoma enhancer connectome links super-enhancers to dependency factors</h3>

<h3>原发性积液淋巴瘤增强剂连接组将超级增强剂与依赖因子联系起来 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20136-w</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33298918/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/09/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Primary effusion lymphoma (PEL) has a very poor prognosis. To evaluate the contributions of enhancers/promoters interactions to PEL cell growth and survival, here we produce H3K27ac HiChIP datasets in PEL cells. This allows us to generate the PEL enhancer connectome, which links enhancers and promoters in PEL genome-wide. We identify more than 8000 genomic interactions in each PEL cell line. By incorporating HiChIP data with H3K27ac ChIP-seq data, we identify interactions between enhancers/enhancers, enhancers/promoters, and promoters/promoters. HiChIP further links PEL super-enhancers to PEL dependency factors MYC, IRF4, MCL1, CCND2, MDM2, and CFLAR. CRISPR knock out of MEF2C and IRF4 significantly reduces MYC and IRF4 super-enhancer H3K27ac signal. Knock out also reduces MYC and IRF4 expression. CRISPRi perturbation of these super-enhancers by tethering transcription repressors to enhancers significantly reduces target gene expression and reduces PEL cell growth. These data provide insights into PEL molecular pathogenesis.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 原发性积液性淋巴瘤（PEL）的预后很差。为了评估增强子/启动子相互作用对PEL细胞生长和存活的贡献，在这里我们在PEL细胞中产生H3K27acHiChIP数据集。这使我们能够生成PEL增强子连接体，该连接体连接整个PEL基因组中的增强子和启动子。我们在每个PEL细胞系中鉴定出8000多个基因组相互作用。通过将HiChIP数据与H3K27acChIP-seq数据合并，我们确定了增强子/增强子，增强子/启动子和启动子/启动子之间的相互作用。HiChIP进一步将PEL超级增强器与PEL依赖因子MYC，IRF4，MCL1，CCND2，MDM2和CFLAR关联。CRISPR敲除MEF2C和IRF4可显着降低MYC和IRF4超级增强子H3K27ac信号。敲除还降低了MYC和IRF4表达。通过将转录阻遏物束缚到增强子上，CRISPRi对这些超级增强子的干扰显着降低了靶基因的表达并降低了PEL细胞的生长。这些数据提供了有关PEL分子发病机理的见解。</p>

<h2>Pubmed_ID: 33293555</h2>

<h3>Design of efficacious somatic cell genome editing strategies for recessive and polygenic diseases</h3>

<h3>隐性和多基因疾病有效的体细胞基因组编辑策略设计 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20065-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33293555/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Compound heterozygous recessive or polygenic diseases could be addressed through gene correction of multiple alleles. However, targeting of multiple alleles using genome editors could lead to mixed genotypes and adverse events that amplify during tissue morphogenesis. Here we demonstrate that Cas9-ribonucleoprotein-based genome editors can correct two distinct mutant alleles within a single human cell precisely. Gene-corrected cells in an induced pluripotent stem cell model of Pompe disease expressed the corrected transcript from both corrected alleles, leading to enzymatic cross-correction of diseased cells. Using a quantitative in silico model for the in vivo delivery of genome editors into the developing human infant liver, we identify progenitor targeting, delivery efficiencies, and suppression of imprecise editing outcomes at the on-target site as key design parameters that control the efficacy of various therapeutic strategies. This work establishes that precise gene editing to correct multiple distinct gene variants could be highly efficacious if designed appropriately.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 复合杂合性隐性或多基因疾病可以通过多个等位基因的基因校正来解决。但是，使用基因组编辑器靶向多个等位基因可能会导致混合的基因型和不良事件在组织形态发生过程中放大。在这里，我们证明基于Cas9-核糖核蛋白的基因组编辑器可以精确地纠正单个人类细胞内的两个不同的突变等位基因。诱导的庞贝病多能干细胞模型中的基因校正细胞表达了两个校正等位基因的校正转录本，从而导致患病细胞的酶促交叉校正。我们使用定量的计算机模拟模型将基因组编辑器体内递送到发育中的人类婴儿肝脏中，我们确定了祖细胞靶向，递送效率以及在靶位点抑制不精确的编辑结果作为控制药物疗效的关键设计参数。各种治疗策略。这项工作建立了正确的基因编辑以纠正多个不同的基因变异可能是非常有效的，如果设计适当的话。</p>

<h2>Pubmed_ID: 33293552</h2>

<h3>SAMMY-seq reveals early alteration of heterochromatin and deregulation of bivalent genes in Hutchinson-Gilford Progeria Syndrome</h3>

<h3>SAMMY-seq揭示了Hutchinson-Gilford早衰综合症的异染色质的早期改变和二价基因的失控 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20048-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33293552/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Hutchinson-Gilford progeria syndrome is a genetic disease caused by an aberrant form of Lamin A resulting in chromatin structure disruption, in particular by interfering with lamina associated domains. Early molecular alterations involved in chromatin remodeling have not been identified thus far. Here, we present SAMMY-seq, a high-throughput sequencing-based method for genome-wide characterization of heterochromatin dynamics. Using SAMMY-seq, we detect early stage alterations of heterochromatin structure in progeria primary fibroblasts. These structural changes do not disrupt the distribution of H3K9me3 in early passage cells, thus suggesting that chromatin rearrangements precede H3K9me3 alterations described at later passages. On the other hand, we observe an interplay between changes in chromatin accessibility and Polycomb regulation, with site-specific H3K27me3 variations and transcriptional dysregulation of bivalent genes. We conclude that the correct assembly of lamina associated domains is functionally connected to the Polycomb repression and rapidly lost in early molecular events of progeria pathogenesis.</p>

<p style="line-height:1.8;margin:20px"> 摘要: Hutchinson-Gilford早衰综合症是一种遗传疾病，由LaminA的异常形式引起，导致染色质结构破坏，特别是通过干扰与叶片相关的结构域。到目前为止，尚未确定与染色质重塑有关的早期分子改变。在这里，我们介绍了SAMMY-seq，这是一种基于高通量测序的异染色质动力学全基因组表征方法。使用SAMMY序列，我们检测早衰原代成纤维细胞中异染色质结构的早期变化。这些结构变化不会破坏H3K9me3在早期传代细胞中的分布，因此表明染色质重排先于H3K9me3在以后的传代中描述的改变。另一方面，我们观察到染色质可及性和Polycomb调控之间的相互作用，以及特定位点的H3K27me3变异和二价基因的转录失调。我们得出的结论是，层板相关域的正确组装在功能上与Polycomb抑制相关，并在早衰发病机理的早期分子事件中迅速丧失。</p>

<h2>Pubmed_ID: 33293549</h2>

<h3>Cerebral small vessel disease genomics and its implications across the lifespan</h3>

<h3>脑小血管疾病基因组学及其在整个生命周期中的意义 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-19111-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33293549/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: White matter hyperintensities (WMH) are the most common brain-imaging feature of cerebral small vessel disease (SVD), hypertension being the main known risk factor. Here, we identify 27 genome-wide loci for WMH-volume in a cohort of 50,970 older individuals, accounting for modification/confounding by hypertension. Aggregated WMH risk variants were associated with altered white matter integrity (p = 2.5×10-7) in brain images from 1,738 young healthy adults, providing insight into the lifetime impact of SVD genetic risk. Mendelian randomization suggested causal association of increasing WMH-volume with stroke, Alzheimer-type dementia, and of increasing blood pressure (BP) with larger WMH-volume, notably also in persons without clinical hypertension. Transcriptome-wide colocalization analyses showed association of WMH-volume with expression of 39 genes, of which four encode known drug targets. Finally, we provide insight into BP-independent biological pathways underlying SVD and suggest potential for genetic stratification of high-risk individuals and for genetically-informed prioritization of drug targets for prevention trials.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 白质高信号（WMH）是脑小血管疾病（SVD）最常见的脑成像特征，高血压是已知的主要危险因素。在这里，我们确定了50970个老年人群中WMH量的27个全基因组位点，这是高血压引起的修饰/混淆。总计的WMH风险变量与来自1,738名年轻健康成年人的脑部图像中白质完整性改变（p=2.5×10-7）相关，从而提供了对SVD遗传风险一生影响的见解。孟德尔随机化表明WMH量增加与中风，阿尔茨海默氏型痴呆和血压（BP）升高与WMH量增加之间存在因果关系，尤其是在没有临床高血压的人群中。整个转录组共定位分析显示WMH量与39个基因的表达相关，其中四个编码已知的药物靶标。最后，我们深入了解SVD背后的BP非依赖性生物途径，并提出了对高风险个体进行遗传分层和为预防性试验确定药物靶标的遗传信息优先次序的潜力。</p>

<h2>Pubmed_ID: 33293539</h2>

<h3>Functional studies of GWAS variants are gaining momentum</h3>

<h3>GWAS变体的功能研究正在蓬勃发展 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20188-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33293539/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Rapidly advancing genomic technologies and cross-disciplinary partnerships are accelerating the biological and clinical interpretation of genome-wide association studies, with some therapies developed based on these findings already being tested in clinical trials. The next decade promises further progress in understanding the function of genetic variants.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 快速发展的基因组技术和跨学科的合作伙伴关系正在加速全基因组关联研究的生物学和临床解释，基于这些发现开发的一些疗法已经在临床试验中进行了测试。下一个十年有望在理解遗传变异的功能方面取得进一步的进展。</p>

<h2>Pubmed_ID: 33293536</h2>

<h3>Pontin arginine methylation by CARM1 is crucial for epigenetic regulation of autophagy</h3>

<h3>CARM1的Pontin精氨酸甲基化对于自噬的表观遗传调控至关重要 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20080-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33293536/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Autophagy is a catabolic process through which cytoplasmic components are degraded and recycled in response to various stresses including starvation. Recently, transcriptional and epigenetic regulations of autophagy have emerged as essential mechanisms for maintaining homeostasis. Here, we identify that coactivator-associated arginine methyltransferase 1 (CARM1) methylates Pontin chromatin-remodeling factor under glucose starvation, and methylated Pontin binds Forkhead Box O 3a (FOXO3a). Genome-wide analyses and biochemical studies reveal that methylated Pontin functions as a platform for recruiting Tip60 histone acetyltransferase with increased H4 acetylation and subsequent activation of autophagy genes regulated by FOXO3a. Surprisingly, CARM1-Pontin-FOXO3a signaling axis can work in the distal regions and activate autophagy genes through enhancer activation. Together, our findings provide a signaling axis of CARM1-Pontin-FOXO3a and further expand the role of CARM1 in nuclear regulation of autophagy.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 自噬是一种分解代谢过程，通过该过程，细胞质组分响应包括饥饿在内的各种应激而降解和再循环。最近，自噬的转录和表观遗传调控已成为维持体内稳态的必要机制。在这里，我们确定在葡萄糖饥饿状态下，与激活剂相关的精氨酸甲基转移酶1（CARM1）甲基化Pontin染色质重塑因子，而甲基化的Pontin结合叉头盒O3a（FOXO3a）。全基因组范围的分析和生化研究表明，甲基化的庞廷蛋白可作为招募Tip60组蛋白乙酰转移酶的平台，具有增加的H4乙酰化作用，并随后激活由FOXO3a调控的自噬基因。出人意料的是，CARM1-Pontin-FOXO3a信号轴可以在远端区域起作用，并通过增强子激活来激活自噬基因。总之，我们的发现提供了CARM1-Pontin-FOXO3a的信号转导轴，并进一步扩展了CARM1在自噬核调控中的作用。</p>

<h2>Pubmed_ID: 33293529</h2>

<h3>Donkey genomes provide new insights into domestication and selection for coat color</h3>

<h3>驴基因组为外套颜色的驯化和选择提供了新见识 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-19813-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33293529/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2015': '11.329', '2016': '12.124', '2017': '12.353', '2018': '11.878', '2019': '12.121'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Current knowledge about the evolutionary history of donkeys is still incomplete due to the lack of archeological and whole-genome diversity data. To fill this gap, we have de novo assembled a chromosome-level reference genome of one male Dezhou donkey and analyzed the genomes of 126 domestic donkeys and seven wild asses. Population genomics analyses indicate that donkeys were domesticated in Africa and conclusively show reduced levels of Y chromosome variability and discordant paternal and maternal histories, possibly reflecting the consequences of reproductive management. We also investigate the genetic basis of coat color. While wild asses show diluted gray pigmentation (Dun phenotype), domestic donkeys display non-diluted black or chestnut coat colors (non-Dun) that were probably established during domestication. Here, we show that the non-Dun phenotype is caused by a 1 bp deletion downstream of the TBX3 gene, which decreases the expression of this gene and its inhibitory effect on pigment deposition.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 由于缺乏考古和全基因组多样性数据，有关驴进化史的当前知识仍然不完整。为了填补这一空白，我们从头开始组装了一只德州公驴的染色体水平参考基因组，并分析了126只家养驴和7只野驴的基因组。种群基因组学分析表明，驴是在非洲驯化的，并最终显示出降低的Y染色体变异水平和不统一的父系和母系历史，可能反映了生殖管理的后果。我们还调查了外套颜色的遗传基础。野驴显示出淡淡的灰色色素沉着（邓表型），而家驴则表现出未稀释的黑色或栗色外套色（非登），这可能是在驯化过程中确定的。在这里，我们显示非Dun表型是由TBX3基因下游1bp缺失引起的，这会降低该基因的表达及其对色素沉积的抑制作用。</p>

<h2>Pubmed_ID: 33239447</h2>

<h3>KRAB zinc finger protein diversification drives mammalian interindividual methylation variability</h3>

<h3>KRAB锌指蛋白多样化驱动哺乳动物个体间甲基化变异 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2017053117</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33239447/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2015': '9.423', '2016': '9.661', '2017': '9.504', '2018': '9.58', '2019': '9.412'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Most transposable elements (TEs) in the mouse genome are heavily modified by DNA methylation and repressive histone modifications. However, a subset of TEs exhibit variable methylation levels in genetically identical individuals, and this is associated with epigenetically conferred phenotypic differences, environmental adaptability, and transgenerational epigenetic inheritance. The evolutionary origins and molecular mechanisms underlying interindividual epigenetic variability remain unknown. Using a repertoire of murine variably methylated intracisternal A-particle (VM-IAP) epialleles as a model, we demonstrate that variable DNA methylation states at TEs are highly susceptible to genetic background effects. Taking a classical genetics approach coupled with genome-wide analysis, we harness these effects and identify a cluster of KRAB zinc finger protein (KZFP) genes that modifies VM-IAPs in trans in a sequence-specific manner. Deletion of the cluster results in decreased DNA methylation levels and altered histone modifications at the targeted VM-IAPs. In some cases, these effects are accompanied by dysregulation of neighboring genes. We find that VM-IAPs cluster together phylogenetically and that this is linked to differential KZFP binding, suggestive of an ongoing evolutionary arms race between TEs and this large family of epigenetic regulators. These findings indicate that KZFP divergence and concomitant evolution of DNA binding capabilities are mechanistically linked to methylation variability in mammals, with implications for phenotypic variation and putative paradigms of mammalian epigenetic inheritance.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 小鼠基因组中的大多数转座因子（TEs）被DNA甲基化和抑制性组蛋白修饰严重修饰。但是，TE的一个子集在遗传上相同的个体中表现出可变的甲基化水平，这与表观遗传赋予的表型差异，环境适应性和跨代表观遗传有关。个体间表观遗传变异性的进化起源和分子机制仍然未知。使用小鼠可变甲基化的脑池内A粒子（VM-IAP）Epialleles作为模型，我们证明TEs的可变DNA甲基化状态高度易受遗传背景的影响。我们采用经典的遗传学方法并结合全基因组分析，我们利用了这些效应，并鉴定了一系列KRAB锌指蛋白（KZFP）基因，它们以序列特异性方式反式修饰VM-IAP。簇的删除导致DNA甲基化水平降低，并改变了靶向VM-IAP的组蛋白修饰。在某些情况下，这些影响伴随着邻近基因的失调。我们发现VM-IAPs系统发生群集在一起，并且这与不同的KZFP绑定相关联，表明TEs和这个大的表观遗传调节子之间正在进行的进化军备竞赛。这些发现表明，在哺乳动物中，KZFP的发散性和DNA结合能力的伴随进化与甲基化变异性存在机械联系，对表型变异和哺乳动物表观遗传的假定范式具有影响。</p>

<h2>Pubmed_ID: 33229585</h2>

<h3>Mycena genomes resolve the evolution of fungal bioluminescence</h3>

<h3>Mycena基因组解决了真菌生物发光的进化 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2010761117</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33229585/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2015': '9.423', '2016': '9.661', '2017': '9.504', '2018': '9.58', '2019': '9.412'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Mushroom-forming fungi in the order Agaricales represent an independent origin of bioluminescence in the tree of life; yet the diversity, evolutionary history, and timing of the origin of fungal luciferases remain elusive. We sequenced the genomes and transcriptomes of five bonnet mushroom species (Mycena spp.), a diverse lineage comprising the majority of bioluminescent fungi. Two species with haploid genome assemblies ∼150 Mb are among the largest in Agaricales, and we found that a variety of repeats between Mycena species were differentially mediated by DNA methylation. We show that bioluminescence evolved in the last common ancestor of mycenoid and the marasmioid clade of Agaricales and was maintained through at least 160 million years of evolution. Analyses of synteny across genomes of bioluminescent species resolved how the luciferase cluster was derived by duplication and translocation, frequently rearranged and lost in most Mycena species, but conserved in the Armillaria lineage. Luciferase cluster members were coexpressed across developmental stages, with the highest expression in fruiting body caps and stipes, suggesting fruiting-related adaptive functions. Our results contribute to understanding a de novo origin of bioluminescence and the corresponding gene cluster in a diverse group of enigmatic fungal species.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 伞菌顺序的蘑菇形成真菌代表了生命树中生物发光的独立起源；然而，真菌荧光素酶的起源的多样性，进化史和时机仍然难以捉摸。我们对五种帽子蘑菇物种（Mycenaspp。）的基因组和转录组进行了测序，这是一个包括大多数生物发光真菌的多样化谱系。Agaricales中有两个单倍体基因组装配体约为150Mb的物种，我们发现Mycena物种之间的多种重复是通过DNA甲基化差异介导的。我们表明，生物发光在类真菌的最后一个祖先和伞菌类的马拉什米进化枝中进化，并且通过至少1.6亿年的进化得以维持。跨生物发光物种基因组的同构分析解决了萤光素酶簇是如何通过重复和易位而衍生的，在大多数Mycena物种中经常重新排列和丢失，但在蜜环菌谱系中是保守的。萤光素酶簇成员在整个发育阶段共表达，在子实体的顶盖和柄中表达最高，表明子实体相关的适应功能。我们的结果有助于理解生物发光的从头起源以及在各种神秘真菌物种中的相应基因簇。</p>

<h2>Pubmed_ID: 33229517</h2>

<h3>Unbiased optical mapping of telomere-integrated endogenous human herpesvirus 6</h3>

<h3>端粒整合的内源性人类疱疹病毒6的无偏光学定位 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2011872117</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33229517/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2015': '9.423', '2016': '9.661', '2017': '9.504', '2018': '9.58', '2019': '9.412'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Next-generation sequencing technologies allowed sequencing of thousands of genomes. However, there are genomic regions that remain difficult to characterize, including telomeres, centromeres, and other low-complexity regions, as well as transposable elements and endogenous viruses. Human herpesvirus 6A and 6B (HHV-6A and HHV-6B) are closely related viruses that infect most humans and can integrate their genomes into the telomeres of infected cells. Integration also occurs in germ cells, meaning that the virus can be inherited and result in individuals harboring the virus in every cell of their body. The integrated virus can reactivate and cause disease in humans. While it is well established that the virus resides in the telomere region, the integration locus is poorly defined due to the low sequence complexity (TTAGGG)n of telomeres that cannot be easily resolved through sequencing. We therefore employed genome imaging of the integrated HHV-6A and HHV-6B genomes using whole-genome optical site mapping technology. Using this technology, we identified which chromosome arm harbors the virus genome and obtained a high-resolution map of the integration loci of multiple patients. Surprisingly, this revealed long telomere sequences at the virus-subtelomere junction that were previously missed using PCR-based approaches. Contrary to what was previously thought, our technique revealed that the telomere lengths of chromosomes harboring the integrated virus genome were comparable to the other chromosomes. Taken together, our data shed light on the genetic structure of the HHV-6A and HHV-6B integration locus, demonstrating the utility of optical mapping for the analysis of genomic regions that are difficult to sequence.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 下一代测序技术可对数千个基因组进行测序。但是，仍然存在难以表征的基因组区域，包括端粒，着丝粒和其他低复杂度区域，以及转座因子和内源性病毒。人类疱疹病毒6A和6B（HHV-6A和HHV-6B）是密切相关的病毒，会感染大多数人，并且可以将其基因组整合到受感染细胞的端粒中。整合还发生在生殖细胞中，这意味着该病毒可以被遗传并导致个人在其身体的每个细胞中都携带该病毒。整合的病毒可以重新激活并引起人类疾病。尽管已经确定病毒驻留在端粒区域，但由于端粒的序列复杂度（TTAGGG）n低，无法通过测序轻松解决，因此整合位点定义不明确。因此，我们使用全基因组光学位点定位技术对整合的HHV-6A和HHV-6B基因组进行了基因组成像。使用这项技术，我们确定了哪个染色体臂上带有病毒基因组，并获得了多个患者整合位点的高分辨率图。令人惊讶的是，这揭示了病毒-亚端粒连接处的长端粒序列，以前使用基于PCR的方法会丢失。与以前的想法相反，我们的技术表明，带有整合病毒基因组的染色体的端粒长度与其他染色体相当。综上所述，我们的数据阐明了HHV-6A和HHV-6B整合基因座的遗传结构，证明了光学作图法可用于分析难以测序的基因组区域。</p>

<h2>Pubmed_ID: 33203681</h2>

<h3>Global analysis of more than 50,000 SARS-CoV-2 genomes reveals epistasis between eight viral genes</h3>

<h3>对超过50,000个SARS-CoV-2基因组的全球分析揭示了八个病毒基因之间的上位性 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2012331117</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33203681/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2015': '9.423', '2016': '9.661', '2017': '9.504', '2018': '9.58', '2019': '9.412'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 12/08/2020</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genome-wide epistasis analysis is a powerful tool to infer gene interactions, which can guide drug and vaccine development and lead to deeper understanding of microbial pathogenesis. We have considered all complete severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) genomes deposited in the Global Initiative on Sharing All Influenza Data (GISAID) repository until four different cutoff dates, and used direct coupling analysis together with an assumption of quasi-linkage equilibrium to infer epistatic contributions to fitness from polymorphic loci. We find eight interactions, of which three are between pairs where one locus lies in gene ORF3a, both loci holding nonsynonymous mutations. We also find interactions between two loci in gene nsp13, both holding nonsynonymous mutations, and four interactions involving one locus holding a synonymous mutation. Altogether, we infer interactions between loci in viral genes ORF3a and nsp2, nsp12, and nsp6, between ORF8 and nsp4, and between loci in genes nsp2, nsp13, and nsp14. The paper opens the prospect to use prominent epistatically linked pairs as a starting point to search for combinatorial weaknesses of recombinant viral pathogens.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 全基因组上位性分析是推断基因相互作用的有力工具，可指导药物和疫苗的开发并导致对微生物发病机理的更深入了解。我们已经考虑了保存在全球所有流感数据倡议（GISAID）库中的所有完整的严重急性呼吸系统综合症冠状病毒2（SARS-CoV-2）基因组，直到四个不同的截止日期，并使用了直接耦合分析和一个准假设-连锁平衡可以推断出多态性位点对适应性的上位贡献。我们发现八种相互作用，其中三个在一对之间，其中一个基因座位于基因ORF3a中，两个基因座均持有非同义突变。我们还发现基因nsp13中的两个基因座之间的相互作用，两者都持有非同义突变，而涉及一个基因座的四个基因座则具有同义突变。总之，我们推断病毒基因ORF3a和nsp2，nsp12和nsp6中的基因座之间，ORF8和nsp4之间以及基因nsp2，nsp13和nsp14中基因座之间的相互作用。本文为利用突出的上位连锁对作为寻找重组病毒病原体的组合弱点的起点开辟了前景。</p>

</body>
</html>

