<!DOCTYPE html> <html lang = "en">
 <body>

<h2>Pubmed_ID: 33637963</h2>

<h3>Author Correction: Genome-wide meta-analysis, fine-mapping and integrative prioritization implicate new Alzheimer's disease risk genes</h3>

<h3>作者更正：全基因组的荟萃分析，精细映射和综合优先排序暗示了新的阿尔茨海默氏病风险基因 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41588-021-00822-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33637963/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature genetics</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/26/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 没有摘要</p>

<h2>Pubmed_ID: 33637760</h2>

<h3>A CSB-PAF1C axis restores processive transcription elongation after DNA damage repair</h3>

<h3>DNA损伤修复后，CSB-PAF1C轴可恢复进行性转录伸长 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21520-w</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33637760/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/26/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Bulky DNA lesions in transcribed strands block RNA polymerase II (RNAPII) elongation and induce a genome-wide transcriptional arrest. The transcription-coupled repair (TCR) pathway efficiently removes transcription-blocking DNA lesions, but how transcription is restored in the genome following DNA repair remains unresolved. Here, we find that the TCR-specific CSB protein loads the PAF1 complex (PAF1C) onto RNAPII in promoter-proximal regions in response to DNA damage. Although dispensable for TCR-mediated repair, PAF1C is essential for transcription recovery after UV irradiation. We find that PAF1C promotes RNAPII pause release in promoter-proximal regions and subsequently acts as a processivity factor that stimulates transcription elongation throughout genes. Our findings expose the molecular basis for a non-canonical PAF1C-dependent pathway that restores transcription throughout the human genome after genotoxic stress.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 转录链中的大块DNA损伤会阻止RNA聚合酶II（RNAPII）延长，并诱导全基因组转录停滞。转录偶联修复（TCR）途径可有效去除转录阻滞性DNA损伤，但如何修复DNA修复后基因组中的转录仍未解决。在这里，我们发现，TCR特异的CSB蛋白将启动子附近区域中的PAF1复合物（PAF1C）加载到RNAPII上，以响应DNA损伤。尽管在TCR介导的修复中不可或缺，但PAF1C对于紫外线照射后的转录恢复至关重要。我们发现PAF1C促进启动子近端区域中的RNAPII暂停释放，并随后作为一种刺激性因子，刺激整个基因的转录延伸。我们的发现揭示了非经典的PAF1C依赖性途径的分子基础，该途径在遗传毒性胁迫后可恢复整个人类基因组的转录。</p>

<h2>Pubmed_ID: 33637740</h2>

<h3>Ecology-guided prediction of cross-feeding interactions in the human gut microbiome</h3>

<h3>生态引导下人类肠道微生物组交叉进食相互作用的预测 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21586-6</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33637740/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/26/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Understanding a complex microbial ecosystem such as the human gut microbiome requires information about both microbial species and the metabolites they produce and secrete. These metabolites are exchanged via a large network of cross-feeding interactions, and are crucial for predicting the functional state of the microbiome. However, till date, we only have information for a part of this network, limited by experimental throughput. Here, we propose an ecology-based computational method, GutCP, using which we predict hundreds of new experimentally untested cross-feeding interactions in the human gut microbiome. GutCP utilizes a mechanistic model of the gut microbiome with the explicit exchange of metabolites and their effects on the growth of microbial species. To build GutCP, we combine metagenomic and metabolomic measurements from the gut microbiome with optimization techniques from machine learning. Close to 65% of the cross-feeding interactions predicted by GutCP are supported by evidence from genome annotations, which we provide for experimental testing. Our method has the potential to greatly improve existing models of the human gut microbiome, as well as our ability to predict the metabolic profile of the gut.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 了解复杂的微生物生态系统（例如人体肠道微生物组）需要有关微生物种类及其产生和分泌的代谢物的信息。这些代谢物通过交叉供养相互作用的大型网络进行交换，对于预测微生物组的功能状态至关重要。但是，到目前为止，我们仅拥有该网络一部分的信息，受实验吞吐量的限制。在这里，我们提出了一种基于生态的计算方法GutCP，通过该方法，我们可以预测人类肠道微生物组中数百种未经实验检验的交叉进食相互作用。GutCP利用肠道微生物组的机械模型进行代谢物的明确交换及其对微生物物种生长的影响。为了构建GutCP，我们将来自肠道微生物组的宏基因组学和代谢组学测量与来自机器学习的优化技术相结合。GutCP预测的接近65％的交叉供养相互作用得到了基因组注释的支持，我们为实验提供了证据。我们的方法具有极大地改善人类肠道微生物组现有模型的潜力，以及我们预测肠道代谢特征的能力。</p>

<h2>Pubmed_ID: 33637722</h2>

<h3>Pre-mitotic genome re-organisation bookends the B cell differentiation process</h3>

<h3>有丝分裂前基因组重组阻碍了B细胞分化过程 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21536-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33637722/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/26/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: During cellular differentiation chromosome conformation is intricately remodelled to support the lineage-specific transcriptional programs required for initiating and maintaining lineage identity. When these changes occur in relation to cell cycle, division and time in response to cellular activation and differentiation signals has yet to be explored, although it has been proposed to occur during DNA synthesis or after mitosis. Here, we elucidate the chromosome conformational changes in B lymphocytes as they differentiate and expand from a naive, quiescent state into antibody secreting plasma cells. We find gene-regulatory chromosome reorganization in late G1 phase before the first division, and that this configuration is remarkably stable as the cells massively and rapidly clonally expand. A second wave of conformational change occurs as cells terminally differentiate into plasma cells, coincident with increased time in G1 phase. These results provide further explanation for how lymphocyte fate is imprinted prior to the first division. They also suggest that chromosome reconfiguration occurs prior to DNA replication and mitosis, and is linked to a gene expression program that controls the differentiation process required for the generation of immunity.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在细胞分化期间，染色体构象被复杂地重塑以支持启动和维持谱系同一性所需的谱系特异性转录程序。当这些变化与细胞周期有关时，响应于细胞活化和分化信号的分裂和时间尚待探索，尽管已经提出在DNA合成过程中或有丝分裂后发生。在这里，我们阐明了B淋巴细胞的染色体构象变化，因为它们从幼稚的静止状态分化并扩展为分泌抗体的浆细胞。我们发现基因调控染色体重组在第一次分裂之前的G1期后期，并且随着细胞大量快速克隆扩展，这种构型非常稳定。随着细胞最终分化为浆细胞，发生第二次构象变化波，这与G1期的时间增加相吻合。这些结果为第一次分裂之前如何印记淋巴细胞命运提供了进一步的解释。他们还暗示，染色体重构发生在DNA复制和有丝分裂之前，并且与控制免疫力产生所需的分化过程的基因表达程序有关。</p>

<h2>Pubmed_ID: 33637709</h2>

<h3>Chromatin information content landscapes inform transcription factor and DNA interactions</h3>

<h3>染色质信息含量的变化告知转录因子和DNA相互作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21534-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33637709/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/26/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Interactions between transcription factors and chromatin are fundamental to genome organization and regulation and, ultimately, cell state. Here, we use information theory to measure signatures of organized chromatin resulting from transcription factor-chromatin interactions encoded in the patterns of the accessible genome, which we term chromatin information enrichment (CIE). We calculate CIE for hundreds of transcription factor motifs across human samples and identify two classes: low and high CIE. The 10-20% of common and tissue-specific high CIE transcription factor motifs, associate with higher protein-DNA residence time, including different binding site subclasses of the same transcription factor, increased nucleosome phasing, specific protein domains, and the genetic control of both chromatin accessibility and gene expression. These results show that variations in the information encoded in chromatin architecture reflect functional biological variation, with implications for cell state dynamics and memory.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 转录因子和染色质之间的相互作用对于基因组的组织和调控乃至细胞状态至关重要。在这里，我们使用信息论来测量由可访问基因组模式中编码的转录因子-染色质相互作用产生的有组织染色质的特征，我们将其称为染色质信息富集（CIE）。我们计算人类样品中数百个转录因子基序的CIE，并确定两类：低CIE和高CIE。常见的和组织特异性的高CIE转录因子基序的10-20％与更高的蛋白质-DNA停留时间相关联，包括相同转录因子的不同结合位点亚类，增加的核小体定相，特定的蛋白质结构域和遗传控制染色质可及性和基因表达。这些结果表明，以染色质结构编码的信息中的变异反映了功能性生物学变异，对细胞状态动态和记忆具有影响。</p>

<h2>Pubmed_ID: 33637701</h2>

<h3>PlasmidHawk improves lab of origin prediction of engineered plasmids using sequence alignment</h3>

<h3>PlasmidHawk使用序列比对提高了工程化质粒起源预测的实验室 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21180-w</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33637701/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/26/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: With advances in synthetic biology and genome engineering comes a heightened awareness of potential misuse related to biosafety concerns. A recent study employed machine learning to identify the lab-of-origin of DNA sequences to help mitigate some of these concerns. Despite their promising results, this deep learning based approach had limited accuracy, was computationally expensive to train, and wasn't able to provide the precise features that were used in its predictions. To address these shortcomings, we developed PlasmidHawk for lab-of-origin prediction. Compared to a machine learning approach, PlasmidHawk has higher prediction accuracy; PlasmidHawk can successfully predict unknown sequences' depositing labs 76% of the time and 85% of the time the correct lab is in the top 10 candidates. In addition, PlasmidHawk can precisely single out the signature sub-sequences that are responsible for the lab-of-origin detection. In summary, PlasmidHawk represents an explainable and accurate tool for lab-of-origin prediction of synthetic plasmid sequences. PlasmidHawk is available at https://gitlab.com/treangenlab/plasmidhawk.git .</p>

<p style="line-height:1.8;margin:20px"> 摘要: 随着合成生物学和基因组工程学的进步，人们对与生物安全问题相关的潜在滥用的认识也日益提高。最近的一项研究使用机器学习来识别DNA序列的起源实验室，以帮助减轻其中的一些担忧。尽管取得了令人鼓舞的结果，但这种基于深度学习的方法准确性有限，训练时在计算上昂贵，并且无法提供用于其预测的精确功能。为了解决这些缺点，我们开发了PlasmidHawk用于原产地预测。与机器学习方法相比，PlasmidHawk具有更高的预测精度；PlasmidHawk可以成功地预测未知序列，在正确的实验室中排名前10位的时间有76％，在正确的实验室中占85％的时间。另外，PlasmidHawk可以精确地挑选出负责起源实验室检测的特征子序列。总之，PlasmidHawk代表了一种可解释且准确的工具，可用于实验室对合成质粒序列的起源进行预测。可以在https://gitlab.com/treangenlab/plasmidhawk.git上找到PlasmidHawk。</p>

<h2>Pubmed_ID: 33636127</h2>

<h3>In vivo structural characterization of the SARS-CoV-2 RNA genome identifies host proteins vulnerable to repurposed drugs</h3>

<h3>SARS-CoV-2 RNA基因组的体内结构鉴定可鉴定易受重用药物影响的宿主蛋白 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2021.02.008</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33636127/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing coronavirus disease 2019 (COVID-19) pandemic. Understanding of the RNA virus and its interactions with host proteins could improve therapeutic interventions for COVID-19. By using icSHAPE, we determined the structural landscape of SARS-CoV-2 RNA in infected human cells and from refolded RNAs, as well as the regulatory untranslated regions of SARS-CoV-2 and six other coronaviruses. We validated several structural elements predicted in silico and discovered structural features that affect the translation and abundance of subgenomic viral RNAs in cells. The structural data informed a deep-learning tool to predict 42 host proteins that bind to SARS-CoV-2 RNA. Strikingly, antisense oligonucleotides targeting the structural elements and FDA-approved drugs inhibiting the SARS-CoV-2 RNA binding proteins dramatically reduced SARS-CoV-2 infection in cells derived from human liver and lung tumors. Our findings thus shed light on coronavirus and reveal multiple candidate therapeutics for COVID-19 treatment.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 严重的急性呼吸系统综合症冠状病毒2（SARS-CoV-2）是导致正在进行的2019年冠状病毒病（COVID-19）大流行的原因。了解RNA病毒及其与宿主蛋白的相互作用可以改善对COVID-19的治疗干预。通过使用icSHAPE，我们确定了SARS-CoV-2RNA在受感染的人细胞中以及从重折叠的RNA以及SARS-CoV-2和其他六种冠状病毒的调控性非翻译区域中的结构态势。我们验证了计算机预测的几个结构元件，并发现了影响细胞中亚基因组病毒RNA的翻译和丰度的结构特征。结构数据提供了一种深度学习工具，可预测与SARS-CoV-2RNA结合的42种宿主蛋白​​。引人注目的是，靶向结构元件的反义寡核苷酸和FDA批准的抑制SARS-CoV-2RNA结合蛋白的药物大大减少了源自人肝和肺肿瘤的细胞中SARS-CoV-2的感染。因此，我们的发现揭示了冠状病毒，并揭示了COVID-19治疗的多种候选疗法。</p>

<h2>Pubmed_ID: 33631421</h2>

<h3>Nuclear Phylotranscriptomics/Phylogenomics Support Numerous Polyploidization Events and Hypotheses for the Evolution of Rhizobial Nitrogen-Fixing Symbiosis in Fabaceae</h3>

<h3>核植物转录组学/拟基因组学支持许多多倍体化事件和假说，为豆科植物的根瘤菌固氮共生进化。 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.molp.2021.02.006</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33631421/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Molecular plant</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/22/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Fabaceae, with 765 genera and ∼19,500 species, are the third-largest angiosperm family and important both economically and ecologically, with global crops intensively studied in part for their nitrogen-fixing ability. However, resolution of Fabaceae phylogeny and divergence times at intrasubfamilial levels have been elusive, precluding a reconstruction of the evolutionary history of symbiotic nitrogen fixation in Fabaceae. Here, we report a highly resolved phylogeny using >1500 nuclear genes from newly sequenced transcriptomes and genomes of 391 species along with other datasets for a total of 463 legumes spanning all six subfamilies and 333 of 765 genera. The subfamilies are maximally supported as monophyletic, with the clade comprising subfamilies Cercidoideae and Detarioideae being the sister of remaining legumes, and Duparquetioideae and Dialioideae being successive sisters to the clade of Papilionoideae and Caesalpinioideae. Molecular clock estimation revealed early radiation of subfamilies near the K/Pg boundary marked by mass extinction and subsequent divergences of most tribe-level clades within ∼15 million years. Phylogenomic analyses of thousands of gene families support 28 proposed WGD/WGT events across Fabaceae, including those at the ancestors of Fabaceae and five of the subfamilies, with support from further analyses for the Fabaceae ancestral polyploidy. The evolution of rhizobial nitrogen-fixing nodulation in Fabaceae was examined through ancestral character reconstruction and phylogenetic analyses of related gene families, providing support for hypotheses of one or two switch(es) to rhizobial nodulation followed by multiple losses. These results form a foundation for further morphological and functional evolutionary analyses across Fabaceae.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 豆科（Fabaceae）有765个属，约有19,500种，是被子植物的第三大家族，在经济和生态上都很重要，对全球作物的固氮能力进行了深入的研究。但是，在亚家族内水平上解决豆科植物系统发育和分歧时间的方法还很困难，这妨碍了豆科植物中共生固氮进化历史的重建。在这里，我们报告了一个高度解析的系统发育史，使用了来自391个物种的新测序转录组和基因组的1500多个核基因，以及其他数据集，涵盖了全部六个亚科的463个豆科和765个属的333个。该亚科得到最大的单系支持，其进化枝包括小豆科的亚科（Cercidoideae）和De科（Detarioideae），其余的豆科为姊妹科（Duparquetioideae）和迪亚科（Dialioideae），是Pa蝶科（Papilionoideae）和Ca科（Caesalpinioideae）的接班姐妹。分子钟估计表明，在K/Pg边界附近的亚科有早期辐射，其特征是生物大灭绝，随后在大约1500万年之内大多数部落级别进化枝都出现了分歧。对成千上万个基因家族的系统生物学分析支持了整个Fabaceae的28个WGD/WGT事件，包括Fabaceae祖先和五个亚科的WGD/WGT事件，并获得了对Fabaceae祖先多倍体的进一步分析的支持。通过祖先特征重建和相关基因家族的系统发育分析检查了豆科植物中的根瘤菌固氮根瘤的演变，为一到两个切换到根瘤菌根瘤继而遭受多次损失的假说提供了支持。这些结果构成了整个豆科的进一步形态和功能进化分析的基础。</p>

<h2>Pubmed_ID: 33631420</h2>

<h3>High-Efficiency and Multiplex Adenine Base Editing in Plants Using New TadA Variants</h3>

<h3>使用新的TadA变体对植物进行高效和多重腺嘌呤碱基编辑 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.molp.2021.02.007</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33631420/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Molecular plant</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/22/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Recently reported adenine base editors (ABEs) exhibit powerful potentials in targeted gene correction as well as developing gain-of-function mutants and novel germplasms for both gene function study and crop breeding. However, editing efficiency varies significantly among different target sites. Here we investigated the activities of three evolved E. coli adenosine deaminase TadA variants (TadA8e, TadA8.17, and TadA8.20) side-by-side in transgenic rice. We found that TadA8e outperforms TadA8.17 and TadA8.20, and induces efficient A-to-G conversion at all tested sites in the rice genome, including those uneditable for ABE7.10 in our previous experiments. Furthermore, V82S/Q154R mutations were incorporated into TadA8e, resulting in a new variant that we named TadA9. Our data shows that TadA9 is broadly compatible with CRISPR/SpCas9, CRISPR/SpCas9-NG, CRISPR/SpRY as well as CRISPR/ScCas9 nickase systems, achieving comparable or enhanced editing in a larger editing window at diverse PAM sites as compared to TadA8e. Finally, TadA9 was employed to simultaneously install novel SNPs in four endogenous herbicide-target genes in the commercial rice cultivar Nangeng 46 for the potential field application in weed control. In conclusion, we successfully generate a series of novel ABEs which can efficiently edit adenosines in the rice genome to a great extent, and our finding implies the promising application of TadA9 as well as TadA8e in other plant base editor design and crop molecular breeding.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 最近报道的腺嘌呤碱基编辑器（ABE）在靶向基因校正以及为基因功能研究和作物育种开发功能获得性突变体和新型种质方面显示出强大的潜力。但是，不同目标站点之间的编辑效率差异很大。在这里，我们并排研究了三个进化的大肠杆菌腺苷脱氨酶TadA变体（TadA8e，TadA8.17和TadA8.20）在转基因水稻中的活性。我们发现TadA8e优于TadA8.17和TadA8.20，并在水稻基因组的所有测试位点（包括在我们先前的实验中无法编辑ABE7.10的位点）诱导有效的A到G转换。此外，V82S/Q154R突变被整合到TadA8e中，产生了一个新的变体，我们将其命名为TadA9。我们的数据显示，TadA9与CRISPR/SpCas9，CRISPR/SpCas9-NG，CRISPR/SpRY以及CRISPR/ScCas9切口酶系统广泛兼容，与TadA8e相比，在不同的PAM站点的更大编辑窗口中实现了可比或增强的编辑。最后，使用TadA9同时在商业水稻品种Nangeng46的四个内源除草剂靶基因中同时安装新的SNP，以在田间控制杂草中潜在应用。总之，我们成功地产生了一系列可以在水稻基因组中高效编辑腺苷的新型ABE，我们的发现表明TadA9和TadA8e在其他植物基础编辑器设计和作物分子育种中的应用前景广阔。</p>

<h2>Pubmed_ID: 33629439</h2>

<h3>Identification of specificity-defining amino acids of the wheat immune receptor Pm2 and powdery mildew effector AvrPm2</h3>

<h3>鉴定小麦免疫受体Pm2和白粉病效应因子AvrPm2的特异性定义氨基酸 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15214</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33629439/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/24/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Plant nucleotide-binding, leucine-rich repeat receptors (NLRs) act as intracellular sensors for pathogen- derived effector proteins and trigger an immune response, frequently resulting in hypersensitive cell death response (HR) of the infected host cell. The wheat (Triticum aestivum) NLR Pm2 confers resistance against the fungal pathogen Blumeria graminis f. sp. tritici (Bgt) if the isolate contains the specific RNase-like effector AvrPm2. We identified and isolated seven new Pm2 alleles (Pm2e - i) in the wheat D-genome ancestor Aegilops tauschii and two new, natural AvrPm2 haplotypes from Bgt. Upon transient co-expression in Nicotiana benthamiana, we observed a variant-specific HR of the Pm2 variants Pm2a and Pm2i towards AvrPm2 or its homolog from the AvrPm2 effector family, BgtE-5843, respectively. Through the introduction of naturally occurring non-synonymous SNPs and structure-guided mutations, we identified single amino acids, in both the wheat NLR Pm2 and the fungal effector proteins AvrPm2 and BgtE5843 responsible for the variant-specific HR of the Pm2 variants. Exchanging these amino acids led to modified HR of the Pm2-AvrPm2 interaction and allowed the identification of the effector head epitope, a 20 amino acid long unit of AvrPm2 involved in HR. Swapping of the AvrPm2 head epitope to the non HR-triggering AvrPm2 family member BgtE-5846 led to gain of HR by Pm2a. Our study presents a molecular approach to identify crucial effector surface structures involved in HR and demonstrates that natural and induced diversity in an immune receptor and its corresponding effectors can provide the basis for understanding and modifying NLR - effector specificity.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 植物核苷酸结合的富含亮氨酸的重复受体（NLR）充当病原体衍生效应蛋白的细胞内传感器，并触发免疫反应，经常导致被感染宿主细胞的超敏细胞死亡反应（HR）。小麦（Triticumaestivum）NLRPm2赋予了对真菌病原体Blumeriagraminisf的抗性。sp。如果分离株含有特定的RNase样效应子AvrPm2，则为Tritici（Bgt）。我们在小麦D基因组祖先牛肝菌（Aegilopstauschii）中鉴定并分离了七个新的Pm2等位基因（Pm2e-i），并从Bgt中分离了两个新的天然AvrPm2单倍型。在烟草中的瞬时共表达后，我们观察到Pm2变体Pm2a和Pm2i对AvrPm2或其来自AvrPm2效应子家族BgtE-5843的同系物的变体特异性HR。通过引入天然存在的非同义SNP和结构指导的突变，我们在小麦NLRPm2和负责Pm2变体的变体特异性HR的真菌效应蛋白AvrPm2和BgtE5843中鉴定出单个氨基酸。交换这些氨基酸导致修饰的Pm2-AvrPm2相互作用的HR，并允许识别效应子头部抗原决定簇，这是一个20个氨基酸长的AvrPm2单元，参与了HR。将AvrPm2头部表位交换至非触发HR的AvrPm2家族成员BgtE-5846导致Pm2a获得了HR。我们的研究提出了一种分子方法来鉴定参与HR的关键效应子表面结构，并证明免疫受体及其相应效应子的天然和诱导多样性可以为理解和修饰NLR效应子特异性提供基础。</p>

<h2>Pubmed_ID: 33627673</h2>

<h3>Genome-wide meta-analysis identifies 127 open-angle glaucoma loci with consistent effect across ancestries</h3>

<h3>全基因组荟萃分析确定了127个开角型青光眼基因座，在祖先之间具有一致的作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20851-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33627673/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/24/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Primary open-angle glaucoma (POAG), is a heritable common cause of blindness world-wide. To identify risk loci, we conduct a large multi-ethnic meta-analysis of genome-wide association studies on a total of 34,179 cases and 349,321 controls, identifying 44 previously unreported risk loci and confirming 83 loci that were previously known. The majority of loci have broadly consistent effects across European, Asian and African ancestries. Cross-ancestry data improve fine-mapping of causal variants for several loci. Integration of multiple lines of genetic evidence support the functional relevance of the identified POAG risk loci and highlight potential contributions of several genes to POAG pathogenesis, including SVEP1, RERE, VCAM1, ZNF638, CLIC5, SLC2A12, YAP1, MXRA5, and SMAD6. Several drug compounds targeting POAG risk genes may be potential glaucoma therapeutic candidates.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 原发性开角型青光眼（POAG）是世界范围内导致失明的可遗传的常见原因。为了识别风险基因座，我们对全基因组关联研究进行了大型多族裔荟萃分析，共研究了34,179例病例和349,321例对照，确定了44个以前未报告的风险基因座，并确认了83个先前已知的基因座。大多数基因座在欧洲，亚洲和非洲的祖先中具有广泛一致的影响。跨谱系数据改善了多个基因座的因果变异的精细映射。多系遗传证据的整合支持已鉴定的POAG风险基因座的功能相关性，并突出了几种基因对POAG发病机制的潜在贡献，包括SVEP1，RERE，VCAM1，ZNF638，CLIC5，SLC2A12，YAP1，MXRA5和SMAD6。靶向POAG风险基因的几种药物可能是潜在的青光眼治疗候选药物。</p>

<h2>Pubmed_ID: 33627664</h2>

<h3>Alternative lengthening of telomeres in childhood neuroblastoma from genome to proteome</h3>

<h3>从基因组到蛋白质组的儿童神经母细胞瘤端粒的替代性延长 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21247-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33627664/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/24/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Telomere maintenance by telomerase activation or alternative lengthening of telomeres (ALT) is a major determinant of poor outcome in neuroblastoma. Here, we screen for ALT in primary and relapsed neuroblastomas (n = 760) and characterize its features using multi-omics profiling. ALT-positive tumors are molecularly distinct from other neuroblastoma subtypes and enriched in a population-based clinical sequencing study cohort for relapsed cases. They display reduced ATRX/DAXX complex abundance, due to either ATRX mutations (55%) or low protein expression. The heterochromatic histone mark H3K9me3 recognized by ATRX is enriched at the telomeres of ALT-positive tumors. Notably, we find a high frequency of telomeric repeat loci with a neuroblastoma ALT-specific hotspot on chr1q42.2 and loss of the adjacent chromosomal segment forming a neo-telomere. ALT-positive neuroblastomas proliferate slowly, which is reflected by a protracted clinical course of disease. Nevertheless, children with an ALT-positive neuroblastoma have dismal outcome.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 通过端粒酶激活或端粒选择性延长（ALT）来维持端粒是神经母细胞瘤预后不良的主要决定因素。在这里，我们筛查了原发性和复发性神经母细胞瘤（n=760）中的ALT，并使用多组学分析来表征其特征。ALT阳性肿瘤在分子上不同于其他神经母细胞瘤亚型，并且在针对复发病例的基于人群的临床测序研究队列中得到了丰富。由于ATRX突变（55％）或蛋白表达水平低，它们显示出ATRX/DAXX复合物丰度降低。ATRX识别的异色组蛋白标记H3K9me3富集于ALT阳性肿瘤的端粒。值得注意的是，我们在chr1q42.2上发现了端粒重复位点的频率很高，并带有一个神经母细胞瘤ALT特异性热点，并且相邻染色体部分的丢失形成了一个新端粒。ALT阳性的神经母细胞瘤增殖缓慢，这反映在疾病的长期临床过程中。尽管如此，患有ALT阳性神经母细胞瘤的儿童的结局却令人沮丧。</p>

<h2>Pubmed_ID: 33627659</h2>

<h3>Revealing the role of the human blood plasma proteome in obesity using genetic drivers</h3>

<h3>使用遗传驱动程序揭示人血浆蛋白质组在肥胖症中的作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21542-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33627659/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/24/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Blood circulating proteins are confounded readouts of the biological processes that occur in different tissues and organs. Many proteins have been linked to complex disorders and are also under substantial genetic control. Here, we investigate the associations between over 1000 blood circulating proteins and body mass index (BMI) in three studies including over 4600 participants. We show that BMI is associated with widespread changes in the plasma proteome. We observe 152 replicated protein associations with BMI. 24 proteins also associate with a genome-wide polygenic score (GPS) for BMI. These proteins are involved in lipid metabolism and inflammatory pathways impacting clinically relevant pathways of adiposity. Mendelian randomization suggests a bi-directional causal relationship of BMI with LEPR/LEP, IGFBP1, and WFIKKN2, a protein-to-BMI relationship for AGER, DPT, and CTSA, and a BMI-to-protein relationship for another 21 proteins. Combined with animal model and tissue-specific gene expression data, our findings suggest potential therapeutic targets further elucidating the role of these proteins in obesity associated pathologies.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 血液循环蛋白是在不同组织和器官中发生的生物过程的混杂读数。许多蛋白质已经与复杂的疾病联系在一起，并且也受到了实质性的遗传控制。在这里，我们在包括4600多个参与者在内的三项研究中研究了1000多种血液循环蛋白与体重指数（BMI）之间的关联。我们表明，体重指数与血浆蛋白质组的广泛变化有关。我们观察到152个复制的蛋白质与BMI的关联。24种蛋白质还与BMI的全基因组多基因评分（GPS）相关。这些蛋白质参与脂质代谢和炎症途径，影响肥胖的临床相关途径。孟德尔随机化表明BMI与LEPR/LEP，IGFBP1和WFIKKN2具有双向因果关系，AGER，DPT和CTSA的蛋白质与BMI之间的关系，以及另外21种蛋白质的BMI与蛋白质之间的关系。结合动物模型和组织特异性基因表达数据，我们的发现表明潜在的治疗靶标进一步阐明了这些蛋白在肥胖相关疾病中的作用。</p>

<h2>Pubmed_ID: 33627650</h2>

<h3>Strand-specific single-cell methylomics reveals distinct modes of DNA demethylation dynamics during early mammalian development</h3>

<h3>特定于链的单细胞甲基组学揭示了哺乳动物早期发育过程中DNA去甲基化动力学的不同模式 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21532-6</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33627650/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/24/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: DNA methylation (5mC) is central to cellular identity. The global erasure of 5mC from the parental genomes during preimplantation mammalian development is critical to reset the methylome of gametes to the cells in the blastocyst. While active and passive modes of demethylation have both been suggested to play a role in this process, the relative contribution of these two mechanisms to 5mC erasure remains unclear. Here, we report a single-cell method (scMspJI-seq) that enables strand-specific quantification of 5mC, allowing us to systematically probe the dynamics of global demethylation. When applied to mouse embryonic stem cells, we identified substantial cell-to-cell strand-specific 5mC heterogeneity, with a small group of cells displaying asymmetric levels of 5mCpG between the two DNA strands of a chromosome suggesting loss of maintenance methylation. Next, in preimplantation mouse embryos, we discovered that methylation maintenance is active till the 16-cell stage followed by passive demethylation in a fraction of cells within the early blastocyst at the 32-cell stage of development. Finally, human preimplantation embryos qualitatively show temporally delayed yet similar demethylation dynamics as mouse embryos. Collectively, these results demonstrate that scMspJI-seq is a sensitive and cost-effective method to map the strand-specific genome-wide patterns of 5mC in single cells.</p>

<p style="line-height:1.8;margin:20px"> 摘要: DNA甲基化（5mC）对细胞身份至关重要。植入前哺乳动物发育过程中从亲本基因组中全局清除5mC对于将配子的甲基化组重置为胚泡中的细胞至关重要。虽然主动和被动去甲基化模式都已被建议在此过程中发挥作用，但这两种机制对5mC擦除的相对贡献仍不清楚。在这里，我们报告了一种单细胞方法（scMspJI-seq），该方法可实现5mC的链特异性定量，从而使我们能够系统地探究整体去甲基化的动力学。当应用于小鼠胚胎干细胞时，我们发现了实质性的细胞间链特异性5mC异质性，一小组细胞在染色体的两条DNA链之间显示5mCpG的不对称水平，表明维持甲基化的损失。接下来，在植入前的小鼠胚胎中，我们发现甲基化维持有效直至16细胞阶段，随后在胚泡早期32细胞发育阶段的一部分细胞中进行被动去甲基化。最后，人类植入前的胚胎定性地显示出时间延迟但与小鼠胚胎相似的去甲基化动力学。总的来说，这些结果表明，scMspJI-seq是一种灵敏且经济高效的方法，可在单个细胞中绘制5mC的链特异性全基因组模式。</p>

<h2>Pubmed_ID: 33627649</h2>

<h3>Accelerating target deconvolution for therapeutic antibody candidates using highly parallelized genome editing</h3>

<h3>使用高度平行的基因组编辑来加速治疗性抗体候选物的靶点解卷积 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21518-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33627649/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/24/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Therapeutic antibodies are transforming the treatment of cancer and autoimmune diseases. Today, a key challenge is finding antibodies against new targets. Phenotypic discovery promises to achieve this by enabling discovery of antibodies with therapeutic potential without specifying the molecular target a priori. Yet, deconvoluting the targets of phenotypically discovered antibodies remains a bottleneck; efficient deconvolution methods are needed for phenotypic discovery to reach its full potential. Here, we report a comprehensive investigation of a target deconvolution approach based on pooled CRISPR/Cas9. Applying this approach within three real-world phenotypic discovery programs, we rapidly deconvolute the targets of 38 of 39 test antibodies (97%), a success rate far higher than with existing approaches. Moreover, the approach scales well, requires much less work, and robustly identifies antibodies against the major histocompatibility complex. Our data establish CRISPR/Cas9 as a highly efficient target deconvolution approach, with immediate implications for the development of antibody-based drugs.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 治疗性抗体正在改变癌症和自身免疫性疾病的治疗方法。今天，一个关键的挑战是找到针对新靶标的抗体。表型发现有望通过实现具有治疗潜力的抗体的发现而无需事先指定分子靶标来实现这一目标。然而，对表型发现的抗体的靶标进行去卷积仍然是一个瓶颈。表型发现需要有效的反卷积方法才能发挥其全部潜力。在这里，我们报告了基于汇集的CRISPR/Cas9的目标反卷积方法的全面研究。将这种方法应用于三个现实世界的表型发现程序中，我们迅速对39种测试抗体中的38种（97％）的靶标进行了卷积，成功率远远高于现有方法。此外，该方法可扩展性好，所需工作量少并且可以可靠地识别针对主要组织相容性复合物的抗体。我们的数据将CRISPR/Cas9确立为一种高效的目标反卷积方法，对基于抗体的药物的开发具有直接的意义。</p>

<h2>Pubmed_ID: 33624402</h2>

<h3>Loquat (Eriobotrya japonica (Thunb.) Lindl) population genomics suggests a two-staged domestication and identifies genes showing convergence/parallel selective sweeps with apple or peach</h3>

<h3>qua（Eriobotrya japonica（Thunb。）Lindl）种群基因组学提出了两阶段驯化并鉴定了与苹果或桃子表现出趋同/平行选择性扫描的基因 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15209</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33624402/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Crop domestication and evolution represent key fields of plant and genetics research. Here, we re-sequenced and analyzed whole genome data from 51 wild accessions and 53 representative cultivars of E. japonica, an important semi-subtropical fruit crop. Population genomics analysis suggested that modern cultivated E. japonica experienced a two-staged domestication fitting the 'marginality model', being initially domesticated in West-northern Hubei province from a mono-phylogenetic wild progenitor, then refined mainly in Jiangsu, Zhejiang and Fujian provinces of China. Cultivated E. japonica has experienced little reduction in genome-wide nucleotide polymorphism compared to wild forms. Genes responsible for sugar biosynthesis were enriched in regions harboring putative selective sweeps. An approach based on co-clustering into gene families and evaluating chromosome colinearity of orthologous and paralogous genes was used to identify convergent/parallel selective sweeps among different crops. Specifically, more than one hundred of orthologs and paralogs undergoing selective sweeps were identified between loquat, apple and peach, among which 14 encoded 'UDP glycosyltransferase 1'. In sum, the study not only provided valuable information for breeding of E. japonica, but also enriched knowledge of crop domestication.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 作物的驯化和进化代表了植物和遗传学研究的关键领域。在这里，我们对来自重要的半亚热带水果作物日本粳稻的51个野生种和53个代表性品种的全基因组数据进行了重新测序和分析。种群基因组学分析表明，现代栽培的日本粳稻经历了两阶段驯化，符合边际模型，最初是从湖北西北部由单一系统发育的野生祖先驯化而来，然后主要在中国江苏，浙江和福建省进行改良。与野生形式相比，栽培的粳稻在全基因组核苷酸多态性方面几乎没有减少。负责糖生物合成的基因富集在具有推定选择性扫描的区域。一种基于共聚为基因家族并评估直系同源和旁系同源基因的染色体共线性的方法来鉴定不同作物之间的趋同/平行选择性扫描。具体而言，在lo，苹果和桃子之间鉴定出一百多个直向同源物和旁系同源物，其中有14个编码的UDP糖基转移酶1。总之，该研究不仅为日本粳稻的育种提供了有价值的信息，而且丰富了丰富的信息。作物驯化知识。</p>

<h2>Pubmed_ID: 33623110</h2>

<h3>3D genome encoded by LINE and SINE repeats</h3>

<h3>LINE和SINE重复序列编码的3D基因组 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41422-021-00485-x</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33623110/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell research</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 没有摘要</p>

<h2>Pubmed_ID: 33623049</h2>

<h3>A synergetic effect of BARD1 mutations on tumorigenesis</h3>

<h3>BARD1突变对肿瘤发生的协同作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21519-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33623049/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: To date, a large number of mutations have been screened from breast and ovarian cancer patients. However, most of them are classified into benign or unidentified alterations due to their undetectable phenotypes. Whether and how they could cause tumors remains unknown, and this significantly limits diagnosis and therapy. Here, in a study of a family with hereditary breast and ovarian cancer, we find that two BARD1 mutations, P24S and R378S, simultaneously exist in cis in surviving cancer patients. Neither of the single mutations causes a functional change, but together they synergetically impair the DNA damage response and lead to tumors in vitro and in vivo. Thus, our report not only demonstrates that BARD1 defects account for tumorigenesis but also uncovers the potential risk of synergetic effects between the large number of cis mutations in individual genes in the human genome.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 迄今为止，已经从乳腺癌和卵巢癌患者中筛选出大量突变。然而，由于它们无法检测到的表型，它们中的大多数被分类为良性或无法识别的改变。它们是否以及如何引起肿瘤仍然未知，这极大地限制了诊断和治疗。在一项对患有遗传性乳腺癌和卵巢癌的家庭的研究中，我们发现存活的癌症患者的顺式同时存在两个BARD1突变P24S和R378S。单个突变均不会引起功能改变，但它们共同协同损害DNA损伤反应并在体外和体内导致肿瘤。因此，我们的报告不仅证明了BARD1缺陷是肿瘤发生的原因，而且还揭示了人类基因组单个基因中大量顺式突变之间协同作用的潜在风险。</p>

<h2>Pubmed_ID: 33623026</h2>

<h3>Genome assembly and population genomic analysis provide insights into the evolution of modern sweet corn</h3>

<h3>基因组装配和种群基因组分析为现代甜玉米的进化提供了见识 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21380-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33623026/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Sweet corn is one of the most important vegetables in the United States and Canada. Here, we present a de novo assembly of a sweet corn inbred line Ia453 with the mutated shrunken2-reference allele (Ia453-sh2). This mutation accumulates more sugar and is present in most commercial hybrids developed for the processing and fresh markets. The ten pseudochromosomes cover 92% of the total assembly and 99% of the estimated genome size, with a scaffold N50 of 222.2 Mb. This reference genome completely assembles the large structural variation that created the mutant sh2-R allele. Furthermore, comparative genomics analysis with six field corn genomes highlights differences in single-nucleotide polymorphisms, structural variations, and transposon composition. Phylogenetic analysis of 5,381 diverse maize and teosinte accessions reveals genetic relationships between sweet corn and other types of maize. Our results show evidence for a common origin in northern Mexico for modern sweet corn in the U.S. Finally, population genomic analysis identifies regions of the genome under selection and candidate genes associated with sweet corn traits, such as early flowering, endosperm composition, plant and tassel architecture, and kernel row number. Our study provides a high-quality reference-genome sequence to facilitate comparative genomics, functional studies, and genomic-assisted breeding for sweet corn.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 甜玉米是美国和加拿大最重要的蔬菜之一。在这里，我们提出了甜玉米自交系Ia453与突变的shrunken2参考等位基因（Ia453-sh2）的从头组装。这种突变积累了更多的糖分，并且存在于为加工和新鲜市场开发的大多数商业杂交种中。十个假染色体占总装配体的92％和估计的基因组大小的99％，支架的N50为222.2Mb。该参考基因组完全组装了产生突变型sh2-R等位基因的巨大结构变异。此外，用六个田间玉米基因组进行的比较基因组分析突出显示了单核苷酸多态性，结构变异和转座子组成的差异。对5,381种不同玉米和端粒种的种系进行系统发育分析，揭示了甜玉米与其他类型玉米之间的遗传关系。我们的研究结果证明了美国北部现代甜玉米在墨西哥北部的共同起源。最后，种群基因组分析确定了正在选择的基因组区域以及与甜玉米性状相关的候选基因，例如早开花，胚乳组成，植物和穗体系结构和内核行号。我们的研究提供了高质量的参考基因组序列，以促进甜玉米的比较基因组学，功能研究和基因组辅助育种。</p>

<h2>Pubmed_ID: 33623021</h2>

<h3>Single-cell multiomics sequencing reveals the functional regulatory landscape of early embryos</h3>

<h3>单细胞多组学测序揭示了早期胚胎的功能调控格局 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21409-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33623021/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Extensive epigenetic reprogramming occurs during preimplantation embryo development. However, it remains largely unclear how the drastic epigenetic reprogramming contributes to transcriptional regulatory network during this period. Here, we develop a single-cell multiomics sequencing technology (scNOMeRe-seq) that enables profiling of genome-wide chromatin accessibility, DNA methylation and RNA expression in the same individual cell. We apply this method to depict a single-cell multiomics map of mouse preimplantation development. We find that genome-wide DNA methylation remodeling facilitates the reconstruction of genetic lineages in early embryos. Further, we construct a zygotic genome activation (ZGA)-associated regulatory network and reveal coordination among multiple epigenetic layers, transcription factors and repeat elements that instruct proper ZGA. Cell fates associated cis-regulatory elements are activated stepwise in post-ZGA stages. Trophectoderm (TE)-specific transcription factors play dual roles in promoting the TE program while repressing the inner cell mass (ICM) program during the ICM/TE separation.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 广泛的表观遗传重编程发生在植入前的胚胎发育过程中。然而，目前还不清楚在此期间剧烈的表观遗传重编程如何对转录调控网络作出贡献。在这里，我们开发了一种单细胞多组学测序技术（scNOMeRe-seq），该技术可在同一细胞内对全基因组染色质可及性，DNA甲基化和RNA表达进行分析。我们应用这种方法来描绘小鼠植入前发育的单细胞多组学图。我们发现全基因组DNA甲基化重塑促进了早期胚胎遗传谱系的重建。此外，我们构建了一个与合子基因组激活（ZGA）相关的调节网络，并揭示了多个表观遗传层，转录因子和指示正确ZGA的重复元件之间的协调。在ZGA后阶段，逐步激活与细胞命运相关的顺式调控元件。滋养外胚层（TE）特异的转录因子在促进TE程序同时在ICM/TE分离过程中抑制内部细胞团（ICM）程序时起着双重作用。</p>

<h2>Pubmed_ID: 33623009</h2>

<h3>Whole genome sequencing in the Middle Eastern Qatari population identifies genetic associations with 45 clinically relevant traits</h3>

<h3>中东卡塔尔人群的全基因组测序可确定具有45种临床相关性状的遗传关联 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21381-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33623009/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Clinical laboratory tests play a pivotal role in medical decision making, but little is known about their genetic variability between populations. We report a genome-wide association study with 45 clinically relevant traits from the population of Qatar using a whole genome sequencing approach in a discovery set of 6218 individuals and replication in 7768 subjects. Trait heritability is more similar between Qatari and European populations (r = 0.81) than with Africans (r = 0.44). We identify 281 distinct variant-trait-associations at genome wide significance that replicate known associations. Allele frequencies for replicated loci show higher correlations with European (r = 0.94) than with African (r = 0.85) or Japanese (r = 0.80) populations. We find differences in linkage disequilibrium patterns and in effect sizes of the replicated loci compared to previous reports. We also report 17 novel and Qatari-predominate signals providing insights into the biological pathways regulating these traits. We observe that European-derived polygenic scores (PGS) have reduced predictive performance in the Qatari population which could have implications for the translation of PGS between populations and their future application in precision medicine.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 临床实验室测试在医疗决策中起着举足轻重的作用，但是人们对其种群之间的遗传变异知之甚少。我们报告了一种全基因组关联研究，该研究使用卡塔尔人口中的45个临床相关性状，在发现的6218个人中进行了全基因组测序，并在7768名受试者中进行了复制。卡塔尔人和欧洲人（r=0.81）之间的性状遗传力比非洲人（r=0.44）更相似。我们确定了281个不同的变异性状关联，它们在基因组范围内具有复制已知关联的意义。复制的基因座的等位基因频率与欧洲人群（r=0.94）的相关性高于非洲人群（r=0.85）或日本人群（r=0.80）。我们发现与以前的报告相比，连锁不平衡模式和复制基因座的效应大小存在差异。我们还报告了17种新颖的信号和卡塔尔信号，从而为调节这些特征的生物学途径提供了见识。我们观察到，欧洲衍生的多基因评分（PGS）降低了卡塔尔人群的预测性能，这可能对人群之间PGS的翻译及其在精密医学中的未来应用产生影响。</p>

<h2>Pubmed_ID: 33619799</h2>

<h3>Different functional roles of RTR complex factors in DNA repair and meiosis in Arabidopsis and tomato</h3>

<h3>RTR复合因子在拟南芥和番茄DNA修复和减数分裂中的不同功能作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15211</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33619799/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/22/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The RTR (RecQ/Top3/Rmi1) complex has been elucidated as essential for ensuring genome stability in eukaryotes. Fundamental for the dissolution of Holliday junction (HJ)-like recombination intermediates, the factors have been shown to play further, partly distinct roles in DNA repair and homologous recombination. Across all kingdoms, disruption of this complex results in characteristic phenotypes including hyperrecombination and sensitivity to genotoxins. The type IA topoisomerase TOP3α has been shown as essential for viability in various animals. In contrast, in the model plant species Arabidopsis, the top3α mutant is viable. rmi1 mutants are deficient in the repair of DNA damage. Moreover, as opposed to other eukaryotes, TOP3α and RMI1 were found to be indispensable for proper meiotic progression, with mutants showing severe meiotic defects and sterility. We now established mutants of both TOP3α and RMI1 in tomato using CRISPR/Cas technology. Surprisingly, we found phenotypes that differed dramatically from those of Arabidopsis: The top3α mutants proved to be embryo-lethal, implying an essential role of the topoisomerase in tomato. In contrast, no defect in somatic DNA repair or meiosis was detectable for rmi1 mutants in tomato. This points to a differentiation of function of RTR complex partners between plant species. Our results indicate that there are relevant differences in the roles of basic factors involved in DNA repair and meiosis within dicotyledons, and thus should be taken as a note of caution when generalising knowledge regarding basic biological processes obtained in the model plant Arabidopsis for the entire plant kingdom.</p>

<p style="line-height:1.8;margin:20px"> 摘要: RTR（RecQ/Top3/Rmi1）复杂已被阐明为确保真核生物中基因组稳定性的必要条件。该因子是霍利迪连接（HJ）样重组中间体溶解的基础，已显示在DNA修复和同源重组中起着进一步，部分不同的作用。在所有王国中，这种复合物的破坏都会导致特征性的表型，包括超重组和对基因毒素的敏感性。IA型拓扑异构酶TOP3α已被证明对各种动物的生存能力至关重要。相反，在模型植物物种拟南芥中，top3α突变体是可行的。rmi1突变体不足以修复DNA损伤。此外，与其他真核生物相反，发现TOP3α和RMI1对于适当的减数分裂进程是必不可少的，突变体显示出严重的减数分裂缺陷和不育。现在，我们使用CRISPR/Cas技术在番茄中建立了TOP3α和RMI1的突变体。出乎意料的是，我们发现其表型与拟南芥的显着不同：top3α突变体被证明具有胚胎致死性，这暗示了拓扑异构酶在番茄中的重要作用。相反，番茄中的rmi1突变体未检测到体细胞DNA修复或减数分裂的缺陷。这表明植物物种之间的RTR复合伴侣功能不同。我们的结果表明，双子叶植物内参与DNA修复和减数分裂的基本因子的作用存在相关差异，因此，当对有关整个植物在模型植物拟南芥中获得的基本生物学过程的知识进行概括时，应注意这一点王国。</p>

<h2>Pubmed_ID: 33619356</h2>

<h3>Enhancing grain-yield-related traits by CRISPR-Cas9 promoter editing of maize CLE genes</h3>

<h3>通过CRISPR-Cas9启动子编辑玉米CLE基因增强谷物产量相关性状 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41477-021-00858-5</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33619356/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature plants</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/22/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Several yield-related traits selected during crop domestication and improvement1,2 are associated with increases in meristem size3, which is controlled by CLE peptide signals in the CLAVATA-WUSCHEL pathway4-13. Here, we engineered quantitative variation for yield-related traits in maize by making weak promoter alleles of CLE genes, and a null allele of a newly identified partially redundant compensating CLE gene, using CRISPR-Cas9 genome editing. These strategies increased multiple maize grain-yield-related traits, supporting the enormous potential for genomic editing in crop enhancement.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在作物驯化和改良过程中选择的几种与产量相关的性状1,2与分生组织大小的增加有关，3受CLAVATA-WUSCHEL途径4-13中的CLE肽信号控制。在这里，我们使用CRISPR-Cas9基因组编辑技术，通过使CLE基因的弱启动子等位基因和新鉴定出的部分冗余补偿CLE基因的无效等位基因工程化了玉米产量相关性状的定量变异。这些策略增加了与玉米产量相关的多个性状，为作物改良中的基因组编辑提供了巨大的潜力。</p>

<h2>Pubmed_ID: 33619257</h2>

<h3>Uniform genomic data analysis in the NCI Genomic Data Commons</h3>

<h3>NCI基因组数据共享中的统一基因组数据分析 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21254-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33619257/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/22/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The goal of the National Cancer Institute's (NCI's) Genomic Data Commons (GDC) is to provide the cancer research community with a data repository of uniformly processed genomic and associated clinical data that enables data sharing and collaborative analysis in the support of precision medicine. The initial GDC dataset include genomic, epigenomic, proteomic, clinical and other data from the NCI TCGA and TARGET programs. Data production for the GDC started in June, 2015 using an OpenStack-based private cloud. By June of 2016, the GDC had analyzed more than 50,000 raw sequencing data inputs, as well as multiple other data types. Using the latest human genome reference build GRCh38, the GDC generated a variety of data types from aligned reads to somatic mutations, gene expression, miRNA expression, DNA methylation status, and copy number variation. In this paper, we describe the pipelines and workflows used to process and harmonize the data in the GDC. The generated data, as well as the original input files from TCGA and TARGET, are available for download and exploratory analysis at the GDC Data Portal and Legacy Archive ( https://gdc.cancer.gov/ ).</p>

<p style="line-height:1.8;margin:20px"> 摘要: 美国国家癌症研究所（NCI）基因组数据共享中心（GDC）的目标是为癌症研究界提供统一处理的基因组和相关临床数据的数据库，从而在精密医学的支持下实现数据共享和协作分析。最初的GDC数据集包括来自NCITCGA和TARGET程序的基因组，表观基因组，蛋白质组学，临床和其他数据。GDC的数据生产于2015年6月开始，使用基于OpenStack的私有云。截至2016年6月，GDC已分析了50,000多个原始测序数据输入以及多种其他数据类型。使用最新的人类基因组参考构建GRCh38，GDC生成了从对齐读取到体细胞突变，基因表达，miRNA表达，DNA甲基化状态和拷贝数变异的各种数据类型。在本文中，我们描述了用于处理和协调GDC中数据的管道和工作流。生成的数据以及TCGA和TARGET的原始输入文件可在GDC数据门户网站和旧版存档（https://gdc.cancer.gov/）上进行下载和探索性分析。</p>

<h2>Pubmed_ID: 33619107</h2>

<h3>Bromodomain proteins regulate human cytomegalovirus latency and reactivation allowing epigenetic therapeutic intervention</h3>

<h3>Bromodomain蛋白调节人类巨细胞病毒潜伏期和再激活，允许表观遗传治疗干预 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2023025118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33619107/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 03/02/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Reactivation of human cytomegalovirus (HCMV) from latency is a major health consideration for recipients of stem-cell and solid organ transplantations. With over 200,000 transplants taking place globally per annum, virus reactivation can occur in more than 50% of cases leading to loss of grafts as well as serious morbidity and even mortality. Here, we present the most extensive screening to date of epigenetic inhibitors on HCMV latently infected cells and find that histone deacetylase inhibitors (HDACis) and bromodomain inhibitors are broadly effective at inducing virus immediate early gene expression. However, while HDACis, such as myeloid-selective CHR-4487, lead to production of infectious virions, inhibitors of bromodomain (BRD) and extraterminal proteins (I-BETs), including GSK726, restrict full reactivation. Mechanistically, we show that BET proteins (BRDs) are pivotally connected to regulation of HCMV latency and reactivation. Through BRD4 interaction, the transcriptional activator complex P-TEFb (CDK9/CycT1) is sequestered by repressive complexes during HCMV latency. Consequently, I-BETs allow release of P-TEFb and subsequent recruitment to promoters via the superelongation complex (SEC), inducing transcription of HCMV lytic genes encoding immunogenic antigens from otherwise latently infected cells. Surprisingly, this occurs without inducing many viral immunoevasins and, importantly, while also restricting viral DNA replication and full HCMV reactivation. Therefore, this pattern of HCMV transcriptional dysregulation allows effective cytotoxic immune targeting and killing of latently infected cells, thus reducing the latent virus genome load. This approach could be safely used to pre-emptively purge the virus latent reservoir prior to transplantation, thereby reducing HCMV reactivation-related morbidity and mortality.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 从潜伏期恢复人类巨细胞病毒（HCMV）是干细胞和实体器官移植接受者的主要健康考虑。全球每年有超过200,000例移植手术，超过50％的病例会发生病毒再激活，从而导致移植物丢失以及严重的发病率甚至死亡。在这里，我们介绍了迄今为止对HCMV潜伏感染细胞的表观遗传抑制剂的最广泛筛选，并发现组蛋白脱乙酰基酶抑制剂（HDACis）和溴结构域抑制剂在诱导病毒即刻早期基因表达方面具有广泛的作用。但是，尽管HDACis（例如，髓样选择性CHR-4487）导致产生感染性病毒粒子，但bromodomain（BRD）和末端外蛋白（I-BETs）的抑制剂（包括GSK726）限制了完全重新激活。从机制上讲，我们表明BET蛋白（BRD）与HCMV潜伏期和再激活的调节至关重要。通过BRD4相互作用，在HCMV潜伏期中，转录激活复合物P-TEFb（CDK9/CycT1）被抑制性复合物隔离。因此，I-BETs允许P-TEFb释放并随后通过超伸长复合物（SEC）募集到启动子，从而诱导HCMV裂解基因的转录，而HCMV裂解基因则编码来自潜在感染细胞的免疫原性抗原。出人意料的是，这发生在没有诱导许多病毒免疫evasins的情况下，重要的是，同时也限制了病毒DNA的复制和HCMV的完全活化。因此，HCMV转录失调的这种模式允许有效的细胞毒性免疫靶向和杀死潜伏感染的细胞，从而减少潜伏病毒基因组负荷。该方法可以安全地用于在移植之前抢先清除病毒潜伏库，从而降低与HCMV激活相关的发病率和死亡率。</p>

<h2>Pubmed_ID: 33618348</h2>

<h3>Genomic Insights into the Formation of Human Populations in East Asia</h3>

<h3>关于东亚人口形成的基因组学见解 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-021-03336-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33618348/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/22/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The deep population history of East Asia remains poorly understood due to a lack of ancient DNA data and sparse sampling of present-day people1,2. We report genome-wide data from 166 East Asians dating to 6000 BCE - 1000 CE and 46 present-day groups. Hunter-gatherers from Japan, the Amur River Basin, and people of Neolithic and Iron Age Taiwan and the Tibetan plateau are linked by a deeply-splitting lineage likely reflecting a Late Pleistocene coastal migration. We follow Holocene expansions from four regions. First, hunter-gatherers of Mongolia and the Amur River Basin have ancestry shared by Mongolic and Tungusic language speakers but do not carry West Liao River farmer ancestry contradicting theories that their expansion spread these proto-languages. Second, Yellow River Basin farmers at ~3000 BCE likely spread Sino-Tibetan languages as their ancestry dispersed both to Tibet where it forms up ~84% to some groups and to the Central Plain where it contributed ~59-84% to Han Chinese. Third, people from Taiwan ~1300 BCE to 800 CE derived ~75% ancestry from a lineage also common in modern Austronesian, Tai-Kadai and Austroasiatic speakers likely deriving from Yangtze River Valley farmers; ancient Taiwan people also derived ~25% ancestry from a northern lineage related to but different from Yellow River farmers implying an additional north-to-south expansion. Fourth, Yamnaya Steppe pastoralist ancestry arrived in western Mongolia after ~3000 BCE but was displaced by previously established lineages even while it persisted in western China as expected if it spread the ancestor of Tocharian Indo-European languages. Two later gene flows affected western Mongolia: after ~2000 BCE migrants with Yamnaya and European farmer ancestry, and episodic impacts of later groups with ancestry from Turan.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 由于缺乏古代DNA数据和当今人口的稀疏采样，东亚的深厚人口历史仍然知之甚少1,2。我们报告了166个东亚人的全基因组数据，可追溯到公元前6000年-1000年，还有46个现今的群体。来自日本，阿穆尔河盆地，新石器时代和铁器时代的台湾以及青藏高原的狩猎者与采集者之间有着深切的联系，这可能反映了晚更新世的沿海移民。我们关注从四个地区的全新世扩张。首先，蒙古和阿穆尔河流域的狩猎者采集者由蒙古语和通古斯语使用者共享血统，但没有携带西辽河农民血统，这与他们的扩张传播这些原始语言的理论相矛盾。其次，公元前3000年左右的黄河流域农民很可能传播汉藏语言，因为他们的祖先分散到西藏，在某些群体中占约84％，而在中原地区，汉族则占约59-84％。第三，从公元前1300年到公元前800年的台湾人，大约有75％的祖先来自现代的南斯拉夫语系，台卡代语和澳大拉西亚语系的人，这些人很可能来自长江流域的农民。台湾古代人还从约有25％的祖先起源于与黄河农民有关但又与黄河农民不同的北部血统，这意味着南北向有进一步的扩张。第四，Yamnaya草原牧民的祖先大约在公元前3000年到达蒙古西部，但是即使它如期传承了Tocharian印欧语种的祖先一样在中国西部仍然存在，却被先前建立的血统所取代。后来的两个基因流影响了蒙古西部：在公元前2000年之后，具有Yamnaya和欧洲农民血统的移民，以及后来来自图兰的血统的群体发生了偶发性影响。</p>

<h2>Pubmed_ID: 33608555</h2>

<h3>Hologenome analysis reveals dual symbiosis in the deep-sea hydrothermal vent snail Gigantopelta aegis</h3>

<h3>完整的基因组分析显示深海热液发泄蜗牛Gigantopelta aegis中的双重共生 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21450-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33608555/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/19/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Animals endemic to deep-sea hydrothermal vents often form obligatory symbioses with bacteria, maintained by intricate host-symbiont interactions. Most genomic studies on holobionts have not investigated both sides to similar depths. Here, we report dual symbiosis in the peltospirid snail Gigantopelta aegis with two gammaproteobacterial endosymbionts: a sulfur oxidiser and a methane oxidiser. We assemble high-quality genomes for all three parties, including a chromosome-level host genome. Hologenomic analyses reveal mutualism with nutritional complementarity and metabolic co-dependency, highly versatile in transporting and using chemical energy. Gigantopelta aegis likely remodels its immune system to facilitate dual symbiosis. Comparisons with Chrysomallon squamiferum, a confamilial snail with a single sulfur-oxidising gammaproteobacterial endosymbiont, show that their sulfur-oxidising endosymbionts are phylogenetically distant. This is consistent with previous findings that they evolved endosymbiosis convergently. Notably, the two sulfur-oxidisers share the same capabilities in biosynthesising nutrients lacking in the host genomes, potentially a key criterion in symbiont selection.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 深海热液喷口特有的动物通常与细菌形成强制性共生，由复杂的宿主-共生体相互作用维持。关于全生命周期的大多数基因组研究都没有研究相似深度的两面。在这里，我们报道了在螺螺蜗牛Gigantopeltaaegis与两种γ-变形杆菌内共生菌：硫氧化剂和甲烷氧化剂的双重共生。我们为所有三个方面组装高质量的基因组，包括染色体级宿主基因组。全基因组分析揭示了具有营养互补性和代谢依赖性的共生关系，在运输和使用化学能方面具有高度的通用性。巨猿盾可能重塑其免疫系统，以促进双重共生。与带有单一硫氧化γ-变形细菌内共生体的同系蜗牛-金鳞瓜（Chrysomallonsquamiferum）的比较表明，它们的硫氧化内共生体在系统发育上距离较远。这与以前的发现一致，即它们共进化为共生共生。值得注意的是，两种硫氧化剂在生物合成宿主基因组中缺乏的营养素方面具有相同的功能，这可能是共生体选择的关键标准。</p>

<h2>Pubmed_ID: 33608531</h2>

<h3>GWAS of peptic ulcer disease implicates Helicobacter pylori infection, other gastrointestinal disorders and depression</h3>

<h3>消化性溃疡疾病的GWAS涉及幽门螺杆菌感染，其他胃肠道疾病和抑郁症 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21280-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33608531/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/19/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genetic factors are recognized to contribute to peptic ulcer disease (PUD) and other gastrointestinal diseases, such as gastro-oesophageal reflux disease (GORD), irritable bowel syndrome (IBS) and inflammatory bowel disease (IBD). Here, genome-wide association study (GWAS) analyses based on 456,327 UK Biobank (UKB) individuals identify 8 independent and significant loci for PUD at, or near, genes MUC1, MUC6, FUT2, PSCA, ABO, CDX2, GAST and CCKBR. There are previously established roles in susceptibility to Helicobacter pylori infection, response to counteract infection-related damage, gastric acid secretion or gastrointestinal motility for these genes. Only two associations have been previously reported for duodenal ulcer, here replicated trans-ancestrally. The results highlight the role of host genetic susceptibility to infection. Post-GWAS analyses for PUD, GORD, IBS and IBD add insights into relationships between these gastrointestinal diseases and their relationships with depression, a commonly comorbid disorder.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 遗传因素被认为是导致消化性溃疡疾病（PUD）和其他胃肠道疾病的原因，例如胃食管反流病（GORD），肠易激综合症（IBS）和炎症性肠病（IBD）。在这里，基于456,327个英国生物库（UKB）个体的全基因组关联研究（GWAS）分析确定了在基因MUC1，MUC6，FUT2，PSCA，ABO，CDX2，GAST和CCKBR或其附近的8个独立且重要的PUD基因座。对于这些基因，在幽门螺杆菌感染的易感性，对付与感染相关的损害的反应，胃酸分泌或胃肠蠕动方面已经确立了作用。先前仅报道过十二指肠溃疡的两个关联，此处是经祖先复制的。结果突出了宿主遗传易感性对感染的作用。GWAS后对PUD，GORD，IBS和IBD的分析增加了对这些胃肠道疾病之间的关系及其与抑郁症（通常为合并症）之间关系的了解。</p>

<h2>Pubmed_ID: 33608517</h2>

<h3>Widespread signatures of natural selection across human complex traits and functional genomic categories</h3>

<h3>跨人类复杂特征和功能基因组类别的自然选择的广泛特征 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21446-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33608517/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/19/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Understanding how natural selection has shaped genetic architecture of complex traits is of importance in medical and evolutionary genetics. Bayesian methods have been developed using individual-level GWAS data to estimate multiple genetic architecture parameters including selection signature. Here, we present a method (SBayesS) that only requires GWAS summary statistics. We analyse data for 155 complex traits (n = 27k-547k) and project the estimates onto those obtained from evolutionary simulations. We estimate that, on average across traits, about 1% of human genome sequence are mutational targets with a mean selection coefficient of ~0.001. Common diseases, on average, show a smaller number of mutational targets and have been under stronger selection, compared to other traits. SBayesS analyses incorporating functional annotations reveal that selection signatures vary across genomic regions, among which coding regions have the strongest selection signature and are enriched for both the number of associated variants and the magnitude of effect sizes.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在医学和进化遗传学中，了解自然选择如何塑造复杂性状的遗传结构非常重要。已经使用个体水平的GWAS数据开发了贝叶斯方法，以估计多个遗传结构参数，包括选择特征。在这里，我们提出一种仅需要GWAS摘要统计信息的方法（SBayesS）。我们分析了155个复杂特征的数据（n=27k-547k），并将估计值投影到了从进化模拟获得的估计值上。我们估计，平均而言，整个性状中，约有1％的人类基因组序列是突变靶标，平均选择系数约为0.001。平均而言，与其他特征相比，常见疾病显示出较少的突变靶标，并且处于更强的选择之下。结合功能注释的SBayesS分析显示，选择标记在基因组区域之间变化，其中编码区域具有最强的选择标记，并且对于相关变体的数量和效应大小的幅度都得到了丰富。</p>

<h2>Pubmed_ID: 33608481</h2>

<h3>The inflated significance of neutral genetic diversity in conservation genetics</h3>

<h3>中性遗传多样性在保护遗传学中的夸大意义 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2015096118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33608481/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 03/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The current rate of species extinction is rapidly approaching unprecedented highs, and life on Earth presently faces a sixth mass extinction event driven by anthropogenic activity, climate change, and ecological collapse. The field of conservation genetics aims at preserving species by using their levels of genetic diversity, usually measured as neutral genome-wide diversity, as a barometer for evaluating population health and extinction risk. A fundamental assumption is that higher levels of genetic diversity lead to an increase in fitness and long-term survival of a species. Here, we argue against the perceived importance of neutral genetic diversity for the conservation of wild populations and species. We demonstrate that no simple general relationship exists between neutral genetic diversity and the risk of species extinction. Instead, a better understanding of the properties of functional genetic diversity, demographic history, and ecological relationships is necessary for developing and implementing effective conservation genetic strategies.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 当前物种灭绝的速度正在迅速接近空前的高点，地球上的生命目前面临着第六次大规模灭绝事件，该事件是由人为活动，气候变化和生态崩溃导致的。保护遗传学领域旨在通过使用物种的遗传多样性水平（通常以中性全基因组多样性来衡量）来保护物种，以此作为评估种群健康和灭绝风险的晴雨表。一个基本的假设是，更高水平的遗传多样性会导致一个物种的适应性和长期存活率提高。在这里，我们反对认为中性遗传多样性对于保护野生种群和物种的重要性。我们证明，中性遗传多样性与物种灭绝风险之间不存在简单的一般关系。相反，必须更好地了解功能遗传多样性，人口历史和生态关系的性质，才能制定和实施有效的保护遗传策略。</p>

<h2>Pubmed_ID: 33606978</h2>

<h3>Functional interrogation of DNA damage response variants with base editing screens</h3>

<h3>通过基本编辑屏幕对DNA损伤反应变体进行功能询问 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2021.01.041</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33606978/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/18/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Mutations in DNA damage response (DDR) genes endanger genome integrity and predispose to cancer and genetic disorders. Here, using CRISPR-dependent cytosine base editing screens, we identify > 2,000 sgRNAs that generate nucleotide variants in 86 DDR genes, resulting in altered cellular fitness upon DNA damage. Among those variants, we discover loss- and gain-of-function mutants in the Tudor domain of the DDR regulator 53BP1 that define a non-canonical surface required for binding the deubiquitinase USP28. Moreover, we characterize variants of the TRAIP ubiquitin ligase that define a domain, whose loss renders cells resistant to topoisomerase I inhibition. Finally, we identify mutations in the ATM kinase with opposing genome stability phenotypes and loss-of-function mutations in the CHK2 kinase previously categorized as variants of uncertain significance for breast cancer. We anticipate that this resource will enable the discovery of additional DDR gene functions and expedite studies of DDR variants in human disease.</p>

<p style="line-height:1.8;margin:20px"> 摘要: DNA损伤反应（DDR）基因的突变危及基因组完整性，并易患癌症和遗传疾病。在这里，我们使用CRISPR依赖的胞嘧啶碱基编辑屏幕，鉴定出>2,000个sgRNA，这些sgRNA在86个DDR基因中产生核苷酸变体，从而导致DNA损伤后细胞适应性发生改变。在这些变体中，我们在DDR调节器53BP1的Tudor域中发现了功能丧失和功能获得的突变体，这些突变体定义了结合去泛素化酶USP28所需的非规范表面。此外，我们表征了TRAIP泛素连接酶的变体，该变体定义了一个域，该域的丢失使细胞对拓扑异构酶I的抑制具有抗性。最后，我们确定了具有相反基因组稳定性表型的ATM激酶突变和CHK2激酶的功能丧失突变，这些突变先前被归类为对乳腺癌具有不确定意义的变异。我们预计，该资源将有助于发现其他DDR基因功能，并加快人类疾病中DDR变体的研究。</p>

<h2>Pubmed_ID: 33602935</h2>

<h3>Inflammation status modulates the effect of host genetic variation on intestinal gene expression in inflammatory bowel disease</h3>

<h3>炎症状态调节炎症性肠病中宿主基因变异对肠道基因表达的影响 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21458-z</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33602935/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/18/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: More than 240 genetic risk loci have been associated with inflammatory bowel disease (IBD), but little is known about how they contribute to disease development in involved tissue. Here, we hypothesized that host genetic variation affects gene expression in an inflammation-dependent way, and investigated 299 snap-frozen intestinal biopsies from inflamed and non-inflamed mucosa from 171 IBD patients. RNA-sequencing was performed, and genotypes were determined using whole exome sequencing and genome wide genotyping. In total, 28,746 genes and 6,894,979 SNPs were included. Linear mixed models identified 8,881 independent intestinal cis-expression quantitative trait loci (cis-eQTLs) (FDR < 0.05) and interaction analysis revealed 190 inflammation-dependent intestinal cis-eQTLs (FDR < 0.05), including known IBD-risk genes and genes encoding immune-cell receptors and antibodies. The inflammation-dependent cis-eQTL SNPs (eSNPs) mainly interact with prevalence of immune cell types. Inflammation-dependent intestinal cis-eQTLs reveal genetic susceptibility under inflammatory conditions that can help identify the cell types involved in and the pathways underlying inflammation, knowledge that may guide future drug development and profile patients for precision medicine in IBD.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 已有240多个遗传风险基因座与炎症性肠病（IBD）相关，但鲜为人知，它们如何导致受累组织的疾病发展。在这里，我们假设宿主的遗传变异会以炎症依赖性的方式影响基因表达，并调查了171名IBD患者发炎和未发炎的粘膜的299份速冻肠活检样品。进行RNA测序，并使用全外显子组测序和全基因组基因分型确定基因型。总共包括28,746个基因和6,894,979个SNP。线性混合模型确定了8,881个独立的肠道顺式表达数量性状基因座（FDR<0.05），并且相互作用分析揭示了190个炎症依赖性肠顺式EQTL（FDR<0.05），包括已知的IBD风险基因和编码免疫细胞受体和抗体。炎症依赖性顺式-eQTLSNP（eSNPs）主要与免疫细胞类型的流行相互作用。炎症依赖性肠道顺式eQTLs在炎症条件下揭示了遗传易感性，可帮助鉴定参与炎症的细胞类型和潜在的炎症途径，这些知识可指导未来的药物开发并为IBD精准医学的患者提供资料。</p>

<h2>Pubmed_ID: 33602924</h2>

<h3>POLRMT mutations impair mitochondrial transcription causing neurological disease</h3>

<h3>POLRMT突变削弱线粒体转录，导致神经系统疾病 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21279-0</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33602924/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/18/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: While >300 disease-causing variants have been identified in the mitochondrial DNA (mtDNA) polymerase γ, no mitochondrial phenotypes have been associated with POLRMT, the RNA polymerase responsible for transcription of the mitochondrial genome. Here, we characterise the clinical and molecular nature of POLRMT variants in eight individuals from seven unrelated families. Patients present with global developmental delay, hypotonia, short stature, and speech/intellectual disability in childhood; one subject displayed an indolent progressive external ophthalmoplegia phenotype. Massive parallel sequencing of all subjects identifies recessive and dominant variants in the POLRMT gene. Patient fibroblasts have a defect in mitochondrial mRNA synthesis, but no mtDNA deletions or copy number abnormalities. The in vitro characterisation of the recombinant POLRMT mutants reveals variable, but deleterious effects on mitochondrial transcription. Together, our in vivo and in vitro functional studies of POLRMT variants establish defective mitochondrial transcription as an important disease mechanism.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 尽管在线粒体DNA（mtDNA）聚合酶γ中发现了300多种致病变体，但没有线粒体表型与负责线粒体基因组转录的RNA聚合酶POLRMT相关。在这里，我们表征了来自七个不相关家庭的八个人中POLRMT变体的临床和分子性质。患者在儿童时期出现整体发育迟缓，肌张力低下，身材矮小和言语/智力障碍；一名受试者表现出惰性渐进性外部眼肌麻痹表型。所有受试者的大规模平行测序鉴定了POLRMT基因的隐性和显性变异。患者成纤维细胞的线粒体mRNA合成有缺陷，但没有mtDNA缺失或拷贝数异常。重组POLRMT突变体的体外表征揭示了对线粒体转录的可变但有害的影响。总之，我们对POLRMT变体的体内和体外功能研究将缺陷的线粒体转录确定为重要的疾病机制。</p>

<h2>Pubmed_ID: 33602921</h2>

<h3>Non-CG methylation and multiple histone profiles associate child abuse with immune and small GTPase dysregulation</h3>

<h3>非CG甲基化和多种组蛋白谱将儿童虐待与免疫和小GTPase失调相关联 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21365-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33602921/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/18/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Early-life adversity (ELA) is a major predictor of psychopathology, and is thought to increase lifetime risk by epigenetically regulating the genome. Here, focusing on the lateral amygdala, a major brain site for emotional homeostasis, we describe molecular cross-talk among multiple mechanisms of genomic regulation, including 6 histone marks and DNA methylation, and the transcriptome, in subjects with a history of ELA and controls. In the healthy brain tissue, we first uncover interactions between different histone marks and non-CG methylation in the CAC context. Additionally, we find that ELA associates with methylomic changes that are as frequent in the CAC as in the canonical CG context, while these two forms of plasticity occur in sharply distinct genomic regions, features, and chromatin states. Combining these multiple data indicates that immune-related and small GTPase signaling pathways are most consistently impaired in the amygdala of ELA individuals. Overall, this work provides insights into genomic brain regulation as a function of early-life experience.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 早期逆境（ELA）是心理病理学的主要预测指标，据认为可通过表观遗传调控基因组来增加终生风险。在这里，我们着眼于侧脑杏仁核（情绪稳态的主要大脑部位），我们描述了具有ELA和控制史的受试者中基因组调控的多种机制之间的分子串扰，包括6个组蛋白标记和DNA甲基化以及转录组。。在健康的大脑组织中，我们首先发现在CAC环境中不同组蛋白标记与非CG甲基化之间的相互作用。此外，我们发现ELA与在规范化CAC中频繁发生的甲基化变化相关，而在规范CG环境中则是如此，而这两种可塑性形式则发生在截然不同的基因组区域，特征和染色质状态中。组合这些多个数据表明，在ELA个体的杏仁核中，与免疫相关的GTPase和小的GTPase信号转导途径受到最一致的损害。总的来说，这项工作提供了对基因组大脑调节的见解，这是早期生活经验的函数。</p>

<h2>Pubmed_ID: 33602909</h2>

<h3>Genome-wide association studies provide insights into the genetic determination of fruit traits of pear</h3>

<h3>全基因组关联研究为了解梨果实性状的遗传测定提供了见识 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21378-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33602909/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/18/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Pear is a major fruit tree crop distributed worldwide, yet its breeding is a very time-consuming process. To facilitate molecular breeding and gene identification, here we have performed genome-wide association studies (GWAS) on eleven fruit traits. We identify 37 loci associated with eight fruit quality traits and five loci associated with three fruit phenological traits. Scans for selective sweeps indicate that traits including fruit stone cell content, organic acid and sugar contents might have been under continuous selection during breeding improvement. One candidate gene, PbrSTONE, identified in GWAS, has been functionally verified to be involved in the regulation of stone cell formation, one of the most important fruit quality traits in pear. Our study provides insights into the complex fruit related biology and identifies genes controlling important traits in pear through GWAS, which extends the genetic resources and basis for facilitating molecular breeding in perennial trees.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 梨是分布在世界各地的主要果树作物，但是其育种是一个非常耗时的过程。为了促进分子育种和基因鉴定，我们在这里对11个水果性状进行了全基因组关联研究（GWAS）。我们确定了与八个水果品质性状相关的37个基因座和与三个水果物候性状相关的五个基因座。选择性扫除的扫描表明，在改良育种过程中，包括果核细胞含量，有机酸和糖含量在内的性状可能一直处于连续选择状态。在GWAS中鉴定出的一种候选基因PbrSTONE已通过功能验证，可参与调节核果细胞的形成，这是梨中最重要的果实品质性状之一。我们的研究为复杂的水果相关生物学提供了见识，并通过GWAS鉴定了控制梨重要性状的基因，从而扩展了遗传资源和促进多年生树木分子育种的基础。</p>

<h2>Pubmed_ID: 33602827</h2>

<h3>Recurrent evolution of vertebrate transcription factors by transposase capture</h3>

<h3>转座酶捕获的脊椎动物转录因子的反复进化 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1126/science.abc6405</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33602827/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Science (New York, N.Y.)</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/19/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genes with novel cellular functions may evolve through exon shuffling, which can assemble novel protein architectures. Here, we show that DNA transposons provide a recurrent supply of materials to assemble protein-coding genes through exon shuffling. We find that transposase domains have been captured-primarily via alternative splicing-to form fusion proteins at least 94 times independently over the course of ~350 million years of tetrapod evolution. We find an excess of transposase DNA binding domains fused to host regulatory domains, especially the Krüppel-associated box (KRAB) domain, and identify four independently evolved KRAB-transposase fusion proteins repressing gene expression in a sequence-specific fashion. The bat-specific KRABINER fusion protein binds its cognate transposons genome-wide and controls a network of genes and cis-regulatory elements. These results illustrate how a transcription factor and its binding sites can emerge.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 具有新型细胞功能的基因可能会通过外显子改组而进化，这可以组装出新颖的蛋白质结构。在这里，我们显示DNA转座子提供了通过外显子改组组装蛋白质编码基因的材料的周期性供应。我们发现转座酶域主要是通过替代剪接捕获的，在大约3.5亿年的四足动物进化过程中，至少独立地形成了94次融合蛋白。我们发现过量的转座酶DNA结合结构域融合到宿主调节域，尤其是与Krüppel相关框（KRAB）域，并确定了四个独立进化的KRAB-转座酶融合蛋白以序列特异性方式抑制基因表达。蝙蝠特异性KRABINER融合蛋白在全基因组范围内结合其同源转座子，并控制基因和顺式调控元件的网络。这些结果说明了转录因子及其结合位点如何出现。</p>

<h2>Pubmed_ID: 33602814</h2>

<h3>RPA-mediated recruitment of Bre1 couples histone H2B ubiquitination to DNA replication and repair</h3>

<h3>RPA介导的Bre1募集使组蛋白H2B泛素化与DNA复制和修复 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2017497118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33602814/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The ubiquitin E3 ligase Bre1-mediated H2B monoubiquitination (H2Bub) is essential for proper DNA replication and repair in eukaryotes. Deficiency in H2Bub causes genome instability and cancer. How the Bre1-H2Bub pathway is evoked in response to DNA replication or repair remains unknown. Here, we identify that the single-stranded DNA (ssDNA) binding factor RPA acts as a key mediator that couples Bre1-mediated H2Bub to DNA replication and repair in yeast. We found that RPA interacts with Bre1 in vitro and in vivo, and this interaction is stimulated by ssDNA. This association ensures the recruitment of Bre1 to replication forks or DNA breaks but does not affect its E3 ligase activity. Disruption of the interaction abolishes the local enrichment of H2Bub, resulting in impaired DNA replication, response to replication stress, and repair by homologous recombination, accompanied by increased genome instability and DNA damage sensitivity. Notably, we found that RNF20, the human homolog of Bre1, interacts with RPA70 in a conserved mode. Thus, RPA functions as a master regulator for the spatial-temporal control of H2Bub chromatin landscape during DNA replication and recombination, extending the versatile roles of RPA in guarding genome stability.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 泛素E3连接酶Bre1介导的H2B单泛素化（H2Bub）对于真核生物中正确的DNA复制和修复至关重要。H2Bub缺乏会导致基因组不稳定和癌症。Bre1-H2Bub途径如何响应DNA复制或修复仍是未知的。在这里，我们确定单链DNA（ssDNA）结合因子RPA充当将Bre1介导的H2Bub与酵母中的DNA复制和修复偶联的关键介体。我们发现RPA在体外和体内均与Bre1相互作用，并且这种相互作用受到ssDNA的刺激。这种关联确保Bre1募集到复制叉或DNA断裂，但不影响其E3连接酶活性。相互作用的破坏消除了H2Bub的局部富集，导致DNA复制受损，对复制压力的反应以及同源重组修复，伴随着基因组不稳定性和DNA损伤敏感性的增加。值得注意的是，我们发现RNF20是Bre1的人类同源物，它以保守模式与RPA70相互作用。因此，RPA充当DNA复制和重组过程中H2Bub染色质景观的时空控制的主要调节剂，从而扩展了RPA在保护基因组稳定性方面的通用作用。</p>

<h2>Pubmed_ID: 33602812</h2>

<h3>Emergence of diauxie as an optimal growth strategy under resource allocation constraints in cellular metabolism</h3>

<h3>在细胞代谢资源分配约束下，diauxie的出现作为一种最佳的生长策略 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2013836118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33602812/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Diauxie, or the sequential consumption of carbohydrates in bacteria such as Escherichia coli, has been hypothesized to be an evolutionary strategy which allows the organism to maximize its instantaneous specific growth-giving the bacterium a competitive advantage. Currently, the computational techniques used in industrial biotechnology fall short of explaining the intracellular dynamics underlying diauxic behavior. In particular, the understanding of the proteome dynamics in diauxie can be improved. We developed a robust iterative dynamic method based on expression- and thermodynamically enabled flux models to simulate the kinetic evolution of carbohydrate consumption and cellular growth. With minimal modeling assumptions, we couple kinetic uptakes, gene expression, and metabolic networks, at the genome scale, to produce dynamic simulations of cell cultures. The method successfully predicts the preferential uptake of glucose over lactose in E. coli cultures grown on a mixture of carbohydrates, a manifestation of diauxie. The simulated cellular states also show the reprogramming in the content of the proteome in response to fluctuations in the availability of carbon sources, and it captures the associated time lag during the diauxie phenotype. Our models suggest that the diauxic behavior of cells is the result of the evolutionary objective of maximization of the specific growth of the cell. We propose that genetic regulatory networks, such as the lac operon in E. coli, are the biological implementation of a robust control system to ensure optimal growth.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 假设Diauxie或细菌（如大肠杆菌）中碳水化合物的顺序消耗是一种进化策略，该策略可使生物体最大化其瞬时特异性生长，从而赋予细菌竞争优势。当前，工业生物技术中使用的计算技术不足以解释潜在的双生行为的细胞内动力学。特别地，可以提高对双生蛋白质组动力学的理解。我们基于表达和热力学启用的通量模型开发了鲁棒的迭代动力学方法，以模拟碳水化合物消耗和细胞生长的动力学演变。在最小的建模假设下，我们将在基因组规模上耦合动力学摄取，基因表达和代谢网络，以产生细胞培养物的动态模拟。该方法成功地预测了在碳水化合物混合物上生长的大肠杆菌培养物中葡萄糖相对于乳糖的优先摄取，这是双生的表现。模拟的细胞状态还显示了蛋白质组中内容的重新编程，以响应碳源可用性的波动，并且捕获了在diauxie表型过程中的相关时滞。我们的模型表明，细胞的双生行为是细胞特定生长最大化的进化目标的结果。我们建议，遗传调控网络，如大肠杆菌中的lac操纵子，是一种生物控制有效的控制系统，以确保最佳生长。</p>

<h2>Pubmed_ID: 33597750</h2>

<h3>Million-year-old DNA sheds light on the genomic history of mammoths</h3>

<h3>数百万年前的DNA揭示了猛ma象的基因组历史 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-021-03224-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597750/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Temporal genomic data hold great potential for studying evolutionary processes such as speciation. However, sampling across speciation events would, in many cases, require genomic time series that stretch well back into the Early Pleistocene subepoch. Although theoretical models suggest that DNA should survive on this timescale1, the oldest genomic data recovered so far are from a horse specimen dated to 780-560 thousand years ago2. Here we report the recovery of genome-wide data from three mammoth specimens dating to the Early and Middle Pleistocene subepochs, two of which are more than one million years old. We find that two distinct mammoth lineages were present in eastern Siberia during the Early Pleistocene. One of these lineages gave rise to the woolly mammoth and the other represents a previously unrecognized lineage that was ancestral to the first mammoths to colonize North America. Our analyses reveal that the Columbian mammoth of North America traces its ancestry to a Middle Pleistocene hybridization between these two lineages, with roughly equal admixture proportions. Finally, we show that the majority of protein-coding changes associated with cold adaptation in woolly mammoths were already present one million years ago. These findings highlight the potential of deep-time palaeogenomics to expand our understanding of speciation and long-term adaptive evolution.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 时间基因组数据在研究进化过程（例如物种形成）方面具有巨大的潜力。但是，在许多情况下，跨物种事件采样需要大量的基因组时间序列，这些时间序列可以追溯到早更新世亚纪。尽管理论模型表明DNA应该可以在这个时间尺度上生存1，但迄今为止回收的最古老的基因组数据是来自780-56万年前的马标本2。在这里，我们报告了从三个猛mm象标本中恢复的全基因组数据，这些标本可以追溯到早期和中期更新世亚纪，其中两个具有超过一百万年的历史。我们发现，在更新世早期，西伯利亚东部存在两个不同的猛mm血统。这些血统中的一个产生了毛茸茸的猛represents象，而另一个则代表了以前未被认可的血统，该血统是最早在北美殖民地的猛ma象的祖先。我们的分析表明，北美的猛ma象起源于这两个世系之间的中更新世杂交，其混合比例大致相等。最后，我们表明，与羊毛猛cold象的冷适应有关的大多数蛋白质编码变化已经存在于一百万年前。这些发现突出了深层古基因组学的潜力，可以扩大我们对物种形成和长期适应性进化的理解。</p>

<h2>Pubmed_ID: 33597548</h2>

<h3>Stability and folding pathways of tetra-nucleosome from six-dimensional free energy surface</h3>

<h3>六维自由能表面四核小体的稳定性和折叠途径 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21377-z</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597548/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The three-dimensional organization of chromatin is expected to play critical roles in regulating genome functions. High-resolution characterization of its structure and dynamics could improve our understanding of gene regulation mechanisms but has remained challenging. Using a near-atomistic model that preserves the chemical specificity of protein-DNA interactions at residue and base-pair resolution, we studied the stability and folding pathways of a tetra-nucleosome. Dynamical simulations performed with an advanced sampling technique uncovered multiple pathways that connect open chromatin configurations with the zigzag crystal structure. Intermediate states along the simulated folding pathways resemble chromatin configurations reported from in situ experiments. We further determined a six-dimensional free energy surface as a function of the inter-nucleosome distances via a deep learning approach. The zigzag structure can indeed be seen as the global minimum of the surface. However, it is not favored by a significant amount relative to the partially unfolded, in situ configurations. Chemical perturbations such as histone H4 tail acetylation and thermal fluctuations can further tilt the energetic balance to stabilize intermediate states. Our study provides insight into the connection between various reported chromatin configurations and has implications on the in situ relevance of the 30 nm fiber.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 染色质的三维组织有望在调节基因组功能中发挥关键作用。其结构和动力学的高分辨率表征可以增进我们对基因调控机制的理解，但仍然具有挑战性。使用保留了蛋白质-DNA相互作用的化学特异性的残基和碱基对分辨率的近原子模型，我们研究了四核小体的稳定性和折叠途径。使用先进的采样技术进行的动力学仿真揭示了将开放染色质构型与之字形晶体结构联系起来的多种途径。沿模拟折叠路径的中间状态类似于原位实验报道的染色质构型。我们通过深度学习方法进一步确定了六维自由能表面作为核小体间距离的函数。之字形结构确实可以看作是曲面的整体最小值。但是，相对于部分展开的原位构型，它并没有受到太多的支持。化学干扰（例如组蛋白H4尾部乙酰化和热波动）会进一步倾斜能量平衡，以稳定中间状态。我们的研究提供了对各种报道的染色质构型之间的联系的洞察力，并对30nm光纤的原位相关性产生了影响。</p>

<h2>Pubmed_ID: 33597547</h2>

<h3>Genome sequences reveal global dispersal routes and suggest convergent genetic adaptations in seahorse evolution</h3>

<h3>基因组序列揭示了全球扩散途径，并暗示了海马进化中的趋同遗传适应 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21379-x</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597547/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Seahorses have a circum-global distribution in tropical to temperate coastal waters. Yet, seahorses show many adaptations for a sedentary, cryptic lifestyle: they require specific habitats, such as seagrass, kelp or coral reefs, lack pelvic and caudal fins, and give birth to directly developed offspring without pronounced pelagic larval stage, rendering long-range dispersal by conventional means inefficient. Here we investigate seahorses' worldwide dispersal and biogeographic patterns based on a de novo genome assembly of Hippocampus erectus as well as 358 re-sequenced genomes from 21 species. Seahorses evolved in the late Oligocene and subsequent circum-global colonization routes are identified and linked to changing dynamics in ocean currents and paleo-temporal seaway openings. Furthermore, the genetic basis of the recurring "bony spines" adaptive phenotype is linked to independent substitutions in a key developmental gene. Analyses thus suggest that rafting via ocean currents compensates for poor dispersal and rapid adaptation facilitates colonizing new habitats.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 海马在热带到温带沿海水域中具有环球分布。然而，海马显示出多种适应久坐，隐秘的生活方式：它们需要特定的栖息地，例如海草，海带或珊瑚礁，缺少骨盆和尾鳍，并且可以生出直接发育的后代，而没有明显的上层幼体阶段，因此长距离繁殖通过常规手段分散效率低下。在这里，我们根据海马直立的从头基因组装配以及来自21个物种的358个重新测序的基因组，研究了全球范围内海马的分布和生物地理分布。渐新世晚期演化出的海马和随后的环全球定居路线被确定，并与洋流和古时空航道的变化动态联系在一起。此外，反复出现的“骨棘”适应性表型的遗传基础与关键发育基因中的独立取代有关。因此，分析表明，通过洋流漂流可以弥补较差的扩散，而快速的适应则有利于定居新的栖息地。</p>

<h2>Pubmed_ID: 33597536</h2>

<h3>Aquila enables reference-assisted diploid personal genome assembly and comprehensive variant detection based on linked reads</h3>

<h3>Aquila支持基于参考读码的二倍体个人基因组组装和全面的变体检测 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21395-x</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597536/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: We introduce Aquila, a new approach to variant discovery in personal genomes, which is critical for uncovering the genetic contributions to health and disease. Aquila uses a reference sequence and linked-read data to generate a high quality diploid genome assembly, from which it then comprehensively detects and phases personal genetic variation. The contigs of the assemblies from our libraries cover >95% of the human reference genome, with over 98% of that in a diploid state. Thus, the assemblies support detection and accurate genotyping of the most prevalent types of human genetic variation, including single nucleotide polymorphisms (SNPs), small insertions and deletions (small indels), and structural variants (SVs), in all but the most difficult regions. All heterozygous variants are phased in blocks that can approach arm-level length. The final output of Aquila is a diploid and phased personal genome sequence, and a phased Variant Call Format (VCF) file that also contains homozygous and a few unphased heterozygous variants. Aquila represents a cost-effective approach that can be applied to cohorts for variation discovery or association studies, or to single individuals with rare phenotypes that could be caused by SVs or compound heterozygosity.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 我们介绍了天鹰座（Aquila），这是一种在个人基因组中发现变异的新方法，这对于揭示对健康和疾病的遗传贡献至关重要。天鹰座使用参考序列和链接阅读的数据来生成高质量的二倍体基因组组装体，然后从中全面检测并分阶段个人遗传变异。我们库中的程序集重叠群覆盖了人类参考基因组的95％以上，其中98％处于二倍体状态。因此，该程序集支持在最困难的区域以外的所有区域检测和准确鉴定人类遗传变异的最普遍类型，包括单核苷酸多态性（SNP），小的插入和缺失（小的indel）和结构变异（SV）。。所有杂合变体均以可接近臂级长度的块为阶段。Aquila的最终输出是一个二倍体和定相的个人基因组序列，以及一个定相的变异体调用格式（VCF）文件，该文件还包含纯合子和一些非定相的杂合子变体。天鹰座代表了一种具有成本效益的方法，可以应用于队列中的变异发现或关联研究，或应用于可能由SV或复合杂合性引起的具有罕见表型的单个个体。</p>

<h2>Pubmed_ID: 33597535</h2>

<h3>Determinants of genome-wide distribution and evolution of uORFs in eukaryotes</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21394-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597535/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Upstream open reading frames (uORFs) play widespread regulatory functions in modulating mRNA translation in eukaryotes, but the principles underlying the genomic distribution and evolution of uORFs remain poorly understood. Here, we analyze ~17 million putative canonical uORFs in 478 eukaryotic species that span most of the extant taxa of eukaryotes. We demonstrate how positive and purifying selection, coupled with differences in effective population size (Ne), has shaped the contents of uORFs in eukaryotes. Besides, gene expression level is important in influencing uORF occurrences across genes in a species. Our analyses suggest that most uORFs might play regulatory roles rather than encode functional peptides. We also show that the Kozak sequence context of uORFs has evolved across eukaryotic clades, and that noncanonical uORFs tend to have weaker suppressive effects than canonical uORFs in translation regulation. This study provides insights into the driving forces underlying uORF evolution in eukaryotes.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33597514</h2>

<h3>Expanded catalog of microbial genes and metagenome-assembled genomes from the pig gut microbiome</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21295-0</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597514/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Gut microbiota plays an important role in pig health and production. Still, availability of sequenced genomes and functional information for most pig gut microbes remains limited. Here we perform a landscape survey of the swine gut microbiome, spanning extensive sample sources by deep metagenomic sequencing resulting in an expanded gene catalog named pig integrated gene catalog (PIGC), containing 17,237,052 complete genes clustered at 90% protein identity from 787 gut metagenomes, of which 28% are unknown proteins. Using binning analysis, 6339 metagenome-assembled genomes (MAGs) were obtained, which were clustered to 2673 species-level genome bins (SGBs), among which 86% (2309) SGBs are unknown based on current databases. Using the present gene catalog and MAGs, we identified several strain-level differences between the gut microbiome of wild boars and commercial Duroc pigs. PIGC and MAGs provide expanded resources for swine gut microbiome-related research.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 肠道菌群在猪的健康和生产中起着重要作用。但是，大多数猪肠道微生物的测序基因组和功能信息的可用性仍然有限。在这里，我们对猪肠道微生物组进行了一次景观调查，通过深度宏基因组测序跨越了广泛的样本来源，形成了名为猪整合基因目录（PIGC）的扩展基因目录，其中包含来自787个肠道元基因组的17,237,052个完整基因，它们以90％的蛋白质同一性聚类，其中28％是未知蛋白质。使用分箱分析，获得了6339个元基因组组装基因组（MAG），它们被聚类到2673个物种水平的基因组箱（SGB），其中基于当前数据库未知86％（2309）个SGB。使用当前的基因目录和MAGs，我们确定了野猪和商业杜洛克猪肠道微生物组之间的几个菌株水平差异。PIGC和MAGs为猪肠道微生物组相关研究提供了扩展的资源。</p>

<h2>Pubmed_ID: 33597508</h2>

<h3>A case-only study to identify genetic modifiers of breast cancer risk for BRCA1/BRCA2 mutation carriers</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20496-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597508/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Breast cancer (BC) risk for BRCA1 and BRCA2 mutation carriers varies by genetic and familial factors. About 50 common variants have been shown to modify BC risk for mutation carriers. All but three, were identified in general population studies. Other mutation carrier-specific susceptibility variants may exist but studies of mutation carriers have so far been underpowered. We conduct a novel case-only genome-wide association study comparing genotype frequencies between 60,212 general population BC cases and 13,007 cases with BRCA1 or BRCA2 mutations. We identify robust novel associations for 2 variants with BC for BRCA1 and 3 for BRCA2 mutation carriers, P < 10-8, at 5 loci, which are not associated with risk in the general population. They include rs60882887 at 11p11.2 where MADD, SP11 and EIF1, genes previously implicated in BC biology, are predicted as potential targets. These findings will contribute towards customising BC polygenic risk scores for BRCA1 and BRCA2 mutation carriers.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33597505</h2>

<h3>Population-specific causal disease effect sizes in functionally important regions impacted by selection</h3>

<h3>受选择影响的功能重要区域中特定人群的因果疾病影响大小 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21286-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597505/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Many diseases exhibit population-specific causal effect sizes with trans-ethnic genetic correlations significantly less than 1, limiting trans-ethnic polygenic risk prediction. We develop a new method, S-LDXR, for stratifying squared trans-ethnic genetic correlation across genomic annotations, and apply S-LDXR to genome-wide summary statistics for 31 diseases and complex traits in East Asians (average N = 90K) and Europeans (average N = 267K) with an average trans-ethnic genetic correlation of 0.85. We determine that squared trans-ethnic genetic correlation is 0.82× (s.e. 0.01) depleted in the top quintile of background selection statistic, implying more population-specific causal effect sizes. Accordingly, causal effect sizes are more population-specific in functionally important regions, including conserved and regulatory regions. In regions surrounding specifically expressed genes, causal effect sizes are most population-specific for skin and immune genes, and least population-specific for brain genes. Our results could potentially be explained by stronger gene-environment interaction at loci impacted by selection, particularly positive selection.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33597294</h2>

<h3>A specific basal body linker protein provides the connection function for basal body inheritance in trypanosomes</h3>

<h3>特定的基础体连接蛋白为锥虫的基础体遗传提供连接功能 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2014040118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33597294/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Centrioles and basal bodies (CBBs) are found in physically linked pairs, and in mammalian cells intercentriole connections (G1-G2 tether and S-M linker) regulate centriole duplication and function. In trypanosomes BBs are not associated with the spindle and function in flagellum/cilia nucleation with an additional role in mitochondrial genome (kinetoplast DNA [kDNA]) segregation. Here, we describe BBLP, a BB/pro-BB (pBB) linker protein in Trypanosoma brucei predicted to be a large coiled-coil protein conserved in the kinetoplastida. Colocalization with the centriole marker SAS6 showed that BBLP localizes between the BB/pBB pair, throughout the cell cycle, with a stronger signal in the old flagellum BB/pBB pair. Importantly, RNA interference (RNAi) depletion of BBLP leads to a conspicuous splitting of the BB/pBB pair associated only with the new flagellum. BBLP RNAi is lethal in the bloodstream form of the parasite and perturbs mitochondrial kDNA inheritance. Immunogold labeling confirmed that BBLP is localized to a cytoskeletal component of the BB/pBB linker, and tagged protein induction showed that BBLP is incorporated de novo in both new and old flagella BB pairs of dividing cells. We show that the two aspects of CBB disengagement-loss of orthogonal orientation and ability to separate and move apart-are consistent but separable events in evolutionarily diverse cells and we provide a unifying model explaining centriole/BB linkage differences between such cells.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在物理连接的配对中发现了质心和基体（CBB），在哺乳动物细胞中，质心间连接（G1-G2系链和S-M接头）调节了质心的复制和功能。在锥虫体中，BB不与纺锤体相关，并在鞭毛/纤毛成核中起作用，在线粒体基因组（动塑料DNA[kDNA]）分离中具有附加作用。在这里，我们描述了BBLP，一种在布鲁氏锥虫中的BB/pro-BB（pBB）接头蛋白，预计是大的卷曲螺旋蛋白，保留在运动塑料中。与着丝粒标记SAS6的共定位显示，在整个细胞周期中，BBLP定位在BB/pBB对之间，并且在旧鞭毛BB/pBB对中具有更强的信号。重要的是，BBLP的RNA干扰（RNAi）消耗导致仅与新鞭毛相关的BB/pBB对明显分裂。BBLPRNAi在寄生虫的血流形式中具有致死性，并扰乱线粒体kDNA遗传。免疫金标记证实BBLP定位于BB/pBB接头的细胞骨架成分，标记的蛋白诱导显示BBLP从头结合在新的和旧的鞭毛BB对分裂细胞中。我们表明正交方向的CBB脱离-损失和分离和移动的能力的两个方面在进化上不同的细胞中是一致但可分离的事件，并且我们提供了一个统一的模型来解释此类细胞之间的中心/BB连锁差异。</p>

<h2>Pubmed_ID: 33595147</h2>

<h3>Developing a rapid and highly efficient cowpea regeneration, transformation and genome editing system using embryonic axis explants</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15202</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33595147/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/17/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Cowpea is one of the most important legume crops planted worldwide, but despite decades of effort, cowpea transformation is still challenging due to inefficient Agrobacterium-mediated T-DNA delivery, transgenic selection and in vitro shoot regeneration. Here, we report a highly efficient transformation system using embryonic axis explants isolated from imbibed mature seeds. We found that removal of the shoot apical meristem from the explants stimulated direct multiple shoot organogenesis from the cotyledonary node tissue. The application of a previously reported ternary transformation vector system provided efficient Agrobacterium-mediated gene delivery, while the utilization of spcN as selectable marker enabled more robust transgenic selection, plant recovery and generated transgenic plants without escapes and chimera formation. Transgenic cowpea plantlets developed exclusively from the cotyledonary nodes at high frequencies of 4 to 37% across a wide range of cowpea genotypes. CRISPR/Cas-mediated gene editing was successfully demonstrated. The transformation principles established here could also be applied to other legumes to increase transformation efficiencies. Supporting information.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33594057</h2>

<h3>Positioning of nucleosomes containing γ-H2AX precedes active DNA demethylation and transcription initiation</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21227-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33594057/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: In addition to nucleosomes, chromatin contains non-histone chromatin-associated proteins, of which the high-mobility group proteins are the most abundant. Chromatin-mediated regulation of transcription involves DNA methylation and histone modifications. However, the order of events and the precise function of high-mobility group proteins during transcription initiation remain unclear. Here we show that high-mobility group AT-hook 2 protein (HMGA2) induces DNA nicks at the transcription start site, which are required by the histone chaperone FACT complex to incorporate nucleosomes containing the histone variant H2A.X. Further, phosphorylation of H2A.X at S139 (γ-H2AX) is required for repair-mediated DNA demethylation and transcription activation. The relevance of these findings is demonstrated within the context of TGFB1 signaling and idiopathic pulmonary fibrosis, suggesting therapies against this lethal disease. Our data support the concept that chromatin opening during transcriptional initiation involves intermediates with DNA breaks that subsequently require DNA repair mechanisms to ensure genome integrity.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33594055</h2>

<h3>Analysis of metagenome-assembled viral genomes from the human gut reveals diverse putative CrAss-like phages with unique genomic features</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21350-w</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33594055/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: CrAssphage is the most abundant human-associated virus and the founding member of a large group of bacteriophages, discovered in animal-associated and environmental metagenomes, that infect bacteria of the phylum Bacteroidetes. We analyze 4907 Circular Metagenome Assembled Genomes (cMAGs) of putative viruses from human gut microbiomes and identify nearly 600 genomes of crAss-like phages that account for nearly 87% of the DNA reads mapped to these cMAGs. Phylogenetic analysis of conserved genes demonstrates the monophyly of crAss-like phages, a putative virus order, and of 5 branches, potential families within that order, two of which have not been identified previously. The phage genomes in one of these families are almost twofold larger than the crAssphage genome (145-192 kilobases), with high density of self-splicing introns and inteins. Many crAss-like phages encode suppressor tRNAs that enable read-through of UGA or UAG stop-codons, mostly, in late phage genes. A distinct feature of the crAss-like phages is the recurrent switch of the phage DNA polymerase type between A and B families. Thus, comparative genomic analysis of the expanded assemblage of crAss-like phages reveals aspects of genome architecture and expression as well as phage biology that were not apparent from the previous work on phage genomics.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33594051</h2>

<h3>Loop competition and extrusion model predicts CTCF interaction specificity</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21368-0</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33594051/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Three-dimensional chromatin looping interactions play an important role in constraining enhancer-promoter interactions and mediating transcriptional gene regulation. CTCF is thought to play a critical role in the formation of these loops, but the specificity of which CTCF binding events form loops and which do not is difficult to predict. Loops often have convergent CTCF binding site motif orientation, but this constraint alone is only weakly predictive of genome-wide interaction data. Here we present an easily interpretable and simple mathematical model of CTCF mediated loop formation which is consistent with Cohesin extrusion and can predict ChIA-PET CTCF looping interaction measurements with high accuracy. Competition between overlapping loops is a critical determinant of loop specificity. We show that this model is consistent with observed chromatin interaction frequency changes induced by CTCF binding site deletion, inversion, and mutation, and is also consistent with observed constraints on validated enhancer-promoter interactions.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33593904</h2>

<h3>Accurate SNV detection in single cells by transposon-based whole-genome amplification of complementary strands</h3>

<h3>通过基于转座子的全基因组互补链扩增，在单个细胞中进行准确的SNV检测 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2013106118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33593904/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Single-nucleotide variants (SNVs), pertinent to aging and disease, occur sporadically in the human genome, hence necessitating single-cell measurements. However, detection of single-cell SNVs suffers from false positives (FPs) due to intracellular single-stranded DNA damage and the process of whole-genome amplification (WGA). Here, we report a single-cell WGA method termed multiplexed end-tagging amplification of complementary strands (META-CS), which eliminates nearly all FPs by virtue of DNA complementarity, and achieved the highest accuracy thus far. We validated META-CS by sequencing kindred cells and human sperm, and applied it to other human tissues. Investigation of mature single human neurons revealed increasing SNVs with age and potentially unrepaired strand-specific oxidative guanine damage. We determined SNV frequencies along the genome in differentiated single human blood cells, and identified cell type-dependent mutational patterns for major types of lymphocytes.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 与衰老和疾病有关的单核苷酸变体（SNV）偶发出现在人类基因组中，因此需要进行单细胞测量。但是，由于细胞内单链DNA损伤和全基因组扩增（WGA）过程，单细胞SNV的检测会出现假阳性（FPs）。在这里，我们报道了一种称为互补链的多末端标记扩增的单细胞WGA方法（META-CS），该方法凭借DNA互补性几乎消除了所有FP，并实现了迄今为止的最高准确性。我们通过对亲缘细胞和人类精子进行测序来验证META-CS，并将其应用于其他人类组织。对成熟的单个人类神经元的研究表明，SNV随着年龄的增长而增加，并且潜在的未修复的链特异性氧化鸟嘌呤损伤。我们确定了分化的单个人类血细胞中沿基因组的SNV频率，并确定了主要淋巴细胞类型的细胞类型依赖性突变模式。</p>

<h2>Pubmed_ID: 33593897</h2>

<h3>Trichoderma reesei Rad51 tolerates mismatches in hybrid meiosis with diverse genome sequences</h3>

<h3>里氏木霉Rad51耐受具有不同基因组序列的杂交减数分裂中的错配 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2007192118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33593897/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/23/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Most eukaryotes possess two RecA-like recombinases (ubiquitous Rad51 and meiosis-specific Dmc1) to promote interhomolog recombination during meiosis. However, some eukaryotes have lost Dmc1. Given that mammalian and yeast Saccharomyces cerevisiae (Sc) Dmc1 have been shown to stabilize recombination intermediates containing mismatches better than Rad51, we used the Pezizomycotina filamentous fungus Trichoderma reesei to address if and how Rad51-only eukaryotes conduct interhomolog recombination in zygotes with high sequence heterogeneity. We applied multidisciplinary approaches (next- and third-generation sequencing technology, genetics, cytology, bioinformatics, biochemistry, and single-molecule biophysics) to show that T. reesei Rad51 (TrRad51) is indispensable for interhomolog recombination during meiosis and, like ScDmc1, TrRad51 possesses better mismatch tolerance than ScRad51 during homologous recombination. Our results also indicate that the ancestral TrRad51 evolved to acquire ScDmc1-like properties by creating multiple structural variations, including via amino acid residues in the L1 and L2 DNA-binding loops.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 大多数真核生物具有两种RecA样重组酶（普遍存在的Rad51和减数分裂特异性Dmc1），以促进减数分裂过程中的同源重组。但是，一些真核生物失去了Dmc1。考虑到哺乳动物和酵母酿酒酵母（Sc）Dmc1已显示出比Rad51更好地稳定含有错配的重组中间产物，我们使用了Pezizomycotina丝状真菌里氏木霉（Trichodermareesei）来研究仅Rad51的真核生物是否以及如何在具有高序列异质性的合子中进行同源重组。我们应用了多学科方法（下一代和第三代测序技术，遗传学，细胞学，生物信息学，生物化学和单分子生物物理学）表明里氏木霉Rad51（TrRad51）对于减数分裂过程中的同源重组是必不可少的，例如ScDmc1，在同源重组期间，TrRad51具有比ScRad51更好的错配耐受性。我们的研究结果还表明，祖先的TrRad51通过创建多个结构变异（包括通过L1和L2DNA结合环中的氨基酸残基）进化为获得类似ScDmc1的特性。</p>

<h2>Pubmed_ID: 33589841</h2>

<h3>Genome sequencing analysis identifies new loci associated with Lewy body dementia and provides insights into its genetic architecture</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41588-021-00785-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589841/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature genetics</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/15/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The genetic basis of Lewy body dementia (LBD) is not well understood. Here, we performed whole-genome sequencing in large cohorts of LBD cases and neurologically healthy controls to study the genetic architecture of this understudied form of dementia, and to generate a resource for the scientific community. Genome-wide association analysis identified five independent risk loci, whereas genome-wide gene-aggregation tests implicated mutations in the gene GBA. Genetic risk scores demonstrate that LBD shares risk profiles and pathways with Alzheimer's disease and Parkinson's disease, providing a deeper molecular understanding of the complex genetic architecture of this age-related neurodegenerative condition.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33589840</h2>

<h3>Genome-wide meta-analysis, fine-mapping and integrative prioritization implicate new Alzheimer's disease risk genes</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41588-020-00776-w</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589840/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature genetics</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/15/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genome-wide association studies have discovered numerous genomic loci associated with Alzheimer's disease (AD); yet the causal genes and variants are incompletely identified. We performed an updated genome-wide AD meta-analysis, which identified 37 risk loci, including new associations near CCDC6, TSPAN14, NCK2 and SPRED2. Using three SNP-level fine-mapping methods, we identified 21 SNPs with >50% probability each of being causally involved in AD risk and others strongly suggested by functional annotation. We followed this with colocalization analyses across 109 gene expression quantitative trait loci datasets and prioritization of genes by using protein interaction networks and tissue-specific expression. Combining this information into a quantitative score, we found that evidence converged on likely causal genes, including the above four genes, and those at previously discovered AD loci, including BIN1, APH1B, PTK2B, PILRA and CASS4.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33589828</h2>

<h3>Daily briefing: The human genome sequence, 20 years on</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/d41586-021-00413-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589828/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33589745</h2>

<h3>Genome-scale profiling of circulating cell-free DNA signatures for early detection of hepatocellular carcinoma in cirrhotic patients</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41422-020-00457-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589745/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell research</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/15/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33589637</h2>

<h3>Asymmetric introgression reveals the genetic architecture of a plumage trait</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21340-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589637/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/15/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genome-wide variation in introgression rates across hybrid zones offers a powerful opportunity for studying population differentiation. One poorly understood pattern of introgression is the geographic displacement of a trait implicated in lineage divergence from genome-wide population boundaries. While difficult to interpret, this pattern can facilitate the dissection of trait genetic architecture because traits become uncoupled from their ancestral genomic background. We studied an example of trait displacement generated by the introgression of head plumage coloration from personata to alba subspecies of the white wagtail. A previous study of their hybrid zone in Siberia revealed that the geographic transition in this sexual signal that mediates assortative mating was offset from other traits and genetic markers. Here we show that head plumage is associated with two small genetic regions. Despite having a simple genetic architecture, head plumage inheritance is consistent with partial dominance and epistasis, which could contribute to its asymmetric introgression.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33589617</h2>

<h3>PrimeDesign software for rapid and simplified design of prime editing guide RNAs</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21337-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589617/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/15/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Prime editing (PE) is a versatile genome editing technology, but design of the required guide RNAs is more complex than for standard CRISPR-based nucleases or base editors. Here we describe PrimeDesign, a user-friendly, end-to-end web application and command-line tool for the design of PE experiments. PrimeDesign can be used for single and combination editing applications, as well as genome-wide and saturation mutagenesis screens. Using PrimeDesign, we construct PrimeVar, a comprehensive and searchable database that includes candidate prime editing guide RNA (pegRNA) and nicking sgRNA (ngRNA) combinations for installing or correcting >68,500 pathogenic human genetic variants from the ClinVar database. Finally, we use PrimeDesign to design pegRNAs/ngRNAs to install a variety of human pathogenic variants in human cells.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33589603</h2>

<h3>Chromosomal coordination and differential structure of asynchronous replicating regions</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21348-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589603/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/15/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Stochastic asynchronous replication timing (AS-RT) is a phenomenon in which the time of replication of each allele is different, and the identity of the early allele varies between cells. By taking advantage of stable clonal pre-B cell populations derived from C57BL6/Castaneous mice, we have mapped the genome-wide AS-RT loci, independently of genetic differences. These regions are characterized by differential chromatin accessibility, mono-allelic expression and include new gene families involved in specifying cell identity. By combining population level mapping with single cell FISH, our data reveal the existence of a novel regulatory program that coordinates a fixed relationship between AS-RT regions on any given chromosome, with some loci set to replicate in a parallel and others set in the anti-parallel orientation. Our results show that AS-RT is a highly regulated epigenetic mark established during early embryogenesis that may be used for facilitating the programming of mono-allelic choice throughout development.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33589587</h2>

<h3>Genetic determinants of risk in autoimmune pulmonary alveolar proteinosis</h3>

<h3>自身免疫性肺泡蛋白沉着病风险的遗传决定因素 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21011-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33589587/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/15/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Pulmonary alveolar proteinosis (PAP) is a devastating lung disease caused by abnormal surfactant homeostasis, with a prevalence of 6-7 cases per million population worldwide. While mutations causing hereditary PAP have been reported, the genetic basis contributing to autoimmune PAP (aPAP) has not been thoroughly investigated. Here, we conducted a genome-wide association study of aPAP in 198 patients and 395 control participants of Japanese ancestry. The common genetic variant, rs138024423 at 6p21, in the major-histocompatibility-complex (MHC) region was significantly associated with disease risk (Odds ratio [OR] = 5.2; P = 2.4 × 10-12). HLA fine-mapping revealed that the common HLA class II allele, HLA-DRB1*08:03, strongly drove this signal (OR = 4.8; P = 4.8 × 10-12), followed by an additional independent risk allele at HLA-DPβ1 amino acid position 8 (OR = 0.28; P = 3.4 × 10-7). HLA-DRB1*08:03 was also associated with an increased level of anti-GM-CSF antibody, a key driver of the disease (β = 0.32; P = 0.035). Our study demonstrated a heritable component of aPAP, suggesting an underlying genetic predisposition toward an abnormal antibody production.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33581057</h2>

<h3>Genome engineering for crop improvement and future agriculture</h3>

<h3>基因组工程，用于作物改良和未来农业 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2021.01.005</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33581057/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Feeding the ever-growing population is a major challenge, especially in light of rapidly changing climate conditions. Genome editing is set to revolutionize plant breeding and could help secure the global food supply. Here, I review the development and application of genome editing tools in plants while highlighting newly developed techniques. I describe new plant breeding strategies based on genome editing and discuss their impact on crop production, with an emphasis on recent advancements in genome editing-based plant improvements that could not be achieved by conventional breeding. I also discuss challenges facing genome editing that must be overcome before realizing the full potential of this technology toward future crops and food production.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 养活不断增长的人口是一项重大挑战，尤其是在气候条件迅速变化的情况下。基因组编辑将彻底改变植物育种，并有助于确保全球粮食供应。在这里，我将重点介绍新开发的技术，同时回顾植物中基因组编辑工具的开发和应用。我描述了基于基因组编辑的新植物育种策略，并讨论了它们对作物生产的影响，重点介绍了传统育种无法实现的基于基因组编辑的植物改良的最新进展。我还讨论了基因组编辑面临的挑战，在实现这项技术对未来作物和粮食生产的全部潜力之前，必须克服这些挑战。</p>

<h2>Pubmed_ID: 33579965</h2>

<h3>Multiple mechanisms drive genomic adaptation to extreme O 2 levels in Drosophila melanogaster</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21281-6</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33579965/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: To detect the genomic mechanisms underlying evolutionary dynamics of adaptation in sexually reproducing organisms, we analyze multigenerational whole genome sequences of Drosophila melanogaster adapting to extreme O2 conditions over an experiment conducted for nearly two decades. We develop methods to analyze time-series genomics data and predict adaptive mechanisms. Here, we report a remarkable level of synchronicity in both hard and soft selective sweeps in replicate populations as well as the arrival of favorable de novo mutations that constitute a few asynchronized sweeps. We additionally make direct experimental observations of rare recombination events that combine multiple alleles on to a single, better-adapted haplotype. Based on the analyses of the genes in genomic intervals, we provide a deeper insight into the mechanisms of genome adaptation that allow complex organisms to survive harsh environments.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33579957</h2>

<h3>The acquisition of molecular drivers in pediatric therapy-related myeloid neoplasms</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21255-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33579957/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Pediatric therapy-related myeloid neoplasms (tMN) occur in children after exposure to cytotoxic therapy and have a dismal prognosis. The somatic and germline genomic alterations that drive these myeloid neoplasms in children and how they arise have yet to be comprehensively described. We use whole exome, whole genome, and/or RNA sequencing to characterize the genomic profile of 84 pediatric tMN cases (tMDS: n = 28, tAML: n = 56). Our data show that Ras/MAPK pathway mutations, alterations in RUNX1 or TP53, and KMT2A rearrangements are frequent somatic drivers, and we identify cases with aberrant MECOM expression secondary to enhancer hijacking. Unlike adults with tMN, we find no evidence of pre-existing minor tMN clones (including those with TP53 mutations), but rather the majority of cases are unrelated clones arising as a consequence of cytotoxic therapy. These studies also uncover rare cases of lineage switch disease rather than true secondary neoplasms.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33579945</h2>

<h3>CTCF loss has limited effects on global genome architecture in Drosophila despite critical regulatory functions</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21366-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33579945/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Vertebrate genomes are partitioned into contact domains defined by enhanced internal contact frequency and formed by two principal mechanisms: compartmentalization of transcriptionally active and inactive domains, and stalling of chromosomal loop-extruding cohesin by CTCF bound at domain boundaries. While Drosophila has widespread contact domains and CTCF, it is currently unclear whether CTCF-dependent domains exist in flies. We genetically ablate CTCF in Drosophila and examine impacts on genome folding and transcriptional regulation in the central nervous system. We find that CTCF is required to form a small fraction of all domain boundaries, while critically controlling expression patterns of certain genes and supporting nervous system function. We also find that CTCF recruits the pervasive boundary-associated factor Cp190 to CTCF-occupied boundaries and co-regulates a subset of genes near boundaries together with Cp190. These results highlight a profound difference in CTCF-requirement for genome folding in flies and vertebrates, in which a large fraction of boundaries are CTCF-dependent and suggest that CTCF has played mutable roles in genome architecture and direct gene expression control during metazoan evolution.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33579944</h2>

<h3>Chromatin accessibility of circulating CD8 + T cells predicts treatment response to PD-1 blockade in patients with gastric cancer</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21299-w</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33579944/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Although tumor genomic profiling has identified small subsets of gastric cancer (GC) patients with clinical benefit from anti-PD-1 treatment, not all responses can be explained by tumor sequencing alone. We investigate epigenetic elements responsible for the differential response to anti-PD-1 therapy by quantitatively assessing the genome-wide chromatin accessibility of circulating CD8+ T cells in patients' peripheral blood. Using an assay for transposase-accessible chromatin using sequencing (ATAC-seq), we identify unique open regions of chromatin that significantly distinguish anti-PD-1 therapy responders from non-responders. GC patients with high chromatin openness of circulating CD8+ T cells are significantly enriched in the responder group. Concordantly, patients with high chromatin openness at specific genomic positions of their circulating CD8+ T cells demonstrate significantly better survival than those with closed chromatin. Here we reveal that epigenetic characteristics of baseline CD8+ T cells can be used to identify metastatic GC patients who may benefit from anti-PD-1 therapy.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33579933</h2>

<h3>Granulovirus PK-1 kinase activity relies on a side-to-side dimerization mode centered on the regulatory αC helix</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21191-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33579933/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The life cycle of Baculoviridae family insect viruses depends on the viral protein kinase, PK-1, to phosphorylate the regulatory protein, p6.9, to induce baculoviral genome release. Here, we report the crystal structure of Cydia pomenella granulovirus PK-1, which, owing to its likely ancestral origin among host cell AGC kinases, exhibits a eukaryotic protein kinase fold. PK-1 occurs as a rigid dimer, where an antiparallel arrangement of the αC helices at the dimer core stabilizes PK-1 in a closed, active conformation. Dimerization is facilitated by C-lobe:C-lobe and N-lobe:N-lobe interactions between protomers, including the domain-swapping of an N-terminal helix that crowns a contiguous β-sheet formed by the two N-lobes. PK-1 retains a dimeric conformation in solution, which is crucial for catalytic activity. Our studies raise the prospect that parallel, side-to-side dimeric arrangements that lock kinase domains in a catalytically-active conformation could function more broadly as a regulatory mechanism among eukaryotic protein kinases.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33577114</h2>

<h3>Highly efficient multiplex editing: One-shot generation of 8x Nicotiana benthamiana and 12x Arabidopsis mutants</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15197</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33577114/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genome editing by RNA-guided nucleases, such as SpCas9, has been used in numerous different plant species. However, to what extent multiple independent loci can be targeted simultaneously by multiplexing has not been well-documented. Here, we developed a toolkit, based on a highly intron-optimized zCas9i gene, which allows assembly of nuclease constructs expressing up to 32 sgRNAs. We used this toolkit to explore the limits of multiplexing in two major model species, and report on isolation of transgene-free octuple Nicotiana benthamiana and duodecuple (12x) Arabidopsis thaliana mutant lines in a single generation (T1 and T2 , respectively). We developed novel counter-selection markers for N. benthamiana, most importantly Sl-FAST2, comparable to the well-established Arabidopsis seed fluorescence marker, and FCY-UPP, based on production of toxic 5-fluorouracyl in presence of a precursor. Targeting eight genes with an array of nine different sgRNAs and relying on FCY-UPP for selection of non-transgenic T1 , we identified N. benthamiana mutant lines with astonishingly high efficiencies: All analyzed plants carried mutations in all genes (~112/116 target sites edited). Furthermore, we targeted 12 genes by an array of 24 sgRNAs in A. thaliana. Efficiency was significantly lower in A. thaliana, and our results indicate Cas9 availability is the limiting factor in such higher order multiplexing applications. We identify a duodecuple mutant line by a combination of phenotypic screening and amplicon sequencing. The resources and results presented provide new perspectives for how multiplexing can be used to generate complex genotypes or to functionally interrogate groups of candidate genes.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33576059</h2>

<h3>A protein kinase-major sperm protein gene hijacked by a necrotrophic fungal pathogen triggers disease susceptibility in wheat</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15194</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33576059/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Septoria nodorum blotch (SNB), a disease caused by the necrotrophic fungal pathogen Parastagonospora nodorum, is a threat to wheat production worldwide. Multiple inverse gene-for-gene interactions involving the recognition of necrotrophic effectors (NEs) by wheat sensitivity genes play major roles in causing SNB. One interaction involves the wheat gene Snn3 and the P. nodorum NE SnTox3. Here, we used a map-based strategy to clone the Snn3-D1 gene from Aegilops tauschii, the D-genome progenitor of common wheat. Snn3-D1 contained protein kinase and major sperm protein domains, both of which were essential for function as confirmed by mutagenesis. As opposed to other characterized interactions in this pathosystem, a compatible Snn3-D1-SnTox3 interaction was light independent, and Snn3-D1 transcriptional expression was down-regulated by light and up-regulated by darkness. Snn3-D1 likely emerged in Ae. tauschii due to a ~218 kb insertion that occurred along the west bank of the Caspian Sea. The identification of this new class of NE sensitivity gene combined with the previously cloned sensitivity genes demonstrates that P. nodorum can take advantage of diverse host targets to trigger SNB susceptibility in wheat.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33574602</h2>

<h3>Liquid chromatin Hi-C characterizes compartment-dependent chromatin interaction dynamics</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41588-021-00784-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33574602/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature genetics</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Nuclear compartmentalization of active and inactive chromatin is thought to occur through microphase separation mediated by interactions between loci of similar type. The nature and dynamics of these interactions are not known. We developed liquid chromatin Hi-C to map the stability of associations between loci. Before fixation and Hi-C, chromosomes are fragmented, which removes strong polymeric constraint, enabling detection of intrinsic locus-locus interaction stabilities. Compartmentalization is stable when fragments are larger than 10-25 kb. Fragmentation of chromatin into pieces smaller than 6 kb leads to gradual loss of genome organization. Lamin-associated domains are most stable, whereas interactions for speckle- and polycomb-associated loci are more dynamic. Cohesin-mediated loops dissolve after fragmentation. Liquid chromatin Hi-C provides a genome-wide view of chromosome interaction dynamics.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33574592</h2>

<h3>Daily briefing: Scientists struggling to keep human genome data free</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/d41586-021-00378-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33574592/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/10/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33574281</h2>

<h3>A genome-wide CRISPR screen identifies host factors that regulate SARS-CoV-2 entry</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21213-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33574281/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The global spread of SARS-CoV-2 is posing major public health challenges. One feature of SARS-CoV-2 spike protein is the insertion of multi-basic residues at the S1/S2 subunit cleavage site. Here, we find that the virus with intact spike (Sfull) preferentially enters cells via fusion at the plasma membrane, whereas a clone (Sdel) with deletion disrupting the multi-basic S1/S2 site utilizes an endosomal entry pathway. Using Sdel as model, we perform a genome-wide CRISPR screen and identify several endosomal entry-specific regulators. Experimental validation of hits from the CRISPR screen shows that host factors regulating the surface expression of angiotensin-converting enzyme 2 (ACE2) affect entry of Sfull virus. Animal-to-animal transmission with the Sdel virus is reduced compared to Sfull in the hamster model. These findings highlight the critical role of the S1/S2 boundary of SARS-CoV-2 spike protein in modulating virus entry and transmission and provide insights into entry of coronaviruses.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33574268</h2>

<h3>A membrane-bound ankyrin repeat protein confers race-specific leaf rust disease resistance in wheat</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20777-x</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33574268/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Plasma membrane-associated and intracellular proteins and protein complexes play a pivotal role in pathogen recognition and disease resistance signaling in plants and animals. The two predominant protein families perceiving plant pathogens are receptor-like kinases and nucleotide binding-leucine-rich repeat receptors (NLR), which often confer race-specific resistance. Leaf rust is one of the most prevalent and most devastating wheat diseases. Here, we clone the race-specific leaf rust resistance gene Lr14a from hexaploid wheat. The cloning of Lr14a is aided by the recently published genome assembly of ArinaLrFor, an Lr14a-containing wheat line. Lr14a encodes a membrane-localized protein containing twelve ankyrin (ANK) repeats and structural similarities to Ca2+-permeable non-selective cation channels. Transcriptome analyses reveal an induction of genes associated with calcium ion binding in the presence of Lr14a. Haplotype analyses indicate that Lr14a-containing chromosome segments were introgressed multiple times into the bread wheat gene pool, but we find no variation in the Lr14a coding sequence itself. Our work demonstrates the involvement of an ANK-transmembrane (TM)-like type of gene family in race-specific disease resistance in wheat. This forms the basis to explore ANK-TM-like genes in disease resistance breeding.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33574239</h2>

<h3>GWAS for autoimmune Addison's disease identifies multiple risk loci and highlights AIRE in disease susceptibility</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21015-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33574239/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Autoimmune Addison's disease (AAD) is characterized by the autoimmune destruction of the adrenal cortex. Low prevalence and complex inheritance have long hindered successful genetic studies. We here report the first genome-wide association study on AAD, which identifies nine independent risk loci (P < 5 × 10-8). In addition to loci implicated in lymphocyte function and development shared with other autoimmune diseases such as HLA, BACH2, PTPN22 and CTLA4, we associate two protein-coding alterations in Autoimmune Regulator (AIRE) with AAD. The strongest, p.R471C (rs74203920, OR = 3.4 (2.7-4.3), P = 9.0 × 10-25) introduces an additional cysteine residue in the zinc-finger motif of the second PHD domain of the AIRE protein. This unbiased elucidation of the genetic contribution to development of AAD points to the importance of central immunological tolerance, and explains 35-41% of heritability (h2).</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33574182</h2>

<h3>Reintroduction of the archaic variant of NOVA1 in cortical organoids alters neurodevelopment</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1126/science.aax2537</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33574182/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Science (New York, N.Y.)</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The evolutionarily conserved splicing regulator neuro-oncological ventral antigen 1 (NOVA1) plays a key role in neural development and function. NOVA1 also includes a protein-coding difference between the modern human genome and Neanderthal and Denisovan genomes. To investigate the functional importance of an amino acid change in humans, we reintroduced the archaic allele into human induced pluripotent cells using genome editing and then followed their neural development through cortical organoids. This modification promoted slower development and higher surface complexity in cortical organoids with the archaic version of NOVA1 Moreover, levels of synaptic markers and synaptic protein coassociations correlated with altered electrophysiological properties in organoids expressing the archaic variant. Our results suggest that the human-specific substitution in NOVA1, which is exclusive to modern humans since divergence from Neanderthals, may have had functional consequences for our species' evolution.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33574059</h2>

<h3>The genomics of ecological flexibility, large brains, and long lives in capuchin monkeys revealed with fecalFACS</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2010632118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33574059/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Ecological flexibility, extended lifespans, and large brains have long intrigued evolutionary biologists, and comparative genomics offers an efficient and effective tool for generating new insights into the evolution of such traits. Studies of capuchin monkeys are particularly well situated to shed light on the selective pressures and genetic underpinnings of local adaptation to diverse habitats, longevity, and brain development. Distributed widely across Central and South America, they are inventive and extractive foragers, known for their sensorimotor intelligence. Capuchins have among the largest relative brain size of any monkey and a lifespan that exceeds 50 y, despite their small (3 to 5 kg) body size. We assemble and annotate a de novo reference genome for Cebus imitator Through high-depth sequencing of DNA derived from blood, various tissues, and feces via fluorescence-activated cell sorting (fecalFACS) to isolate monkey epithelial cells, we compared genomes of capuchin populations from tropical dry forests and lowland rainforests and identified population divergence in genes involved in water balance, kidney function, and metabolism. Through a comparative genomics approach spanning a wide diversity of mammals, we identified genes under positive selection associated with longevity and brain development. Additionally, we provide a technological advancement in the use of noninvasive genomics for studies of free-ranging mammals. Our intra- and interspecific comparative study of capuchin genomics provides insights into processes underlying local adaptation to diverse and physiologically challenging environments, as well as the molecular basis of brain evolution and longevity.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33571428</h2>

<h3>Cone-shaped HIV-1 capsids are transported through intact nuclear pores</h3>

<h3>圆锥形的HIV-1衣壳通过完整的核孔运输 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2021.01.025</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33571428/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/18/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Human immunodeficiency virus (HIV-1) remains a major health threat. Viral capsid uncoating and nuclear import of the viral genome are critical for productive infection. The size of the HIV-1 capsid is generally believed to exceed the diameter of the nuclear pore complex (NPC), indicating that capsid uncoating has to occur prior to nuclear import. Here, we combined correlative light and electron microscopy with subtomogram averaging to capture the structural status of reverse transcription-competent HIV-1 complexes in infected T cells. We demonstrated that the diameter of the NPC in cellulo is sufficient for the import of apparently intact, cone-shaped capsids. Subsequent to nuclear import, we detected disrupted and empty capsid fragments, indicating that uncoating of the replication complex occurs by breaking the capsid open, and not by disassembly into individual subunits. Our data directly visualize a key step in HIV-1 replication and enhance our mechanistic understanding of the viral life cycle.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 人类免疫缺陷病毒（HIV-1）仍然是主要的健康威胁。病毒衣壳的脱壳和病毒基因组的核导入对于生产性感染至关重要。一般认为，HIV-1衣壳的大小超过核孔复合体（NPC）的直径，表明衣壳必须在核输入之前进行脱壳。在这里，我们结合相关的光学和电子显微镜与子图平均，以捕获感染的T细胞中具有逆转录能力的HIV-1复合物的结构状态。我们证明了纤维素中NPC的直径足以进口看似完整的圆锥形衣壳。核导入后，我们检测到了被破坏的空衣壳片段，这表明复制复合物的脱壳是通过打开衣壳而不是分解成单个亚基而发生的。我们的数据直接显示了HIV-1复制的关键步骤，并增强了我们对病毒生命周期的机械理解。</p>

<h2>Pubmed_ID: 33571105</h2>

<h3>The origin and early spread of SARS-CoV-2 in Europe</h3>

<h3>SARS-CoV-2在欧洲的起源和早期传播 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2012008118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33571105/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 03/02/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The investigation of migratory patterns during the SARS-CoV-2 pandemic before spring 2020 border closures in Europe is a crucial first step toward an in-depth evaluation of border closure policies. Here we analyze viral genome sequences using a phylodynamic model with geographic structure to estimate the origin and spread of SARS-CoV-2 in Europe prior to border closures. Based on SARS-CoV-2 genomes, we reconstruct a partial transmission tree of the early pandemic and coinfer the geographic location of ancestral lineages as well as the number of migration events into and between European regions. We find that the predominant lineage spreading in Europe during this time has a most recent common ancestor in Italy and was probably seeded by a transmission event in either Hubei, China or Germany. We do not find evidence for preferential migration paths from Hubei into different European regions or from each European region to the others. Sustained local transmission is first evident in Italy and then shortly thereafter in the other European regions considered. Before the first border closures in Europe, we estimate that the rate of occurrence of new cases from within-country transmission was within the bounds of the estimated rate of new cases from migration. In summary, our analysis offers a view on the early state of the epidemic in Europe and on migration patterns of the virus before border closures. This information will enable further study of the necessity and timeliness of border closures.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在2020年春季关闭欧洲边界之前，对SARS-CoV-2大流行期间的迁徙方式进行调查是迈向深入评估边界关闭政策的关键的第一步。在这里，我们使用具有地理结构的系统动力学模型分析病毒基因组序列，以评估边境封闭前SARS-CoV-2在欧洲的起源和传播。基于SARS-CoV-2基因组，我们构建了早期大流行的部分传播树，并提出了祖传世系的地理位置以及进出欧洲地区和在欧洲地区之间的迁移事件的数量。我们发现，这段时期在欧洲传播的主要血统在意大利具有最近的共同祖先，可能是由于湖北，中国或德国的传播事件而引起的。我们没有证据表明从湖北到欧洲不同地区或从每个欧洲地区到其他地区的优先移民途径。持续的本地传播首先在意大利出现，然后不久在其他所考虑的欧洲地区出现。在欧洲首次关闭边境之前，我们估计来自国内传输的新病例发生率处于估计的来自移民的新病例率的范围之内。总而言之，我们的分析提供了欧洲流行病的早期状态以及边境关闭前病毒的迁移方式的观点。这些信息将使人们能够进一步研究关闭边境的必要性和及时性。</p>

<h2>Pubmed_ID: 33568816</h2>

<h3>Non-coding deletions identify Maenli lncRNA as a limb-specific En1 regulator</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-021-03208-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33568816/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/10/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Long non-coding RNAs (lncRNAs) can be important components in gene-regulatory networks1, but the exact nature and extent of their involvement in human Mendelian disease is largely unknown. Here we show that genetic ablation of a lncRNA locus on human chromosome 2 causes a severe congenital limb malformation. We identified homozygous 27-63-kilobase deletions located 300 kilobases upstream of the engrailed-1 gene (EN1) in patients with a complex limb malformation featuring mesomelic shortening, syndactyly and ventral nails (dorsal dimelia). Re-engineering of the human deletions in mice resulted in a complete loss of En1 expression in the limb and a double dorsal-limb phenotype that recapitulates the human disease phenotype. Genome-wide transcriptome analysis in the developing mouse limb revealed a four-exon-long non-coding transcript within the deleted region, which we named Maenli. Functional dissection of the Maenli locus showed that its transcriptional activity is required for limb-specific En1 activation in cis, thereby fine-tuning the gene-regulatory networks controlling dorso-ventral polarity in the developing limb bud. Its loss results in the En1-related dorsal ventral limb phenotype, a subset of the full En1-associated phenotype. Our findings demonstrate that mutations involving lncRNA loci can result in human Mendelian disease.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33568662</h2>

<h3>Genetic determinants of daytime napping and effects on cardiometabolic health</h3>

<h3>白天午睡的遗传决定因素及其对心脏代谢健康的影响 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20585-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33568662/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/10/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Daytime napping is a common, heritable behavior, but its genetic basis and causal relationship with cardiometabolic health remain unclear. Here, we perform a genome-wide association study of self-reported daytime napping in the UK Biobank (n = 452,633) and identify 123 loci of which 61 replicate in the 23andMe research cohort (n = 541,333). Findings include missense variants in established drug targets for sleep disorders (HCRTR1, HCRTR2), genes with roles in arousal (TRPC6, PNOC), and genes suggesting an obesity-hypersomnolence pathway (PNOC, PATJ). Association signals are concordant with accelerometer-measured daytime inactivity duration and 33 loci colocalize with loci for other sleep phenotypes. Cluster analysis identifies three distinct clusters of nap-promoting mechanisms with heterogeneous associations with cardiometabolic outcomes. Mendelian randomization shows potential causal links between more frequent daytime napping and higher blood pressure and waist circumference.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 白天打apping是一种常见的可遗传行为，但其遗传基础和与心脏代谢健康之间的因果关系仍不清楚。在这里，我们在UKBiobank（n=452,633）中进行了自我报告的日间午睡的全基因组关联研究，并确定了23andMe研究队列（n=541,333）中的123个基因座。发现的结果包括已建立的睡眠障碍药物靶标（HCRTR1，HCRTR2）的错义变体，在唤醒中起作用的基因（TRPC6，PNOC）和暗示肥胖症-性高通透性途径的基因（PNOC，PATJ）。关联信号与加速度计测得的白天不活动时间一致，并且33个基因座与其他睡眠表型的基因座共定位。聚类分析确定了三个不同的午睡促进机制聚类，它们与心脏代谢的结果之间存在异质关联。孟德尔随机化显示日间午睡次数增加与血压和腰围较高之间存在潜在的因果关系。</p>

<h2>Pubmed_ID: 33568653</h2>

<h3>A pediatric brain tumor atlas of genes deregulated by somatic genomic rearrangement</h3>

<h3>儿科脑肿瘤基因图谱的体细胞基因组重排失控 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21081-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33568653/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/10/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The global impact of somatic structural variants (SSVs) on gene expression in pediatric brain tumors has not been thoroughly characterised. Here, using whole-genome and RNA sequencing from 854 tumors of more than 30 different types from the Children's Brain Tumor Tissue Consortium, we report the altered expression of hundreds of genes in association with the presence of nearby SSV breakpoints. SSV-mediated expression changes involve gene fusions, altered cis-regulation, or gene disruption. SSVs considerably extend the numbers of patients with tumors somatically altered for critical pathways, including receptor tyrosine kinases (KRAS, MET, EGFR, NF1), Rb pathway (CDK4), TERT, MYC family (MYC, MYCN, MYB), and HIPPO (NF2). Compared to initial tumors, progressive or recurrent tumors involve a distinct set of SSV-gene associations. High overall SSV burden associates with TP53 mutations, histone H3.3 gene H3F3C mutations, and the transcription of DNA damage response genes. Compared to adult cancers, pediatric brain tumors would involve a different set of genes with SSV-altered cis-regulation. Our comprehensive and pan-histology genomic analyses reveal SSVs to play a major role in shaping the transcriptome of pediatric brain tumors.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 体细胞结构变体（SSV）对小儿脑肿瘤中基因表达的全球影响尚未得到充分表征。在这里，我们使用来自儿童脑肿瘤组织联盟的854种30多种不同类型肿瘤的全基因组和RNA测序，我们报告了数百种基因的表达发生改变，并伴随着附近的SSV断点的存在。SSV介导的表达变化涉及基因融合，顺式调控改变或基因破坏。SSV大大增加了因关键途径而发生体细胞改变的患者人数，包括受体酪氨酸激酶（KRAS，MET，EGFR，NF1），Rb途径（CDK4），TERT，MYC家族（MYC，MYCN，MYB）和HIPPO（NF2）。与最初的肿瘤相比，进行性或复发性肿瘤涉及一组独特的SSV基因关联。总的SSV负担高与TP53突变，组蛋白H3.3基因H3F3C突变以及DNA损伤反应基因的转录有关。与成人癌症相比，小儿脑肿瘤将涉及一组具有SSV改变的顺式调控的基因。我们全面的和泛组织学的基因组分析表明，SSV在塑造小儿脑肿瘤的转录组中起主要作用。</p>

<h2>Pubmed_ID: 33568644</h2>

<h3>Nascent RNA sequencing identifies a widespread sigma70-dependent pausing regulated by Gre factors in bacteria</h3>

<h3>新生RNA测序可确定细菌中Gre因子调控的广泛依赖sigma70的暂停 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21150-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33568644/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/10/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Promoter-proximal pausing regulates eukaryotic gene expression and serves as checkpoints to assemble elongation/splicing machinery. Little is known how broadly this type of pausing regulates transcription in bacteria. We apply nascent elongating transcript sequencing combined with RNase I footprinting for genome-wide analysis of σ70-dependent transcription pauses in Escherichia coli. Retention of σ70 induces strong backtracked pauses at a 10-20-bp distance from many promoters. The pauses in the 10-15-bp register of the promoter are dictated by the canonical -10 element, 6-7 nt spacer and "YR+1Y" motif centered at the transcription start site. The promoters for the pauses in the 16-20-bp register contain an additional -10-like sequence recognized by σ70. Our in vitro analysis reveals that DNA scrunching is involved in these pauses relieved by Gre cleavage factors. The genes coding for transcription factors are enriched in these pauses, suggesting that σ70 and Gre proteins regulate transcription in response to changing environmental cues.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 启动子近端暂停调节真核基因表达，并充当组装延伸/剪接机制的检查点。很少有人知道这种暂停在多大程度上调节细菌的转录。我们将新生的伸长转录本测序与RNaseI足迹相结合，用于大肠埃希氏菌中σ70依赖性转录暂停的全基因组分析。保留σ70会导致距许多启动子10-20bp的距离处产生强烈的回溯停顿。启动子的10-15-bp寄存器中的停顿由规范的-10元件，6-7nt间隔子和以转录起始位点为中心的“YR+1Y”基元决定。16-20bp寄存器中的暂停启动子包含一个由σ70识别的额外的-10-like序列。我们的体外分析表明，DNA收缩与Gre裂解因子缓解的这些停顿有关。编码转录因子的基因在这些暂停中富集，表明σ70和Gre蛋白响应环境信号的变化调节转录。</p>

<h2>Pubmed_ID: 33568532</h2>

<h3>A switch to feeding on cycads generates parallel accelerated evolution of toxin tolerance in two clades of Eumaeus caterpillars (Lepidoptera: Lycaenidae)</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2018965118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33568532/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: We assembled a complete reference genome of Eumaeus atala, an aposematic cycad-eating hairstreak butterfly that suffered near extinction in the United States in the last century. Based on an analysis of genomic sequences of Eumaeus and 19 representative genera, the closest relatives of Eumaeus are Theorema and Mithras We report natural history information for Eumaeus, Theorema, and Mithras Using genomic sequences for each species of Eumaeus, Theorema, and Mithras (and three outgroups), we trace the evolution of cycad feeding, coloration, gregarious behavior, and other traits. The switch to feeding on cycads and to conspicuous coloration was accompanied by little genomic change. Soon after its origin, Eumaeus split into two fast evolving lineages, instead of forming a clump of close relatives in the phylogenetic tree. Significant overlap of the fast evolving proteins in both clades indicates parallel evolution. The functions of the fast evolving proteins suggest that the caterpillars developed tolerance to cycad toxins with a range of mechanisms including autophagy of damaged cells, removal of cell debris by macrophages, and more active cell proliferation.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33563994</h2>

<h3>Programmable human histone phosphorylation and gene activation using a CRISPR/Cas9-based chromatin kinase</h3>

<h3>使用基于CRISPR / Cas9的染色质激酶进行可编程的人类组蛋白磷酸化和基因激活 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21188-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33563994/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Histone phosphorylation is a ubiquitous post-translational modification that allows eukaryotic cells to rapidly respond to environmental stimuli. Despite correlative evidence linking histone phosphorylation to changes in gene expression, establishing the causal role of this key epigenomic modification at diverse loci within native chromatin has been hampered by a lack of technologies enabling robust, locus-specific deposition of endogenous histone phosphorylation. To address this technological gap, here we build a programmable chromatin kinase, called dCas9-dMSK1, by directly fusing nuclease-null CRISPR/Cas9 to a hyperactive, truncated variant of the human MSK1 histone kinase. Targeting dCas9-dMSK1 to human promoters results in increased target histone phosphorylation and gene activation and demonstrates that hyperphosphorylation of histone H3 serine 28 (H3S28ph) in particular plays a causal role in the transactivation of human promoters. In addition, we uncover mediators of resistance to the BRAF V600E inhibitor PLX-4720 in human melanoma cells using genome-scale screening with dCas9-dMSK1. Collectively, our findings enable a facile way to reshape human chromatin using CRISPR/Cas9-based epigenome editing and further define the causal link between histone phosphorylation and human gene activation.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 组蛋白磷酸化是一种普遍存在的翻译后修饰，可让真核细胞对环境刺激做出快速反应。尽管有相关证据表明组蛋白磷酸化与基因表达的变化有关，但由于缺乏能够使内源性组蛋白磷酸化稳定，特定于位点的技术的存在，阻碍了在关键基因组修饰在天然染色质内的不同基因位点上的因果作用。为了解决这个技术差距，在这里我们通过将无核酸酶的CRISPR/Cas9直接融合到人MSK1组蛋白激酶的高活性，截短的变体中，构建了一种称为dCas9-dMSK1的可编程染色质激酶。将dCas9-dMSK1靶向人类启动子会导致靶组蛋白磷酸化和基因激活增加，并证明组蛋白H3丝氨酸28（H3S28ph）的过度磷酸化在人类启动子的反式激活中尤其起着因果作用。此外，我们使用dCas9-dMSK1基因组规模筛选发现了人类黑色素瘤细胞对BRAFV600E抑制剂PLX-4720的抗性介体。总而言之，我们的发现为使用基于CRISPR/Cas9的表观基因组编辑重塑人类染色质提供了一种简便的方法，并进一步定义了组蛋白磷酸化与人类基因激活之间的因果关系。</p>

<h2>Pubmed_ID: 33563981</h2>

<h3>L1 retrotransposons exploit RNA m 6 A modification as an evolutionary driving force</h3>

<h3>L1反转录转座子利用RNA m 6 A修饰作为进化驱动力 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21197-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33563981/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: L1 retrotransposons can pose a threat to genome integrity. The host has evolved to restrict L1 replication. However, mechanisms underlying L1 propagation out of the host surveillance remains unclear. Here, we propose an evolutionary survival strategy of L1, which exploits RNA m6A modification. We discover that m6A 'writer' METTL3 facilitates L1 retrotransposition, whereas m6A 'eraser' ALKBH5 suppresses it. The essential m6A cluster that is located on L1 5' UTR serves as a docking site for eukaryotic initiation factor 3 (eIF3), enhances translational efficiency and promotes the formation of L1 ribonucleoprotein. Furthermore, through the comparative analysis of human- and primate-specific L1 lineages, we find that the most functional m6A motif-containing L1s have been positively selected and became a distinctive feature of evolutionarily young L1s. Thus, our findings demonstrate that L1 retrotransposons hijack the RNA m6A modification system for their successful replication.</p>

<p style="line-height:1.8;margin:20px"> 摘要: L1逆转座子可能对基因组完整性构成威胁。主机已发展为限制L1复制。但是，L1传播出宿主监视的基础机制仍不清楚。在这里，我们提出了一种利用RNAm6A修饰的L1进化生存策略。我们发现，m6A写入器METTL3有助于L1逆转置，而m6A擦除器ALKBH5抑制了它。位于L15UTR上的必需m6A簇充当真核起始因子3（eIF3）的停靠位点，提高翻译效率并促进L1核糖核蛋白的形成。此外，通过对人类和灵长类动物特定L1谱系的比较分析，我们发现含有功能性最强的m6A基序的L1已被积极选择，并成为进化上年轻的L1的显着特征。因此，我们的发现表明L1逆转座子劫持了RNAm6A修饰系统以使其成功复制。</p>

<h2>Pubmed_ID: 33563978</h2>

<h3>Evidence for SARS-CoV-2 related coronaviruses circulating in bats and pangolins in Southeast Asia</h3>

<h3>在东南亚的蝙蝠和穿山甲中传播与SARS-CoV-2相关的冠状病毒的证据 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21240-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33563978/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Among the many questions unanswered for the COVID-19 pandemic are the origin of SARS-CoV-2 and the potential role of intermediate animal host(s) in the early animal-to-human transmission. The discovery of RaTG13 bat coronavirus in China suggested a high probability of a bat origin. Here we report molecular and serological evidence of SARS-CoV-2 related coronaviruses (SC2r-CoVs) actively circulating in bats in Southeast Asia. Whole genome sequences were obtained from five independent bats (Rhinolophus acuminatus) in a Thai cave yielding a single isolate (named RacCS203) which is most related to the RmYN02 isolate found in Rhinolophus malayanus in Yunnan, China. SARS-CoV-2 neutralizing antibodies were also detected in bats of the same colony and in a pangolin at a wildlife checkpoint in Southern Thailand. Antisera raised against the receptor binding domain (RBD) of RmYN02 was able to cross-neutralize SARS-CoV-2 despite the fact that the RBD of RacCS203 or RmYN02 failed to bind ACE2. Although the origin of the virus remains unresolved, our study extended the geographic distribution of genetically diverse SC2r-CoVs from Japan and China to Thailand over a 4800-km range. Cross-border surveillance is urgently needed to find the immediate progenitor virus of SARS-CoV-2.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 关于SAV-CoV-2的起源以及中间动物宿主在早期动物向人类传播中的潜在作用，其中许多尚未解决的COVID-19大流行问题都没有得到解决。在中国发现的RaTG13蝙蝠冠状病毒很可能是蝙蝠起源的原因。在这里，我们报告了SARS-CoV-2相关冠状病毒（SC2r-CoVs）在东南亚蝙蝠中活跃循环的分子和血清学证据。从泰国洞穴中的五个独立蝙蝠（Rhinolophusacuminatus）获得了完整的基因组序列，产生了一个分离株（命名为RacCS203），该分离株与在中国云南马来犀中发现的RmYN02分离株最相关。在泰国南部野生动物检查站的同一殖民地的蝙蝠和穿山甲中也检测到SARS-CoV-2中和抗体。尽管RacCS203或RmYN02的RBD未能结合ACE2，但针对RmYN02的受体结合域（RBD）产生的抗血清却能够交叉中和SARS-CoV-2。尽管该病毒的起源尚未解决，但我们的研究将遗传多样性SC2r-CoV的地理分布从日本和中国扩展到了4800公里范围内的泰国。迫切需要进行跨境监视，以找到SARS-CoV-2的直接祖细胞病毒。</p>

<h2>Pubmed_ID: 33563969</h2>

<h3>Live-cell single particle tracking of PRC1 reveals a highly dynamic system with low target site occupancy</h3>

<h3>PRC1的活细胞单粒子跟踪揭示了目标位置占用率低的高度动态系统 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21130-6</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33563969/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Polycomb repressive complex 1 (PRC1) is an essential chromatin-based repressor of gene transcription. How PRC1 engages with chromatin to identify its target genes and achieve gene repression remains poorly defined, representing a major hurdle to our understanding of Polycomb system function. Here, we use genome engineering and single particle tracking to dissect how PRC1 binds to chromatin in live mouse embryonic stem cells. We observe that PRC1 is highly dynamic, with only a small fraction stably interacting with chromatin. By integrating subunit-specific dynamics, chromatin binding, and abundance measurements, we discover that PRC1 exhibits low occupancy at target sites. Furthermore, we employ perturbation approaches to uncover how specific components of PRC1 define its kinetics and chromatin binding. Together, these discoveries provide a quantitative understanding of chromatin binding by PRC1 in live cells, suggesting that chromatin modification, as opposed to PRC1 complex occupancy, is central to gene repression.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 聚梳阻遏复合物1（PRC1）是基因转录中基于染色质的必需阻遏物。PRC1如何与染色质结合以鉴定其靶基因并实现基因阻抑的方法仍然不清楚，这代表了我们对Polycomb系统功能的理解的主要障碍。在这里，我们使用基因组工程和单粒子跟踪来剖析PRC1如何与活小鼠胚胎干细胞中的染色质结合。我们观察到PRC1是高度动态的，只有一小部分与染色质稳定相互作用。通过整合特定于亚基的动力学，染色质结合和丰度测量，我们发现PRC1在目标站点展示低的占有率。此外，我们采用扰动方法来揭示PRC1的特定成分如何定义其动力学和染色质结合。总之，这些发现提供了对PRC1在活细胞中与染色质结合的定量理解，表明与PRC1复合物占用相反，染色质修饰是基因抑制的关键。</p>

<h2>Pubmed_ID: 33563757</h2>

<h3>Deficiency of the novel high mobility group protein HMGXB4 protects against systemic inflammation-induced endotoxemia in mice</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2021862118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33563757/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Sepsis is a major cause of mortality in intensive care units, which results from a severely dysregulated inflammatory response that ultimately leads to organ failure. While antibiotics can help in the early stages, effective strategies to curtail inflammation remain limited. The high mobility group (HMG) proteins are chromosomal proteins with important roles in regulating gene transcription. While HMGB1 has been shown to play a role in sepsis, the role of other family members including HMGXB4 remains unknown. We found that expression of HMGXB4 is strongly induced in response to lipopolysaccharide (LPS)-elicited inflammation in murine peritoneal macrophages. Genetic deletion of Hmgxb4 protected against LPS-induced lung injury and lethality and cecal ligation and puncture (CLP)-induced lethality in mice, and attenuated LPS-induced proinflammatory gene expression in cultured macrophages. By integrating genome-wide transcriptome profiling and a publicly available ChIP-seq dataset, we identified HMGXB4 as a transcriptional activator that regulates the expression of the proinflammatory gene, Nos2 (inducible nitric oxide synthase 2) by binding to its promoter region, leading to NOS2 induction and excessive NO production and tissue damage. Similar to Hmgxb4 ablation in mice, administration of a pharmacological inhibitor of NOS2 robustly decreased LPS-induced pulmonary vascular permeability and lethality in mice. Additionally, we identified the cell adhesion molecule, ICAM1, as a target of HMGXB4 in endothelial cells that facilitates inflammation by promoting monocyte attachment. In summary, our study reveals a critical role of HMGXB4 in exacerbating endotoxemia via transcriptional induction of Nos2 and Icam1 gene expression and thus targeting HMGXB4 may be an effective therapeutic strategy for the treatment of sepsis.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33558760</h2>

<h3>Acute BAF perturbation causes immediate changes in chromatin accessibility</h3>

<h3>急性BAF扰动会导致染色质可及性立即变化 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41588-021-00777-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33558760/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature genetics</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Cancer-associated, loss-of-function mutations in genes encoding subunits of the BRG1/BRM-associated factor (BAF) chromatin-remodeling complexes1-8 often cause drastic chromatin accessibility changes, especially in important regulatory regions9-19. However, it remains unknown how these changes are established over time (for example, immediate consequences or long-term adaptations), and whether they are causative for intracomplex synthetic lethalities, abrogating the formation or activity of BAF complexes9,20-24. In the present study, we use the dTAG system to induce acute degradation of BAF subunits and show that chromatin alterations are established faster than the duration of one cell cycle. Using a pharmacological inhibitor and a chemical degrader of the BAF complex ATPase subunits25,26, we show that maintaining genome accessibility requires constant ATP-dependent remodeling. Completely abolishing BAF complex function by acute degradation of a synthetic lethal subunit in a paralog-deficient background results in an almost complete loss of chromatin accessibility at BAF-controlled sites, especially also at superenhancers, providing a mechanism for intracomplex synthetic lethalities.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 编码BRG1/BRM相关因子（BAF）染色质重塑复合物1-8的亚基的基因中与癌症相关的功能丧失突变通常会导致严重的染色质可及性变化，尤其是在重要的调控区域9-19中。然而，仍不清楚如何随时间建立这些变化（例如，直接后果或长期适应），以及它们是否对复合体内合成杀伤力起因，从而消除了BAF复合物的形成或活性9,20-24。在本研究中，我们使用dTAG系统诱导BAF亚基的急性降解，并证明染色质改变的建立速度快于一个细胞周期的持续时间。使用BAF复合ATPase亚基25,26的药理抑制剂和化学降解剂，我们表明维持基因组可及性需要恒定的ATP依赖性重塑。在同源物缺乏的背景下，合成致死亚基的急剧降解会完全废除BAF复杂功能，导致BAF控制位点（特别是在超级增强子处）几乎完全丧失了染色质可及性，从而提供了内部复杂合成杀伤力的机制。</p>

<h2>Pubmed_ID: 33558551</h2>

<h3>Publisher Correction: Genome-wide analyses of behavioural traits are subject to bias by misreports and longitudinal changes</h3>

<h3>出版商更正：全基因组的行为特征分析可能会因错误报道和纵向变化而产生偏差 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21294-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33558551/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 没有摘要</p>

<h2>Pubmed_ID: 33558518</h2>

<h3>Genome-wide fine-mapping identifies pleiotropic and functional variants that predict many traits across global cattle populations</h3>

<h3>全基因组范围的精细定位识别多效性和功能性变异，这些变异预测了全球牛群的许多特征 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-021-21001-0</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33558518/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The difficulty in finding causative mutations has hampered their use in genomic prediction. Here, we present a methodology to fine-map potentially causal variants genome-wide by integrating the functional, evolutionary and pleiotropic information of variants using GWAS, variant clustering and Bayesian mixture models. Our analysis of 17 million sequence variants in 44,000+ Australian dairy cattle for 34 traits suggests, on average, one pleiotropic QTL existing in each 50 kb chromosome-segment. We selected a set of 80k variants representing potentially causal variants within each chromosome segment to develop a bovine XT-50K genotyping array. The custom array contains many pleiotropic variants with biological functions, including splicing QTLs and variants at conserved sites across 100 vertebrate species. This biology-informed custom array outperformed the standard array in predicting genetic value of multiple traits across populations in independent datasets of 90,000+ dairy cattle from the USA, Australia and New Zealand.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 寻找致病突变的困难已经阻碍了它们在基因组预测中的使用。在这里，我们提出了一种方法，可以通过使用GWAS，变体聚类和贝叶斯混合模型整合变体的功能，进化和多效性信息，在全基因组范围内精细定位潜在的致病变体。我们对44,000多头澳大利亚奶牛中的1,700万个序列变异进行了34个性状分析，结果表明，平均而言，每个50kb染色体片段中存在一个多效性QTL。我们选择了一组代表每个染色体区段内潜在因果变异的80k变异，以开发牛XT-50K基因分型阵列。定制阵列包含许多具有生物学功能的多效变体，包括在100个脊椎动物物种的保守位点处剪接QTL和变体。在预测来自美国，澳大利亚和新西兰的90,000多头奶牛的独立数据集中跨群体的多种性状的遗传价值时，这种具有生物学信息的定制阵列优于标准阵列。</p>

<h2>Pubmed_ID: 33558242</h2>

<h3>The Myc-associated zinc finger protein (MAZ) works together with CTCF to control cohesin positioning and genome organization</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2023127118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33558242/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The Myc-associated zinc finger protein (MAZ) is often found at genomic binding sites adjacent to CTCF, a protein which affects large-scale genome organization through its interaction with cohesin. We show here that, like CTCF, MAZ physically interacts with a cohesin subunit and can arrest cohesin sliding independently of CTCF. It also shares with CTCF the ability to independently pause the elongating form of RNA polymerase II, and consequently affects RNA alternative splicing. CTCF/MAZ double sites are more effective at sequestering cohesin than sites occupied only by CTCF. Furthermore, depletion of CTCF results in preferential loss of CTCF from sites not occupied by MAZ. In an assay for insulation activity like that used for CTCF, binding of MAZ to sites between an enhancer and promoter results in down-regulation of reporter gene expression, supporting a role for MAZ as an insulator protein. Hi-C analysis of the effect of MAZ depletion on genome organization shows that local interactions within topologically associated domains (TADs) are disrupted, as well as contacts that establish the boundaries of individual TADs. We conclude that MAZ augments the action of CTCF in organizing the genome, but also shares properties with CTCF that allow it to act independently.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33558228</h2>

<h3>The mop1 mutation affects the recombination landscape in maize</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2009475118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33558228/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Meiotic recombination is a fundamental process that generates genetic diversity and ensures the accurate segregation of homologous chromosomes. While a great deal is known about genetic factors that regulate recombination, relatively little is known about epigenetic factors, such as DNA methylation. In maize, we examined the effects on meiotic recombination of a mutation in a component of the RNA-directed DNA methylation pathway, Mop1 (Mediator of paramutation1), as well as a mutation in a component of the trans-acting small interference RNA biogenesis pathway, Lbl1 (Leafbladeless1). MOP1 is of particular interest with respect to recombination because it is responsible for methylation of transposable elements that are immediately adjacent to transcriptionally active genes. In the mop1 mutant, we found that meiotic recombination is uniformly decreased in pericentromeric regions but is generally increased in gene rich chromosomal arms. This observation was further confirmed by cytogenetic analysis showing that although overall crossover numbers are unchanged, they occur more frequently in chromosomal arms in mop1 mutants. Using whole genome bisulfite sequencing, our data show that crossover redistribution is driven by loss of CHH (where H = A, T, or C) methylation within regions near genes. In contrast to what we observed in mop1 mutants, no significant changes were observed in the frequency of meiotic recombination in lbl1 mutants. Our data demonstrate that CHH methylation has a significant impact on the overall recombination landscape in maize despite its low frequency relative to CG and CHG methylation.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 暂时不能翻译</p>

<h2>Pubmed_ID: 33547243</h2>

<h3>Reduced proinsecticide activation by cytochrome P450 confers coumaphos resistance in the major bee parasite Varroa destructor</h3>

<h3>细胞色素P450减少的杀虫剂活化可在主要的蜜蜂寄生虫Varroa破坏物中赋予库马磷抗性 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2020380118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33547243/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Varroa destructor is one of the main problems in modern beekeeping. Highly selective acaricides with low toxicity to bees are used internationally to control this mite. One of the key acaricides is the organophosphorus (OP) proinsecticide coumaphos, that becomes toxic after enzymatic activation inside Varroa We show here that mites from the island Andros (AN-CR) exhibit high levels of coumaphos resistance. Resistance is not mediated by decreased coumaphos uptake, target-site resistance, or increased detoxification. Reduced proinsecticide activation by a cytochrome P450 enzyme was the main resistance mechanism, a powerful and rarely encountered evolutionary solution to insecticide selection pressure. After treatment with sublethal doses of [14C] coumaphos, susceptible mite extracts had substantial amounts of coroxon, the activated metabolite of coumaphos, while resistant mites had only trace amounts. This indicates a suppression of the P450 (CYP)-mediated activation step in the AN-CR mites. Bioassays with coroxon to bypass the activation step showed that resistance was dramatically reduced. There are 26 CYPs present in the V. destructor genome. Transcriptome analysis revealed overexpression in resistant mites of CYP4DP24 and underexpression of CYP3012A6 and CYP4EP4 RNA interference of CYP4EP4 in the susceptible population, to mimic underexpression seen in the resistant mites, prevented coumaphos activation and decreased coumaphos toxicity.</p>

<p style="line-height:1.8;margin:20px"> 摘要: Varroa的破坏因子是现代养蜂业的主要问题之一。对蜜蜂具有低毒性的高选择性杀螨剂已在国际上用于控制这种螨虫。关键的杀螨剂之一是有机磷（OP）杀虫剂库玛磷，它在Varroa内部经过酶活化后变得有毒。我们在这里表明，来自安德罗斯岛（AN-CR）的螨虫表现出高水平的库玛磷抗性。抵抗力不是由减少的香豆磷摄取量，目标部位的抵抗力或排毒增加所介导的。细胞色素P450酶降低了杀虫剂的激活是主要的抗药性机制，这是杀虫剂选择压力的强大且鲜为人知的进化解决方案。用亚致死剂量的[14C]香豆磷处理后，易感的螨虫提取物中含有大量的coroxon（即库玛磷的活化代谢产物），而抗药性螨虫则只有微量。这表明抑制了AN-CR螨虫中P450（CYP）介导的活化步骤。使用Coroxon绕过激活步骤的生物测定表明，耐药性显着降低。灭弧菌基因组中存在26种CYP。转录组分析显示易感人群中CYP4DP24抗性螨的过表达以及CYP3012A6和CYP4EP4RNA的表达不足，以模仿易感人群中的CYP4EP4表达不足，阻止了库马磷的活化并降低了库马磷的毒性。</p>

<h2>Pubmed_ID: 33536335</h2>

<h3>SLFN11 promotes CDT1 degradation by CUL4 in response to replicative DNA damage, while its absence leads to synthetic lethality with ATR/CHK1 inhibitors</h3>

<h3>SLFN11响应复制性DNA损伤而促进CUL4降解CDT1，而其缺失会导致ATR / CHK1抑制剂的合成杀伤力 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2015654118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33536335/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Schlafen-11 (SLFN11) inactivation in ∼50% of cancer cells confers broad chemoresistance. To identify therapeutic targets and underlying molecular mechanisms for overcoming chemoresistance, we performed an unbiased genome-wide RNAi screen in SLFN11-WT and -knockout (KO) cells. We found that inactivation of Ataxia Telangiectasia- and Rad3-related (ATR), CHK1, BRCA2, and RPA1 overcome chemoresistance to camptothecin (CPT) in SLFN11-KO cells. Accordingly, we validate that clinical inhibitors of ATR (M4344 and M6620) and CHK1 (SRA737) resensitize SLFN11-KO cells to topotecan, indotecan, etoposide, cisplatin, and talazoparib. We uncover that ATR inhibition significantly increases mitotic defects along with increased CDT1 phosphorylation, which destabilizes kinetochore-microtubule attachments in SLFN11-KO cells. We also reveal a chemoresistance mechanism by which CDT1 degradation is retarded, eventually inducing replication reactivation under DNA damage in SLFN11-KO cells. In contrast, in SLFN11-expressing cells, SLFN11 promotes the degradation of CDT1 in response to CPT by binding to DDB1 of CUL4CDT2 E3 ubiquitin ligase associated with replication forks. We show that the C terminus and ATPase domain of SLFN11 are required for DDB1 binding and CDT1 degradation. Furthermore, we identify a therapy-relevant ATPase mutant (E669K) of the SLFN11 gene in human TCGA and show that the mutant contributes to chemoresistance and retarded CDT1 degradation. Taken together, our study reveals new chemotherapeutic insights on how targeting the ATR pathway overcomes chemoresistance of SLFN11-deficient cancers. It also demonstrates that SLFN11 irreversibly arrests replication by degrading CDT1 through the DDB1-CUL4CDT2 ubiquitin ligase.</p>

<p style="line-height:1.8;margin:20px"> 摘要: Schlafen-11（SLFN11）失活在约50％的癌细胞中具有广泛的化学抗性。为了确定克服化学耐药性的治疗靶标和潜在分子机制，我们在SLFN11-WT和-knockout（KO）细胞中进行了无偏见的全基因组RNAi筛选。我们发现失活的共济失调毛细血管扩张和Rad3相关（ATR），CHK1，BRCA2和RPA1克服了SLFN11-KO细胞中喜树碱（CPT）的化学耐药性。因此，我们验证了ATR（M4344和M6620）和CHK1（SRA737）的临床抑制剂可使SLFN11-KO细胞对托泊替康，吲哚酚，依托泊苷，顺铂和他拉唑帕尼重新敏感。我们发现，ATR抑制作用会显着增加有丝分裂缺陷以及CDT1磷酸化的增加，从而使SLFN11-KO细胞中的动粒微管附着不稳定。我们还揭示了CDT1降解受阻的化学抗性机制，最终在SLFN11-KO细胞的DNA损伤下诱导复制再激活。相反，在表达SLFN11的细胞中，SLFN11通过与复制叉相关的CUL4CDT2E3泛素连接酶的DDB1结合而促进CDT1降解，以响应CPT。我们表明，SLFN11的C末端和ATPase域对于DDB1结合和CDT1降解是必需的。此外，我们在人类TCGA中鉴定了SLFN11基因的治疗相关ATPase突变体（E669K），并表明该突变体有助于化学抗性和CDT1降解受阻。综上所述，我们的研究揭示了关于靶向ATR途径如何克服SLFN11缺陷型癌症的化学耐药性的新化学疗法见解。它还表明SLFN11通过通过DDB1-CUL4CDT2泛素连接酶降解CDT1不可逆转地阻止了复制。</p>

<h2>Pubmed_ID: 33536333</h2>

<h3>Multiseriate cortical sclerenchyma enhance root penetration in compacted soils</h3>

<h3>多重皮层皮质硬化增强了土壤在致密土壤中的根系渗透 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2012087118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33536333/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Mechanical impedance limits soil exploration and resource capture by plant roots. We examine the role of root anatomy in regulating plant adaptation to mechanical impedance and identify a root anatomical phene in maize (Zea mays) and wheat (Triticum aestivum) associated with penetration of hard soil: Multiseriate cortical sclerenchyma (MCS). We characterize this trait and evaluate the utility of MCS for root penetration in compacted soils. Roots with MCS had a greater cell wall-to-lumen ratio and a distinct UV emission spectrum in outer cortical cells. Genome-wide association mapping revealed that MCS is heritable and genetically controlled. We identified a candidate gene associated with MCS. Across all root classes and nodal positions, maize genotypes with MCS had 13% greater root lignin concentration compared to genotypes without MCS. Genotypes without MCS formed MCS upon exogenous ethylene exposure. Genotypes with MCS had greater lignin concentration and bending strength at the root tip. In controlled environments, MCS in maize and wheat was associated improved root tensile strength and increased penetration ability in compacted soils. Maize genotypes with MCS had root systems with 22% greater depth and 49% greater shoot biomass in compacted soils in the field compared to lines without MCS. Of the lines we assessed, MCS was present in 30 to 50% of modern maize, wheat, and barley cultivars but was absent in teosinte and wild and landrace accessions of wheat and barley. MCS merits investigation as a trait for improving plant performance in maize, wheat, and other grasses under edaphic stress.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 机械阻抗限制了土壤探索和植物根系对资源的捕获。我们检查了根部解剖结构在调节植物对机械抗性的适应性中的作用，并确定了与硬土渗透有关的玉米（Zeamays）和小麦（Triticumaestivum）中的根部解剖表型：多序列皮层硬皮病（MCS）。我们表征该特征并评估MCS在压实土壤中用于根渗透的效用。带有MCS的根在外皮层细胞中具有更高的细胞壁与腔比，并且具有独特的UV发射光谱。全基因组关联图谱表明，MCS是可遗传的，并且是遗传控制的。我们确定了与MCS相关的候选基因。在所有根系和节点位置上，具有MCS的玉米基因型的根木质素浓度比没有MCS的基因型高13％。没有MCS的基因型在外源性乙烯暴露下形成MCS。MCS基因型在根尖具有更高的木质素浓度和弯曲强度。在受控环境中，玉米和小麦中的MCS可以提高根系抗张强度并增加在压实土壤中的渗透能力。与没有MCS的品系相比，在田间压实的土壤中，带有MCS的玉米基因型的根系深度增加22％，芽生物量增加49％。在我们评估的品系中，MCS存在于现代玉米，小麦和大麦品种的30％至50％中，但在小麦和大麦的teosinte和野生以及地方品种中却不存在。MCS值得研究，因为它可以改善在干旱胁迫下玉米，小麦和其他草的植物生长。</p>

<h2>Pubmed_ID: 33531363</h2>

<h3>Redox regulation of NADP-malate dehydrogenase is vital for land plants under fluctuating light environment</h3>

<h3>NADP-苹果酸脱氢酶的氧化还原调节对光环境变化的陆地植物至关重要 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2016903118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33531363/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Many enzymes involved in photosynthesis possess highly conserved cysteine residues that serve as redox switches in chloroplasts. These redox switches function to activate or deactivate enzymes during light-dark transitions and have the function of fine-tuning their activities according to the intensity of light. Accordingly, many studies on chloroplast redox regulation have been conducted under the hypothesis that "fine regulation of the activities of these enzymes is crucial for efficient photosynthesis." However, the impact of the regulatory system on plant metabolism is still unclear. To test this hypothesis, we here studied the impact of the ablation of a redox switch in chloroplast NADP-malate dehydrogenase (MDH). By genome editing, we generated a mutant plant whose MDH lacks one of its redox switches and is active even in dark conditions. Although NADPH consumption by MDH in the dark is expected to be harmful to plant growth, the mutant line did not show any phenotypic differences under standard long-day conditions. In contrast, the mutant line showed severe growth retardation under short-day or fluctuating light conditions. These results indicate that thiol-switch redox regulation of MDH activity is crucial for maintaining NADPH homeostasis in chloroplasts under these conditions.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 许多参与光合作用的酶具有高度保守的半胱氨酸残基，可作为叶绿体中的氧化还原开关。这些氧化还原开关的作用是在明暗过渡期间激活或灭活酶，并具有根据光强度微调其活性的功能。因此，在以下假设下进行了许多关于叶绿体氧化还原调节的研究：“这些酶活性的精细调节对于有效的光合作用至关重要。”但是，调节系统对植物代谢的影响仍不清楚。为了验证该假设，我们在这里研究了叶绿体NADP-苹果酸脱氢酶（MDH）中氧化还原开关消融的影响。通过基因组编辑，我们生成了一种突变植物，其MDH缺少其氧化还原开关之一，即使在黑暗条件下也具有活性。尽管在黑暗中MDH消耗NADPH可能对植物生长有害，但在标准的长期条件下，突变株系没有表现出任何表型差异。相反，突变株在短日照或波动的光照条件下表现出严重的生长迟缓。这些结果表明，在这些条件下，MDH活性的硫醇开关氧化还原调节对于维持叶绿体中NADPH的稳态至关重要。</p>

<h2>Pubmed_ID: 33526700</h2>

<h3>Cross-talk of insulin-like peptides, juvenile hormone, and 20-hydroxyecdysone in regulation of metabolism in the mosquito Aedes aegypti</h3>

<h3>胰岛素样肽，少年激素和20-羟基蜕皮酮的串扰在蚊埃及伊蚊的代谢调控中的作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2023470118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33526700/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Female mosquitoes feed sequentially on carbohydrates (nectar) and proteins (blood) during each gonadotrophic cycle to become reproductively competent and effective disease vectors. Accordingly, metabolism is synchronized to support this reproductive cyclicity. However, regulatory pathways linking metabolism to reproductive cycles are not fully understood. Two key hormones, juvenile hormone (JH) and ecdysteroids (20-hydroxyecdysone, 20E, is the most active form) govern female mosquito reproduction. Aedes aegypti genome codes for eight insulin-like peptides (ILPs) that are critical for controlling metabolism. We examined the effects of the JH and 20E pathways on mosquito ILP expression to decipher regulation of metabolism in a reproducing female mosquito. Chromatin immunoprecipitation assays showed genomic interactions between ilp genes and the JH receptor, methoprene-tolerant, a transcription factor, Krüppel homolog 1 (Kr-h1), and two isoforms of the ecdysone response early gene, E74. The luciferase reporter assays showed that Kr-h1 activates ilps 2, 6, and 7, but represses ilps 4 and 5 The 20E pathway displayed the opposite effect in the regulation of ilps E74B repressed ilps 2 and 6, while E74A activated ilps 4 and 5 Combining RNA interference, CRISPR gene tagging and enzyme-linked immunosorbent assay, we have shown that the JH and 20E regulate protein levels of all eight Ae. aegypti ILPs. Thus, we have established a regulatory axis between ILPs, JH, and 20E in coordination of metabolism during gonadotrophic cycles of Ae. aegypti.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在每个性腺营养循环中，雌性蚊子依次以碳水化合物（花蜜）和蛋白质（血液）为食，成为具有繁殖能力和有效疾病的媒介。因此，新陈代谢同步以支持该生殖周期。但是，尚不完全了解将代谢与生殖周期联系起来的调控途径。雌性蚊子的繁殖有两种主要激素，少年激素（JH）和蜕皮类固醇（20-羟基蜕皮激素20E是最活跃的形式）。埃及伊蚊基因组编码八种胰岛素样肽（ILP），它们对控制新陈代谢至关重要。我们检查了JH和20E途径对蚊子ILP表达的影响，以破译繁殖雌性蚊子的新陈代谢调节。染色质免疫沉淀试验显示，ilp基因与JH受体，甲氧戊二烯耐受性，转录因子Krüppel同系物1（Kr-h1）和蜕皮激素响应早期基因E74的两个同工型之间存在基因组相互作用。萤光素酶报告基因检测显示Kr-h1激活ilps2、6和7，但抑制ilps4和5。20E途径在调节ilps中显示相反的作用；E74B抑制ilps2和6，而E74A激活ilps4和5。结合RNA干扰，CRISPR基因标记和酶联免疫吸附测定，我们显示JH和20E调节所有8种Ae的蛋白质水平。埃及ILP。因此，我们已经建立了ILP，JH和20E之间的调控轴，以协调Ae的促性腺营养循环中的代谢。埃及</p>

<h2>Pubmed_ID: 33526697</h2>

<h3>Genome-wide role of codon usage on transcription and identification of potential regulators</h3>

<h3>密码子使用在全基因组范围内对转录和潜在调节子的识别的作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2022590118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33526697/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Codon usage bias is a fundamental feature of all genomes and plays an important role in determining gene expression levels. The codon usage was thought to influence gene expression mainly due to its impact on translation. Recently, however, codon usage was shown to affect transcription of fungal and mammalian genes, indicating the existence of a gene regulatory phenomenon with unknown mechanism. In Neurospora, codon usage biases strongly correlate with mRNA levels genome-wide, and here we show that the correlation between codon usage and RNA levels is maintained in the nucleus. In addition, codon optimality is tightly correlated with both total and nuclear RNA levels, suggesting that codon usage broadly influences mRNA levels through transcription in a translation-independent manner. A large-scale RNA sequencing-based genetic screen in Neurospora identified 18 candidate factors that when deleted decreased the genome-wide correlation between codon usage and RNA levels and reduced the codon usage effect on gene expression. Most of these factors, such as the H3K36 methyltransferase, are chromatin regulators or transcription factors. Together, our results suggest that the transcriptional effect of codon usage is mediated by multiple transcriptional regulatory mechanisms.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 密码子使用偏倚是所有基因组的基本特征，并且在确定基因表达水平中起重要作用。人们认为密码子的使用主要是由于其对翻译的影响而影响基因表达。然而，最近，密码子的使用已显示出会影响真菌和哺乳动物基因的转录，表明存在机制未知的基因调控现象。在Neurospora中，密码子使用偏倚与全基因组的mRNA水平密切相关，在这里我们表明密码子使用与RNA水平之间的相关性在细胞核中得以维持。此外，密码子最佳性与总RNA和核RNA水平都紧密相关，这表明密码子的使用可通过转录以与翻译无关的方式广泛影响mRNA的水平。Neurospora中基于大规模RNA测序的遗传筛选确定了18个候选因子，这些因子在删除后降低了密码子使用和RNA水平之间的全基因组关联，并降低了密码子使用对基因表达的影响。这些因子中的大多数，例如H3K36甲基转移酶，都是染色质调节剂或转录因子。在一起，我们的结果表明密码子使用的转录作用是由多种转录调控机制介导的。</p>

<h2>Pubmed_ID: 33526668</h2>

<h3>Lipopeptide-mediated bacterial interaction enables cooperative predator defense</h3>

<h3>脂肽介导的细菌相互作用可实现捕食者的协同防御 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2013759118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33526668/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Bacteria are inherently social organisms whose actions should ideally be studied within an interactive ecological context. We show that the exchange and modification of natural products enables two unrelated bacteria to defend themselves against a common predator. Amoebal predation is a major cause of death in soil bacteria and thus it exerts a strong selective pressure to evolve defensive strategies. A systematic analysis of binary combinations of coisolated bacteria revealed strains that were individually susceptible to predation but together killed their predator. This cooperative defense relies on a Pseudomonas species producing syringafactin, a lipopeptide, which induces the production of peptidases in a Paenibacillus strain. These peptidases then degrade the innocuous syringafactin into compounds, which kill the predator. A combination of bioprospecting, coculture experiments, genome modification, and transcriptomics unravel this novel natural product-based defense strategy.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 细菌是固有的社会有机体，理想情况下应在互动的生态环境下研究其作用。我们表明，天然产物的交换和修饰使两种不相关的细菌能够抵御常见的捕食者。变形虫的掠食是土壤细菌死亡的主要原因，因此，它会施加强大的选择性压力来发展防御策略。对共分离细菌的二元组合的系统分析显示，这些菌株容易被捕食，但一起杀死了它们的捕食者。这种协同防御依赖于假单胞菌属物种，其产生丁香脂素，一种脂肽，其诱导在杆状芽孢杆菌菌株中肽酶的产生。然后，这些肽酶将无毒的丁香脂素降解为可杀死捕食者的化合物。生物勘探，共培养实验，基因组修饰和转录组学相结合，揭示了这种新颖的基于天然产物的防御策略。</p>

<h2>Pubmed_ID: 33526660</h2>

<h3>High-resolution imaging reveals compartmentalization of mitochondrial protein synthesis in cultured human cells</h3>

<h3>高分辨率成像揭示了培养的人类细胞中线粒体蛋白合成的区室化 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2008778118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33526660/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Human mitochondria contain their own genome, mitochondrial DNA, that is expressed in the mitochondrial matrix. This genome encodes 13 vital polypeptides that are components of the multisubunit complexes that couple oxidative phosphorylation (OXPHOS). The inner mitochondrial membrane that houses these complexes comprises the inner boundary membrane that runs parallel to the outer membrane, infoldings that form the cristae membranes, and the cristae junctions that separate the two. It is in these cristae membranes that the OXPHOS complexes have been shown to reside in various species. The majority of the OXPHOS subunits are nuclear-encoded and must therefore be imported from the cytosol through the outer membrane at contact sites with the inner boundary membrane. As the mitochondrially encoded components are also integral members of these complexes, where does protein synthesis occur? As transcription, mRNA processing, maturation, and at least part of the mitoribosome assembly process occur at the nucleoid and the spatially juxtaposed mitochondrial RNA granules, is protein synthesis also performed at the RNA granules close to these entities, or does it occur distal to these sites? We have adapted a click chemistry-based method coupled with stimulated emission depletion nanoscopy to address these questions. We report that, in human cells in culture, within the limits of our methodology, the majority of mitochondrial protein synthesis is detected at the cristae membranes and is spatially separated from the sites of RNA processing and maturation.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 人类线粒体包含自己的基因组，即线粒体DNA，在线粒体基质中表达。该基因组编码13种重要多肽，这些多肽是偶联氧化磷酸化（OXPHOS）的多亚基复合物的组成部分。容纳这些复合物的线粒体内膜包括平行于外膜的内边界膜，形成cr膜的折叠以及将两者分开的cr连接。OXPHOS复合物已在这些cr状膜中存在于各种物种中。大多数OXPHOS亚基是核编码的，因此必须从胞质溶胶通过外膜与内边界膜的接触部位从细胞质中导入。由于线粒体编码的成分也是这些复合物的组成部分，因此蛋白质合成在哪里发生？当转录，mRNA加工，成熟以及至少一部分线粒体组装过程发生在核苷酸和空间并置的线粒体RNA颗粒上时，蛋白质合成也在这些实体附近的RNA颗粒上进行，还是发生在这些实体的远端网站？我们采用了一种基于点击化学的方法，结合了激发发射损耗纳米技术，以解决这些问题。我们报告说，在培养的人类细胞中，在我们方法学的范围内，大多数线粒体蛋白合成是在cr膜上检测到的，并且在空间上与RNA加工和成熟的位置分开。</p>

<h2>Pubmed_ID: 33526659</h2>

<h3>Conjugative plasmids interact with insertion sequences to shape the horizontal transfer of antimicrobial resistance genes</h3>

<h3>结合质粒与插入序列相互作用以形成抗菌素耐药基因的水平转移 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2008731118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33526659/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: It is well established that plasmids play an important role in the dissemination of antimicrobial resistance (AMR) genes; however, little is known about the role of the underlying interactions between different plasmid categories and other mobile genetic elements (MGEs) in shaping the promiscuous spread of AMR genes. Here, we developed a tool designed for plasmid classification, AMR gene annotation, and plasmid visualization and found that most plasmid-borne AMR genes, including those localized on class 1 integrons, are enriched in conjugative plasmids. Notably, we report the discovery and characterization of a massive insertion sequence (IS)-associated AMR gene transfer network (245 combinations covering 59 AMR gene subtypes and 53 ISs) linking conjugative plasmids and phylogenetically distant pathogens, suggesting a general evolutionary mechanism for the horizontal transfer of AMR genes mediated by the interaction between conjugative plasmids and ISs. Moreover, our experimental results confirmed the importance of the observed interactions in aiding the horizontal transfer and expanding the genetic range of AMR genes within complex microbial communities.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 众所周知，质粒在抗菌素耐药性（AMR）基因的传播中起着重要的作用。然而，关于不同质粒类别与其他移动遗传元件（MGE）之间潜在相互作用的作用在塑造AMR基因的混杂传播方面所起的作用还知之甚少。在这里，我们开发了一种用于质粒分类，AMR基因注释和质粒可视化的工具，发现大多数质粒携带的AMR基因，包括位于1类整合素上的基因，都富含共轭质粒。值得注意的是，我们报告了发现和表征大规模连接序列（IS）的AMR基因转移网络（245个组合，涵盖59个AMR基因亚型和53个IS），它们连接了共轭质粒和系统发生距离较远的病原体，提示了水平方向的一般进化机制结合质粒和ISs相互作用介导的AMR基因转移。此外，我们的实验结果证实了所观察到的相互作用在辅助水平转移和扩大复杂微生物群落内AMR基因的遗传范围方面的重要性。</p>

<h2>Pubmed_ID: 33504594</h2>

<h3>Cancer recurrence and lethality are enabled by enhanced survival and reversible cell cycle arrest of polyaneuploid cells</h3>

<h3>暂时不能翻译</h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2020838118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33504594/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/16/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: We present a unifying theory to explain cancer recurrence, therapeutic resistance, and lethality. The basis of this theory is the formation of simultaneously polyploid and aneuploid cancer cells, polyaneuploid cancer cells (PACCs), that avoid the toxic effects of systemic therapy by entering a state of cell cycle arrest. The theory is independent of which of the classically associated oncogenic mutations have already occurred. PACCs have been generally disregarded as senescent or dying cells. Our theory states that therapeutic resistance is driven by PACC formation that is enabled by accessing a polyploid program that allows an aneuploid cancer cell to double its genomic content, followed by entry into a nondividing cell state to protect DNA integrity and ensure cell survival. Upon removal of stress, e.g., chemotherapy, PACCs undergo depolyploidization and generate resistant progeny that make up the bulk of cancer cells within a tumor.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 我们提出了一个统一的理论来解释癌症的复发，治疗耐药性和致死性。该理论的基础是同时形成多倍体和非整倍体癌细胞，即多非整倍体癌细胞（PACC），它们通过进入细胞周期停滞状态而避免了全身疗法的毒性作用。该理论与经典关联的致癌突变中已经发生的突变无关。通常将PACC视为衰老或垂死细胞。我们的理论认为，治疗耐药性是由PACC形成驱动的，该过程可通过访问一个多倍体程序来实现，该程序使非整倍性癌细胞的基因组含量翻倍，然后进入非分裂细胞状态以保护DNA完整性并确保细胞存活。在消除压力（例如化学疗法）后，PACC经历多倍体化并产生抗性后代，其构成肿瘤内的大部分癌细胞。</p>

<h2>Pubmed_ID: 33384301</h2>

<h3>In situ genome sequencing resolves DNA sequence and structure in intact biological samples</h3>

<h3>原位基因组测序可解析完整生物样品中的DNA序列和结构 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1126/science.aay3446</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33384301/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Science (New York, N.Y.)</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 02/26/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Understanding genome organization requires integration of DNA sequence and three-dimensional spatial context; however, existing genome-wide methods lack either base pair sequence resolution or direct spatial localization. Here, we describe in situ genome sequencing (IGS), a method for simultaneously sequencing and imaging genomes within intact biological samples. We applied IGS to human fibroblasts and early mouse embryos, spatially localizing thousands of genomic loci in individual nuclei. Using these data, we characterized parent-specific changes in genome structure across embryonic stages, revealed single-cell chromatin domains in zygotes, and uncovered epigenetic memory of global chromosome positioning within individual embryos. These results demonstrate how IGS can directly connect sequence and structure across length scales from single base pairs to whole organisms.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 了解基因组的组织需要整合DNA序列和三维空间环境。然而，现有的全基因组方法缺乏碱基对序列解析或直接的空间定位。在这里，我们描述了原位基因组测序（IGS），这是一种同时对完整的生物样品中的基因组进行测序和成像的方法。我们将IGS应用于人类成纤维细胞和早期小鼠胚胎，在空间上将成千上万个基因组位点定位在单个核中。利用这些数据，我们表征了跨胚胎阶段的基因组结构中父母特异性的变化，揭示了受精卵中的单细胞染色质结构域，以及未发现的单个染色体内全局染色体定位的表观遗传记忆。这些结果表明，IGS如何直接将序列和结构从单碱基对到整个生物的整个长度尺度连接起来。</p>

</body>
</html>

