<!DOCTYPE html> <html lang = "en">
 <body>

<h2>Pubmed_ID: 33443212</h2>

<h3>Efficient CRISPR-mediated base editing in Agrobacterium spp</h3>

<h3>农杆菌属中高效CRISPR介导的碱基编辑 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2013338118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443212/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Agrobacterium spp. are important plant pathogens that are the causative agents of crown gall or hairy root disease. Their unique infection strategy depends on the delivery of part of their DNA to plant cells. Thanks to this capacity, these phytopathogens became a powerful and indispensable tool for plant genetic engineering and agricultural biotechnology. Although Agrobacterium spp. are standard tools for plant molecular biologists, current laboratory strains have remained unchanged for decades and functional gene analysis of Agrobacterium has been hampered by time-consuming mutation strategies. Here, we developed clustered regularly interspaced short palindromic repeats (CRISPR)-mediated base editing to enable the efficient introduction of targeted point mutations into the genomes of both Agrobacterium tumefaciens and Agrobacterium rhizogenes As an example, we generated EHA105 strains with loss-of-function mutations in recA, which were fully functional for maize (Zea mays) transformation and confirmed the importance of RolB and RolC for hairy root development by A. rhizogenes K599. Our method is highly effective in 9 of 10 colonies after transformation, with edits in at least 80% of the cells. The genomes of EHA105 and K599 were resequenced, and genome-wide off-target analysis was applied to investigate the edited strains after curing of the base editor plasmid. The off-targets present were characteristic of Cas9-independent off-targeting and point to TC motifs as activity hotspots of the cytidine deaminase used. We anticipate that CRISPR-mediated base editing is the start of "engineering the engineer," leading to improved Agrobacterium strains for more efficient plant transformation and gene editing.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 农杆菌属是重要的植物病原体，是冠gall病或毛根病的病原体。它们独特的感染策略取决于将其部分DNA传递至植物细胞。由于具有这种能力，这些植物病原体已成为植物基因工程和农业生物技术的强大而必不可少的工具。虽然农杆菌属。作为植物分子生物学家的标准工具，目前的实验室菌株数十年来保持不变，并且耗时的突变策略阻碍了农杆菌的功能基因分析。在这里，我们开发了成簇的规则间隔的短回文重复序列（CRISPR）介导的碱基编辑，以将靶向点突变有效地引入根癌农杆菌和发根农杆菌的基因组中。例如，我们生成了功能丧失的EHA105菌株recA中的突变对玉米（Zeamays）转化具有完全的功能，并证实了RolB和RolC对发根农杆菌K599毛状根发育的重要性。我们的方法在转化后的10个菌落中有9个具有很高的效率，至少可以编辑80％的细胞。对EHA105和K599的基因组进行重新测序，并在碱基编辑质粒固化后应用全基因组脱靶分析来研究编辑菌株。存在的脱靶是独立于Cas9的脱靶的特征，并且指向TC基序作为所使用的胞苷脱氨酶的活性热点。我们预期CRISPR介导的碱基编辑是“工程师的工程”的开始，从而导致改良农杆菌菌株的产生，从而更有效地进行植物转化和基因编辑。</p>

<h2>Pubmed_ID: 33443210</h2>

<h3>Precise genome engineering in Drosophila using prime editing</h3>

<h3>使用主要编辑在果蝇中进行精确的基因组工程 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2021996118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443210/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Precise genome editing is a valuable tool to study gene function in model organisms. Prime editing, a precise editing system developed in mammalian cells, does not require double-strand breaks or donor DNA and has low off-target effects. Here, we applied prime editing for the model organism Drosophila melanogaster and developed conditions for optimal editing. By expressing prime editing components in cultured cells or somatic cells of transgenic flies, we precisely introduce premature stop codons in three classical visible marker genes, ebony, white, and forked Furthermore, by restricting editing to germ cells, we demonstrate efficient germ-line transmission of a precise edit in ebony to 36% of progeny. Our results suggest that prime editing is a useful system in Drosophila to study gene function, such as engineering precise point mutations, deletions, or epitope tags.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 精确的基因组编辑是研究模型生物中基因功能的宝贵工具。原始编辑是在哺乳动物细胞中开发的精确编辑系统，不需要双链断裂或供体DNA，而且脱靶效应低。在这里，我们对模型生物果蝇（Drosophilamelanogaster）应用了主要编辑，并开发了最佳编辑条件。通过在转基因果蝇的培养细胞或体细胞中表达主要的编辑成分，我们在三个经典的可见标记基因（乌木，白色和叉状）中精确地引入了提前终止密码子。此外，通过将编辑限制在生殖细胞中，我们证明了有效的种系传递乌木的精确编辑占后代的36％。我们的结果表明，主要编辑是果蝇研究基因功能（如工程精确点突变，缺失或表位标签）的有用系统。</p>

<h2>Pubmed_ID: 33443184</h2>

<h3>Real-time observation of Cas9 postcatalytic domain motions</h3>

<h3>实时观察Cas9催化后域运动 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2010650118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443184/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: CRISPR-Cas9 from Streptococcus pyogenes is an RNA-guided DNA endonuclease, which has become the most popular genome editing tool. Coordinated domain motions of Cas9 prior to DNA cleavage have been extensively characterized but our understanding of Cas9 conformations postcatalysis is limited. Because Cas9 can remain stably bound to the cleaved DNA for hours, its postcatalytic conformation may influence genome editing mechanisms. Here, we use single-molecule fluorescence resonance energy transfer to characterize the HNH domain motions of Cas9 that are coupled with cleavage activity of the target strand (TS) or nontarget strand (NTS) of DNA substrate. We reveal an NTS-cleavage-competent conformation following the HNH domain conformational activation. The 3' flap generated by NTS cleavage can be rapidly digested by a 3' to 5' single-stranded DNA-specific exonuclease, indicating Cas9 exposes the 3' flap for potential interaction with the DNA repair machinery. We find evidence that the HNH domain is highly flexible post-TS cleavage, explaining a recent observation that the HNH domain was not visible in a postcatalytic cryo-EM structure. Our results illuminate previously unappreciated regulatory roles of DNA cleavage activity on Cas9's conformation and suggest possible biotechnological applications.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 化脓链球菌的CRISPR-Cas9是一种RNA引导的DNA内切核酸酶，已成为最受欢迎的基因组编辑工具。DNA切割之前，Cas9的协调域运动已被广泛表征，但我们对催化后的Cas9构象的了解有限。由于Cas9可以稳定地与切割的DNA结合数小时，因此它的催化后构象可能会影响基因组编辑机制。在这里，我们使用单分子荧光共振能量转移来表征Cas9的HNH结构域运动，并与DNA底物的目标链（TS）或非目标链（NTS）的裂解活性相结合。我们揭示了HNH域构象激活后的NTS裂解能力构象。NTS切割产生的3个襟翼可以被3到5个单链DNA特异性核酸外切酶快速消化，表明Cas9暴露了3个襟翼与DNA修复机制的潜在相互作用。我们发现证据表明HNH域在TS裂解后具有高度的柔性，解释了最近的观察结果，即HNH域在催化后的冷冻EM结构中不可见。我们的结果阐明了DNA裂解活性对Cas9s构象的前所未有的调节作用，并暗示了可能的生物技术应用。</p>

<h2>Pubmed_ID: 33443182</h2>

<h3>The evolution of skin pigmentation-associated variation in West Eurasia</h3>

<h3>西欧亚大陆与皮肤色素沉着有关的变异的演变 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2009227118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443182/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Skin pigmentation is a classic example of a polygenic trait that has experienced directional selection in humans. Genome-wide association studies have identified well over a hundred pigmentation-associated loci, and genomic scans in present-day and ancient populations have identified selective sweeps for a small number of light pigmentation-associated alleles in Europeans. It is unclear whether selection has operated on all of the genetic variation associated with skin pigmentation as opposed to just a small number of large-effect variants. Here, we address this question using ancient DNA from 1,158 individuals from West Eurasia covering a period of 40,000 y combined with genome-wide association summary statistics from the UK Biobank. We find a robust signal of directional selection in ancient West Eurasians on 170 skin pigmentation-associated variants ascertained in the UK Biobank. However, we also show that this signal is driven by a limited number of large-effect variants. Consistent with this observation, we find that a polygenic selection test in present-day populations fails to detect selection with the full set of variants. Our data allow us to disentangle the effects of admixture and selection. Most notably, a large-effect variant at SLC24A5 was introduced to Western Europe by migrations of Neolithic farming populations but continued to be under selection post-admixture. This study shows that the response to selection for light skin pigmentation in West Eurasia was driven by a relatively small proportion of the variants that are associated with present-day phenotypic variation.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 皮肤色素沉着是人类经历定向选择的多基因性状的经典例子。全基因组关联研究已经确定了一百多个与色素沉着相关的基因座，并且在当今和古代人群中的基因组扫描已经确定了欧洲人中少数与色素沉着相关的轻等位基因的选择性扫描。目前尚不清楚选择是否针对与皮肤色素沉着相关的所有遗传变异，而不是仅针对少数几个大效应变异。在这里，我们使用来自西欧亚大陆的1,158位个体（覆盖40,000年）的古代DNA结合英国生物库的全基因组关联摘要统计信息来解决此问题。我们在英国生物库中确定的170种与皮肤色素沉着相关的变体上发现了古老的西欧亚大陆方向选择的有力信号。但是，我们还显示此信号是由有限数量的大型效果变量驱动的。与此观察结果一致，我们发现当今人群中的多基因选择测试无法检测出具有完整变体集的选择。我们的数据使我们能够弄清楚混合和选择的影响。最值得注意的是，由于新石器时代农业人口的迁徙，SLC24A5的一个大效应变种被引入西欧，但在混合后仍处于选择状态。这项研究表明，西欧亚大陆对浅色皮肤色素沉着的选择反应是由相对较小比例的与当今表型变异相关的变异驱动的。</p>

<h2>Pubmed_ID: 33443179</h2>

<h3>Mfd regulates RNA polymerase association with hard-to-transcribe regions in vivo, especially those with structured RNAs</h3>

<h3>Mfd调节体内RNA聚合酶与难以转录的区域的结合，尤其是那些具有结构化RNA的区域 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2008498118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443179/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: RNA polymerase (RNAP) encounters various roadblocks during transcription. These obstacles can impede RNAP movement and influence transcription, ultimately necessitating the activity of RNAP-associated factors. One such factor is the bacterial protein Mfd, a highly conserved DNA translocase and evolvability factor that interacts with RNAP. Although Mfd is thought to function primarily in the repair of DNA lesions that stall RNAP, increasing evidence suggests that it may also be important for transcription regulation. However, this is yet to be fully characterized. To shed light on Mfd's in vivo functions, we identified the chromosomal regions where it associates. We analyzed Mfd's impact on RNAP association and transcription regulation genome-wide. We found that Mfd represses RNAP association at many chromosomal regions. We found that these regions show increased RNAP pausing, suggesting that they are hard to transcribe. Interestingly, we noticed that the majority of the regions where Mfd regulates transcription contain highly structured regulatory RNAs. The RNAs identified regulate a myriad of biological processes, ranging from metabolism to transfer RNA regulation to toxin-antitoxin (TA) functions. We found that cells lacking Mfd are highly sensitive to toxin overexpression. Finally, we found that Mfd promotes mutagenesis in at least one toxin gene, suggesting that its function in regulating transcription may promote evolution of certain TA systems and other regions containing strong RNA secondary structures. We conclude that Mfd is an RNAP cofactor that is important, and at times critical, for transcription regulation at hard-to-transcribe regions, especially those that express structured regulatory RNAs.</p>

<p style="line-height:1.8;margin:20px"> 摘要: RNA聚合酶（RNAP）在转录过程中遇到各种障碍。这些障碍会阻碍RNAP的移动并影响转录，最终需要RNAP相关因子的活性。这样的因子之一是细菌蛋白Mfd，它是一种高度保守的DNA转移酶，并且是与RNAP相互作用的进化因子。尽管人们认为Mfd主要起使RNAP失速的DNA损伤修复的作用，但越来越多的证据表明Mfd对于转录调控也可能很重要。但是，这尚未完全表征。为了阐明Mfds的体内功能，我们鉴定了与其相关的染色体区域。我们分析了Mfds对全基因组RNAP关联和转录调控的影响。我们发现Mfd在许多染色体区域抑制RNA缔合。我们发现这些区域显示出增加的RNAP暂停，表明它们很难转录。有趣的是，我们注意到Mfd调节转录的大多数区域都包含高度结构化的调节RNA。鉴定出的RNA调节无数的生物过程，从代谢到将RNA调节转移到毒素-抗毒素（TA）功能。我们发现缺乏Mfd的细胞对毒素过度表达高度敏感。最后，我们发现Mfd促进至少一种毒素基因的诱变，表明其在调节转录中的功能可能促进某些TA系统和其他包含强RNA二级结构的区域的进化。我们得出的结论是，Mfd是一种RNAP辅因子，对于难以转录的区域（尤其是那些表达结构性调节RNA的区域）的转录调控非常重要，有时甚至很关键。</p>

<h2>Pubmed_ID: 33443177</h2>

<h3>Ancient DNA from Guam and the peopling of the Pacific</h3>

<h3>关岛的古代DNA与太平洋人民 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2022112118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443177/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Humans reached the Mariana Islands in the western Pacific by ∼3,500 y ago, contemporaneous with or even earlier than the initial peopling of Polynesia. They crossed more than 2,000 km of open ocean to get there, whereas voyages of similar length did not occur anywhere else until more than 2,000 y later. Yet, the settlement of Polynesia has received far more attention than the settlement of the Marianas. There is uncertainty over both the origin of the first colonizers of the Marianas (with different lines of evidence suggesting variously the Philippines, Indonesia, New Guinea, or the Bismarck Archipelago) as well as what, if any, relationship they might have had with the first colonizers of Polynesia. To address these questions, we obtained ancient DNA data from two skeletons from the Ritidian Beach Cave Site in northern Guam, dating to ∼2,200 y ago. Analyses of complete mitochondrial DNA genome sequences and genome-wide SNP data strongly support ancestry from the Philippines, in agreement with some interpretations of the linguistic and archaeological evidence, but in contradiction to results based on computer simulations of sea voyaging. We also find a close link between the ancient Guam skeletons and early Lapita individuals from Vanuatu and Tonga, suggesting that the Marianas and Polynesia were colonized from the same source population, and raising the possibility that the Marianas played a role in the eventual settlement of Polynesia.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 人类在大约3500年前到达了西太平洋的马里亚纳群岛，与波利尼西亚的最初人种入侵同时甚至早于人们。他们越过了2,000多公里的远洋到达那里，而类似长度的航行直到2000多年前才在其他任何地方发生。但是，波利尼西亚的定居点比马里亚纳斯的定居点受到了更多关注。马里亚纳群岛的第一批殖民者的起源（证据各异，分别表明菲律宾，印度尼西亚，新几内亚或the斯麦群岛）以及它们与阿拉伯殖民者之间的关系（如果有的话）都不确定。波利尼西亚的第一批殖民者。为了解决这些问题，我们从关岛北部Ritidian海滩洞穴遗址的两个骨骼中获得了古代DNA数据，可追溯到大约2200年前。完整的线粒体DNA基因组序列分析和全基因组SNP数据有力地支持了菲律宾的血统，与对语言和考古证据的某些解释一致，但与基于海上航行的计算机模拟的结果相矛盾。我们还发现古代关岛骨骼与瓦努阿图和汤加的早期Lapita个体之间存在密切联系，这表明马里亚纳群岛和波利尼西亚是从相同的来源人口中殖民出来的，这增加了马里亚纳群岛在波利尼西亚的最终定居中发挥作用的可能性。</p>

<h2>Pubmed_ID: 33443164</h2>

<h3>A genome-wide microRNA screen identifies the microRNA-183/96/182 cluster as a modulator of circadian rhythms</h3>

<h3>全基因组的microRNA筛选将microRNA-183 / 96/182簇识别为昼夜节律的调节剂 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2020454118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443164/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The regulatory mechanisms of circadian rhythms have been studied primarily at the level of the transcription-translation feedback loops of protein-coding genes. Regulatory modules involving noncoding RNAs are less thoroughly understood. In particular, emerging evidence has revealed the important role of microRNAs (miRNAs) in maintaining the robustness of the circadian system. To identify miRNAs that have the potential to modulate circadian rhythms, we conducted a genome-wide miRNA screen using U2OS luciferase reporter cells. Among 989 miRNAs in the library, 120 changed the period length in a dose-dependent manner. We further validated the circadian regulatory function of an miRNA cluster, miR-183/96/182, both in vitro and in vivo. We found that all three members of this miRNA cluster can modulate circadian rhythms. Particularly, miR-96 directly targeted a core circadian clock gene, PER2. The knockout of the miR-183/96/182 cluster in mice showed tissue-specific effects on circadian parameters and altered circadian rhythms at the behavioral level. This study identified a large number of miRNAs, including the miR-183/96/182 cluster, as circadian modulators. We provide a resource for further understanding the role of miRNAs in the circadian network and highlight the importance of miRNAs as a genome-wide layer of circadian clock regulation.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 昼夜节律的调节机制主要是在蛋白质编码基因的转录-翻译反馈环水平上进行研究。涉及非编码RNA的调控模块了解得不够彻底。特别是，新出现的证据揭示了microRNA（miRNA）在维持昼夜节律系统健壮性方面的重要作用。为了鉴定具有可能调节昼夜节律的miRNA，我们使用U2OS荧光素酶报道细胞进行了全基因组的miRNA筛选。文库中的989个miRNA中，有120个以剂量依赖性方式改变了周期长度。我们进一步验证了miRNA簇miR-183/96/182在体外和体内的昼夜节律调节功能。我们发现，这个miRNA簇的所有三个成员都可以调节昼夜节律。特别是，miR-96直接靶向核心生物钟基因PER2。敲除小鼠中的miR-183/96/182簇显示出对昼夜节律参数的组织特异性作用，并在行为水平上改变了昼夜节律。这项研究确定了包括miR-183/96/182簇在内的大量miRNA作为昼夜节律调节剂。我们提供了进一步了解miRNA在昼夜节律网络中的作用的资源，并强调了miRNA作为昼夜节律调控的全基因组层的重要性。</p>

<h2>Pubmed_ID: 33443159</h2>

<h3>The Parkinson's disease-associated gene ITPKB protects against α-synuclein aggregation by regulating ER-to-mitochondria calcium release</h3>

<h3>帕金森病相关基因ITPKB通过调节ER到线粒体的钙释放来防止α-突触核蛋白聚集 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2006476118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443159/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Inositol-1,4,5-triphosphate (IP3) kinase B (ITPKB) is a ubiquitously expressed lipid kinase that inactivates IP3, a secondary messenger that stimulates calcium release from the endoplasmic reticulum (ER). Genome-wide association studies have identified common variants in the ITPKB gene locus associated with reduced risk of sporadic Parkinson's disease (PD). Here, we investigate whether ITPKB activity or expression level impacts PD phenotypes in cellular and animal models. In primary neurons, knockdown or pharmacological inhibition of ITPKB increased levels of phosphorylated, insoluble α-synuclein pathology following treatment with α-synuclein preformed fibrils (PFFs). Conversely, ITPKB overexpression reduced PFF-induced α-synuclein aggregation. We also demonstrate that ITPKB inhibition or knockdown increases intracellular calcium levels in neurons, leading to an accumulation of calcium in mitochondria that increases respiration and inhibits the initiation of autophagy, suggesting that ITPKB regulates α-synuclein pathology by inhibiting ER-to-mitochondria calcium transport. Furthermore, the effects of ITPKB on mitochondrial calcium and respiration were prevented by pretreatment with pharmacological inhibitors of the mitochondrial calcium uniporter complex, which was also sufficient to reduce α-synuclein pathology in PFF-treated neurons. Taken together, these results identify ITPKB as a negative regulator of α-synuclein aggregation and highlight modulation of ER-to-mitochondria calcium flux as a therapeutic strategy for the treatment of sporadic PD.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 肌醇-1,4,5-三磷酸（IP3）激酶B（ITPKB）是一种普遍表达的脂质激酶，可以使IP3失活，IP3是一种二级信使，可以刺激钙从内质网（ER）释放。全基因组关联研究已经确定了ITPKB基因位点的常见变异，这些变异与散发性帕金森病（PD）的风险降低有关。在这里，我们调查ITPKB活性或表达水平是否会影响细胞和动物模型中的PD表型。在原代神经元中，用α-突触核蛋白预成纤维（PFFs）处理后，ITPKB的敲低或药理抑制作用会增加磷酸化的，不可溶的α-突触核蛋白病理学水平。相反，ITPKB的过表达减少了PFF诱导的α-突触核蛋白聚集。我们还证明了ITPKB抑制或敲低会增加神经元中的细胞内钙水平，从而导致线粒体中钙的积累，从而增加呼吸并抑制自噬的发生，这表明ITPKB通过抑制ER至线粒体的钙转运来调节α-突触核蛋白的病理状态。。此外，通过用线粒体钙单转运蛋白复合物的药理抑制剂进行预处理可以防止ITPKB对线粒体钙和呼吸的影响，这也足以减少PFF治疗的神经元的α-突触核蛋白病理。综上所述，这些结果确定ITPKB为α-突触核蛋白聚集的负调节剂，并强调了ER至线粒体钙通量的调节作为散发PD的治疗策略。</p>

<h2>Pubmed_ID: 33443141</h2>

<h3>Genome-wide surveillance of transcription errors in response to genotoxic stress</h3>

<h3>全基因组监测对遗传毒性胁迫的转录错误 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2004077118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33443141/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Mutagenic compounds are a potent source of human disease. By inducing genetic instability, they can accelerate the evolution of human cancers or lead to the development of genetically inherited diseases. Here, we show that in addition to genetic mutations, mutagens are also a powerful source of transcription errors. These errors arise in dividing and nondividing cells alike, affect every class of transcripts inside cells, and, in certain cases, greatly exceed the number of mutations that arise in the genome. In addition, we reveal the kinetics of transcription errors in response to mutagen exposure and find that DNA repair is required to mitigate transcriptional mutagenesis after exposure. Together, these observations have far-reaching consequences for our understanding of mutagenesis in human aging and disease, and suggest that the impact of DNA damage on human physiology has been greatly underestimated.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 致突变化合物是人类疾病的重要来源。通过诱发遗传不稳定，它们可以加速人类癌症的发展或导致遗传性疾病的发展。在这里，我们表明，除了遗传突变外，诱变剂还是转录错误的有力来源。这些错误会在分裂和非分裂的细胞中产生，影响细胞内的每一类转录物，在某些情况下，大大超过基因组中发生的突变数量。此外，我们揭示了响应诱变剂暴露而发生转录错误的动力学，并发现需要DNA修复以减轻暴露后的诱变。总之，这些观察结果对于我们对人类衰老和疾病中的诱变的理解产生了深远的影响，并且表明DNA损害对人类生理的影响已被大大低估了。</p>

<h2>Pubmed_ID: 33442061</h2>

<h3>Functional refolding of the penetration protein on a non-enveloped virus</h3>

<h3>渗透蛋白在非包膜病毒上的功能性重折叠 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-020-03124-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33442061/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2019': '42.779', '2018': '43.07', '2017': '41.577', '2016': '40.137', '2015': '38.138'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/13/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: A non-enveloped virus requires a membrane lesion to deliver its genome into a target cell1. For rotaviruses, membrane perforation is a principal function of the viral outer-layer protein, VP42,3. Here we describe the use of electron cryomicroscopy to determine how VP4 performs this function and show that when activated by cleavage to VP8* and VP5*, VP4 can rearrange on the virion surface from an 'upright' to a 'reversed' conformation. The reversed structure projects a previously buried 'foot' domain outwards into the membrane of the host cell to which the virion has attached. Electron cryotomograms of virus particles entering cells are consistent with this picture. Using a disulfide mutant of VP4, we have also stabilized a probable intermediate in the transition between the two conformations. Our results define molecular mechanisms for the first steps of the penetration of rotaviruses into the membranes of target cells and suggest similarities with mechanisms postulated for other viruses.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 非包膜病毒需要膜病灶才能将其基因组传递到靶细胞中1。对于轮状病毒，膜穿孔是病毒外层蛋白VP42,3的主要功能。在这里，我们描述了使用电子冷冻显微镜来确定VP4如何执行此功能，并显示当通过裂解为VP8*和VP5*激活时，VP4可以在病毒粒子表面上从直立构象重新排列为反向构象。反向结构将先前埋藏的足结构域向外投射到病毒体已附着的宿主细胞的膜中。病毒颗粒进入细胞的电子冰冻图与该图一致。使用VP4的二硫键突变体，我们还在两个构象之间的过渡中稳定了可能的中间体。我们的研究结果确定了轮状病毒进入靶细胞膜的第一步的分子机制，并暗示了与其他病毒的机制相似。</p>

<h2>Pubmed_ID: 33442060</h2>

<h3>m 6 A RNA methylation regulates the fate of endogenous retroviruses</h3>

<h3>m 6 RNA甲基化调节内源性逆转录病毒的命运 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-020-03135-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33442060/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2019': '42.779', '2018': '43.07', '2017': '41.577', '2016': '40.137', '2015': '38.138'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/13/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Endogenous retroviruses (ERVs) are abundant and heterogenous groups of integrated retroviral sequences that affect genome regulation and cell physiology throughout their RNA-centred life cycle1. Failure to repress ERVs is associated with cancer, infertility, senescence and neurodegenerative diseases2,3. Here, using an unbiased genome-scale CRISPR knockout screen in mouse embryonic stem cells, we identify m6A RNA methylation as a way to restrict ERVs. Methylation of ERV mRNAs is catalysed by the complex of methyltransferase-like METTL3-METTL144 proteins, and we found that depletion of METTL3-METTL14, along with their accessory subunits WTAP and ZC3H13, led to increased mRNA abundance of intracisternal A-particles (IAPs) and related ERVK elements specifically, by targeting their 5' untranslated region. Using controlled auxin-dependent degradation of the METTL3-METTL14 enzymatic complex, we showed that IAP mRNA and protein abundance is dynamically and inversely correlated with m6A catalysis. By monitoring chromatin states and mRNA stability upon METTL3-METTL14 double depletion, we found that m6A methylation mainly acts by reducing the half-life of IAP mRNA, and this occurs by the recruitment of the YTHDF family of m6A reader proteins5. Together, our results indicate that RNA methylation provides a protective effect in maintaining cellular integrity by clearing reactive ERV-derived RNA species, which may be especially important when transcriptional silencing is less stringent.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 内源性逆转录病毒（ERV）是大量且异源的整合逆转录病毒序列，它们在以RNA为中心的整个生命周期中影响基因组调控和细胞生理。无法抑制ERV与癌症，不育，衰老和神经退行性疾病相关[2,3]。在这里，在小鼠胚胎干细胞中使用公正的基因组规模的CRISPR基因敲除筛选，我们确定m6ARNA甲基化是一种限制ERV的方法。ERVmRNA的甲基化是由甲基转移酶样METTL3-METTL144蛋白的复合物催化的，我们发现METTL3-METTL14的耗竭以及其辅助亚基WTAP和ZC3H13导致增加了脑池内A颗粒（IAPs）的mRNA丰度。和相关的ERVK元素，具体是针对其5个非翻译区域。使用受控的生长素依赖的METTL3-METTL14酶复合物的降解，我们表明IAPmRNA和蛋白丰度与m6A催化动力学相关且呈负相关。通过监测METTL3-METTL14双耗竭时的染色质状态和mRNA稳定性，我们发现m6A甲基化主要是通过降低IAPmRNA的半衰期来起作用的，这是通过募集mTHA阅读器蛋白YTHDF家族而发生的。总之，我们的结果表明，RNA甲基化可通过清除反应性ERV衍生的RNA种类来提供维持细胞完整性的保护作用，这在转录沉默不太严格时尤其重要。</p>

<h2>Pubmed_ID: 33441560</h2>

<h3>Genome-wide macroevolutionary signatures of key innovations in butterflies colonizing new host plants</h3>

<h3>蝴蝶在新宿主植物中定殖的关键创新的全基因组宏观进化特征 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20507-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33441560/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/13/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The mega-diversity of herbivorous insects is attributed to their co-evolutionary associations with plants. Despite abundant studies on insect-plant interactions, we do not know whether host-plant shifts have impacted both genomic adaptation and species diversification over geological times. We show that the antagonistic insect-plant interaction between swallowtail butterflies and the highly toxic birthworts began 55 million years ago in Beringia, followed by several major ancient host-plant shifts. This evolutionary framework provides a valuable opportunity for repeated tests of genomic signatures of macroevolutionary changes and estimation of diversification rates across their phylogeny. We find that host-plant shifts in butterflies are associated with both genome-wide adaptive molecular evolution (more genes under positive selection) and repeated bursts of speciation rates, contributing to an increase in global diversification through time. Our study links ecological changes, genome-wide adaptations and macroevolutionary consequences, lending support to the importance of ecological interactions as evolutionary drivers over long time periods.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 食草昆虫的巨大多样性归因于它们与植物的共同进化联系。尽管对昆虫与植物之间的相互作用进行了大量研究，但我们不知道宿主植物的转移是否已影响了整个地质时期的基因组适应性和物种多样性。我们表明燕尾蝴蝶和剧毒的双生艾草之间的拮抗性昆虫与植物的相互作用始于5500万年前的贝林西亚，随后发生了几次重大的古代寄主植物转移。这种进化框架为重复测试宏观进化变化的基因组特征以及评估整个系统发育的多样性速率提供了宝贵的机会。我们发现蝴蝶中的寄主植物转移与全基因组范围内的适应性分子进化（正选择下的更多基因）和物种形成速率的反复爆发有关，这有助于随着时间的流逝实现全球多样化。我们的研究将生态变化，全基因组适应和宏观进化的后果联系在一起，从而为长期相互作用中生态相互作用作为进化驱动力的重要性提供了支持。</p>

<h2>Pubmed_ID: 33438731</h2>

<h3>BiG-SLiCE: A highly scalable tool maps the diversity of 1.2 million biosynthetic gene clusters</h3>

<h3>BiG-SLiCE：高度可扩展的工具可绘制120万个生物合成基因簇的图谱 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa154</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33438731/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2019': '5.993', '2018': '4.688', '2017': '7.267', '2016': '6.871', '2015': '7.463'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/13/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Background: Genome mining for biosynthetic gene clusters (BGCs) has become an integral part of natural product discovery. The >200,000 microbial genomes now publicly available hold information on abundant novel chemistry. One way to navigate this vast genomic diversity is through comparative analysis of homologous BGCs, which allows identification of cross-species patterns that can be matched to the presence of metabolites or biological activities. However, current tools are hindered by a bottleneck caused by the expensive network-based approach used to group these BGCs into gene cluster families (GCFs). Results: Here, we introduce BiG-SLiCE, a tool designed to cluster massive numbers of BGCs. By representing them in Euclidean space, BiG-SLiCE can group BGCs into GCFs in a non-pairwise, near-linear fashion. We used BiG-SLiCE to analyze 1,225,071 BGCs collected from 209,206 publicly available microbial genomes and metagenome-assembled genomes within 10 days on a typical 36-core CPU server. We demonstrate the utility of such analyses by reconstructing a global map of secondary metabolic diversity across taxonomy to identify uncharted biosynthetic potential. BiG-SLiCE also provides a "query mode" that can efficiently place newly sequenced BGCs into previously computed GCFs, plus a powerful output visualization engine that facilitates user-friendly data exploration. Conclusions: BiG-SLiCE opens up new possibilities to accelerate natural product discovery and offers a first step towards constructing a global and searchable interconnected network of BGCs. As more genomes are sequenced from understudied taxa, more information can be mined to highlight their potentially novel chemistry. BiG-SLiCE is available via https://github.com/medema-group/bigslice.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 背景：用于生物合成基因簇（BGC）的基因组挖掘已成为天然产物发现不可或缺的一部分。现在可公开获得的>200,000个微生物基因组拥有丰富的新型化学信息。导航这种巨大的基因组多样性的一种方法是通过对同源BGC进行比较分析，从而可以鉴定可以与代谢物或生物活性相匹配的跨物种模式。但是，当前的工具受到瓶颈的困扰，瓶颈是由昂贵的基于网络的方法造成的，该方法用于将这些BGC分组为基因簇家族（GCF）。结果：在这里，我们介绍BiG-SLiCE，这是一种用于对大量BGC进行聚类的工具。通过在欧几里得空间中表示它们，BiG-SLiCE可以以非成对，近乎线性的方式将BGC分组为GCF。我们使用BiG-SLiCE在典型的36核CPU服务器上，在10天内分析了从209,206个可公开获得的微生物基因组和元基因组组装的基因组中收集的1,225,071个BGC。我们通过跨分类学重建次级代谢多样性的全球图谱来鉴定未知的生物合成潜力，从而证明了此类分析的效用。BiG-SLiCE还提供了一种“查询模式”，可以将新排序的BGC高效地放置到先前计算的GCF中，另外还提供了一个功能强大的输出可视化引擎，可以方便用户进行数据探索。结论：BiG-SLiCE为加快天然产物的发现提供了新的可能性，并为构建全球可搜索的BGC互连网络迈出了第一步。随着从未被充分研究的分类单元中测序出更多的基因组，可以挖掘更多的信息来突出其潜在的新颖化学。BiG-SLiCE可通过https://github.com/medema-group/bigslice获得。</p>

<h2>Pubmed_ID: 33438729</h2>

<h3>Genome diversity in Ukraine</h3>

<h3>乌克兰的基因组多样性 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa159</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33438729/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2019': '5.993', '2018': '4.688', '2017': '7.267', '2016': '6.871', '2015': '7.463'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/13/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Background: The main goal of this collaborative effort is to provide genome-wide data for the previously underrepresented population in Eastern Europe, and to provide cross-validation of the data from genome sequences and genotypes of the same individuals acquired by different technologies. We collected 97 genome-grade DNA samples from consented individuals representing major regions of Ukraine that were consented for public data release. BGISEQ-500 sequence data and genotypes by an Illumina GWAS chip were cross-validated on multiple samples and additionally referenced to 1 sample that has been resequenced by Illumina NovaSeq6000 S4 at high coverage. Results: The genome data have been searched for genomic variation represented in this population, and a number of variants have been reported: large structural variants, indels, copy number variations, single-nucletide polymorphisms, and microsatellites. To our knowledge, this study provides the largest to-date survey of genetic variation in Ukraine, creating a public reference resource aiming to provide data for medical research in a large understudied population. Conclusions: Our results indicate that the genetic diversity of the Ukrainian population is uniquely shaped by evolutionary and demographic forces and cannot be ignored in future genetic and biomedical studies. These data will contribute a wealth of new information bringing forth a wealth of novel, endemic and medically related alleles.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 背景：这项合作努力的主要目标是为东欧以前代表性不足的人群提供全基因组数据，并对通过不同技术获得的同一个体的基因组序列和基因型提供的数据进行交叉验证。我们从代表乌克兰主要地区的同意个人中收集了97个基因组级DNA样本，并同意进行公共数据发布。IlluminaGWAS芯片对BGISEQ-500序列数据和基因型进行了交叉验证，涉及多个样品，另外还引用了1个样品，这些样品已被IlluminaNovaSeq6000S4在高覆盖率下重新测序。结果：已经在基因组数据中搜索了该种群中代表的基因组变异，并报告了许多变异：大型结构变异，插入缺失，拷贝数变异，单核苷酸多态性和微卫星。据我们所知，这项研究提供了乌克兰最大的遗传变异最新调查，创建了一个公共参考资源，旨在为大量未充分研究的人群提供医学研究数据。结论：我们的结果表明，乌克兰人口的遗传多样性是受进化和人口统计学影响的独特形状，在未来的遗传和生物医学研究中不可忽视。这些数据将提供大量新信息，带来大量新颖，地方性和医学相关的等位基因。</p>

<h2>Pubmed_ID: 33436567</h2>

<h3>Genome-wide analyses of behavioural traits are subject to bias by misreports and longitudinal changes</h3>

<h3>行为特征的全基因组分析容易因误报和纵向变化而产生偏差 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20237-6</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33436567/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genome-wide association studies (GWAS) have discovered numerous genetic variants associated with human behavioural traits. However, behavioural traits are subject to misreports and longitudinal changes (MLC) which can cause biases in GWAS and follow-up analyses. Here, we demonstrate that individuals with higher disease burden in the UK Biobank (n = 455,607) are more likely to misreport or reduce their alcohol consumption levels, and propose a correction procedure to mitigate the MLC-induced biases. The alcohol consumption GWAS signals removed by the MLC corrections are enriched in metabolic/cardiovascular traits. Almost all the previously reported negative estimates of genetic correlations between alcohol consumption and common diseases become positive/non-significant after the MLC corrections. We also observe MLC biases for smoking and physical activities in the UK Biobank. Our findings provide a plausible explanation of the controversy about the effects of alcohol consumption on health outcomes and a caution for future analyses of self-reported behavioural traits in biobank data.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 全基因组关联研究（GWAS）已发现许多与人类行为特征相关的遗传变异。但是，行为特征容易出现误报和纵向变化（MLC），可能导致GWAS和后续分析产生偏差。在这里，我们证明了英国生物库中疾病负担较高的患者（n=455,607）更容易误报或降低其饮酒水平，并提出纠正程序以减轻MLC引起的偏见。通过MLC校正消除的酒精消耗GWAS信号丰富了新陈代谢/心血管特征。在进行MLC校正后，几乎所有先前报道的饮酒与常见疾病之间遗传相关性的负估计都变为正/无意义。我们还在英国生物库中观察到MLC在吸烟和体育锻炼方面的偏见。我们的发现为有关饮酒对健康结果影响的争议提供了合理的解释，并为将来在生物库数据中对自我报告的行为特征进行分析提供了警告。</p>

<h2>Pubmed_ID: 33436566</h2>

<h3>Insights into genome recoding from the mechanism of a classic +1-frameshifting tRNA</h3>

<h3>从经典的+1移码tRNA机制了解基因组编码 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20373-z</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33436566/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: While genome recoding using quadruplet codons to incorporate non-proteinogenic amino acids is attractive for biotechnology and bioengineering purposes, the mechanism through which such codons are translated is poorly understood. Here we investigate translation of quadruplet codons by a +1-frameshifting tRNA, SufB2, that contains an extra nucleotide in its anticodon loop. Natural post-transcriptional modification of SufB2 in cells prevents it from frameshifting using a quadruplet-pairing mechanism such that it preferentially employs a triplet-slippage mechanism. We show that SufB2 uses triplet anticodon-codon pairing in the 0-frame to initially decode the quadruplet codon, but subsequently shifts to the +1-frame during tRNA-mRNA translocation. SufB2 frameshifting involves perturbation of an essential ribosome conformational change that facilitates tRNA-mRNA movements at a late stage of the translocation reaction. Our results provide a molecular mechanism for SufB2-induced +1 frameshifting and suggest that engineering of a specific ribosome conformational change can improve the efficiency of genome recoding.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 尽管使用四联体密码子并入非蛋白原性氨基酸进行基因组编码对于生物技术和生物工程学目的很有吸引力，但人们对这种密码子翻译的机制了解甚少。在这里，我们研究了通过+1-移码tRNASufB2翻译四联体密码子的过程，该逆转录子在其反密码子环中包含一个额外的核苷酸。SufB2在细胞中的自然转录后修饰可防止其使用四联体配对机制移码，从而优先使用三联体滑移机制。我们显示SufB2在0帧中使用三联体反密码子-密码子配对以最初解码四联密码子，但随后在tRNA-mRNA移位期间移至+1-帧。SufB2移码涉及基本核糖体构象变化的扰动，该变化有助于tRNA-mRNA在易位反应后期移动。我们的研究结果提供了SufB2诱导的+1移码的分子机制，并建议对特定核糖体构象变化进行改造可以提高基因组编码的效率。</p>

<h2>Pubmed_ID: 33434495</h2>

<h3>A modular master regulator landscape controls cancer transcriptional identity</h3>

<h3>模块化的主调控因子环境可控制癌症的转录身份 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.11.045</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33434495/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Despite considerable efforts, the mechanisms linking genomic alterations to the transcriptional identity of cancer cells remain elusive. Integrative genomic analysis, using a network-based approach, identified 407 master regulator (MR) proteins responsible for canalizing the genetics of individual samples from 20 cohorts in The Cancer Genome Atlas (TCGA) into 112 transcriptionally distinct tumor subtypes. MR proteins could be further organized into 24 pan-cancer, master regulator block modules (MRBs), each regulating key cancer hallmarks and predictive of patient outcome in multiple cohorts. Of all somatic alterations detected in each individual sample, >50% were predicted to induce aberrant MR activity, yielding insight into mechanisms linking tumor genetics and transcriptional identity and establishing non-oncogene dependencies. Genetic and pharmacological validation assays confirmed the predicted effect of upstream mutations and MR activity on downstream cellular identity and phenotype. Thus, co-analysis of mutational and gene expression profiles identified elusive subtypes and provided testable hypothesis for mechanisms mediating the effect of genetic alterations.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 尽管付出了巨大的努力，将基因组改变与癌细胞的转录同一性联系起来的机制仍然难以捉摸。使用基于网络的方法进行的综合基因组分析，确定了407个主要调节剂（MR）蛋白，这些蛋白将来自癌症基因组图谱（TCGA）中20个队列的单个样本的遗传基因转化为112种转录上不同的肿瘤亚型。MR蛋白可以进一步组织为24个全癌，主调节模块模块（MRB），每个模块都可调节关键的癌症标志并预测多个队列中的患者预后。在每个个体样品中检测到的所有体细胞变化中，预计>50％会诱导MR异常活动，从而深入了解将肿瘤遗传学和转录同一性联系起来的机制，并建立非癌基因依赖性。遗传和药理学验证试验证实了上游突变和MR活性对下游细胞特性和表型的预期影响。因此，突变和基因表达谱的共同分析确定了难以捉摸的亚型，并为介导遗传改变影响的机制提供了可检验的假设。</p>

<h2>Pubmed_ID: 33432631</h2>

<h3>Genome-wide occupancy of Arabidopsis SWI/SNF chromatin remodeler SPLAYED provides insights into its interplay with its close homolog BRAHMA and Polycomb proteins</h3>

<h3>拟南芥SWI / SNF染色质重塑剂SPLAYED在全基因组中的占有率提供了与其紧密同源的BRAHMA和Polycomb蛋白相互作用的见解 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1111/tpj.15159</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33432631/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: The Plant journal : for cell and molecular biology   ||| Not finding IF</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: SPLAYED (SYD) is a SWItch/Sucrose Non-Fermentable (SWI/SNF)-type chromatin remodeler identified in Arabidopsis thaliana (Arabidopsis). It is believed to play both redundant and differential roles with its closest homolog BRAHMA (BRM) in diverse plant growth and development processes. To better understand how SYD functions, we profiled the genome-wide occupancy of SYD and its impact on the global transcriptome and deposition of trimethylation of histone H3 on lysine 27 (H3K27me3). To map the global occupancy of SYD, we generated a GFP-tagged transgenic line and used it for ChIP-seq experiments, by which more than 6,000 SYD target genes were identified. Through integrating SYD occupancy and transcriptome profiles, we found that SYD preferentially targets to nucleosome-free regions of expressed genes. Further analysis revealed that SYD occupancy peaks exhibit five distinct patterns, which were also shared by BRM and BAF60, a conserved SWI/SNF complex component, indicating the common target sites of these SWI/SNF chromatin remodelers and the functional relevance of such distinct patterns. To investigate the interplay between SYD and Polycomb-group (PcG) proteins, we performed a genome-wide analysis of H3K27me3 in syd-5. We observed both increases and decreases in H3K27me3 levels at a few hundred genes in syd-5 compared to wild-type (WT). Our results imply that SYD can act antagonistically or synergistically with PcG at specific genes. Together, our SYD genome-wide occupancy data and the transcriptome and H3K27me3 profiles provide a much-needed resource for dissecting SYD's crucial roles in the regulation of plant growth and development.</p>

<p style="line-height:1.8;margin:20px"> 摘要: SPLAYED（SYD）是拟南芥（Arabidopsis）中鉴定的SWItch/蔗糖不可发酵（SWI/SNF）型染色质重塑剂。据信，它与最接近的同源物BRAHMA（BRM）在多种植物生长和发育过程中均扮演着冗余和差异的角色。为了更好地理解SYD的功能，我们分析了SYD在全基因组中的占有率及其对全局转录组的影响以及组蛋白H3的三甲基化在赖氨酸27（H3K27me3）上的沉积。为了绘制SYD的全球占有率图，我们生成了一个GFP标记的转基因品系，并将其用于ChIP-seq实验，通过该实验鉴定了6,000多个SYD目标基因。通过整合SYD占用和转录组配置文件，我们发现SYD优先靶向表达基因的无核小体区域。进一步的分析表明，SYD的占用峰表现出五个不同的模式，BRM和BAF60（一个保守的SWI/SNF复杂组分）也共享这五个模式，表明这些SWI/SNF染色质重塑剂的共同目标位点以及这些不同模式的功能相关性。为了研究SYD和Polycomb-group（PcG）蛋白之间的相互作用，我们对syd-5中的H3K27me3进行了全基因组分析。我们观察到与野生型（WT）相比，syd-5中数百个基因的H3K27me3水平升高和降低。我们的结果表明，SYD可以在特定基因上与PcG拮抗或协同作用。总之，我们的SYD基因组范围的占用数据以及转录组和H3K27me3谱提供了剖析SYD在调节植物生长和发育中的关键作用的急需资源。</p>

<h2>Pubmed_ID: 33432184</h2>

<h3>A novel ACE2 isoform is expressed in human respiratory epithelia and is upregulated in response to interferons and RNA respiratory virus infection</h3>

<h3>一种新型的ACE2亚型在人类呼吸道上皮细胞中表达，并在干扰素和RNA呼吸道病毒感染中被上调 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41588-020-00759-x</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33432184/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature genetics  {'ISSN': '1061-4036', 'Name': 'nature genetics', '2019': '27.605', '2018': '25.455', '2017': '27.125', '2016': '27.959', '2015': '31.616'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Angiotensin-converting enzyme 2 (ACE2) is the main entry point in airway epithelial cells for SARS-CoV-2. ACE2 binding to the SARS-CoV-2 protein spike triggers viral fusion with the cell plasma membrane, resulting in viral RNA genome delivery into the host. Despite ACE2's critical role in SARS-CoV-2 infection, full understanding of ACE2 expression, including in response to viral infection, remains unclear. ACE2 was thought to encode five transcripts and one protein of 805 amino acids. In the present study, we identify a novel short isoform of ACE2 expressed in the airway epithelium, the main site of SARS-CoV-2 infection. Short ACE2 is substantially upregulated in response to interferon stimulation and rhinovirus infection, but not SARS-CoV-2 infection. This short isoform lacks SARS-CoV-2 spike high-affinity binding sites and, altogether, our data are consistent with a model where short ACE2 is unlikely to directly contribute to host susceptibility to SARS-CoV-2 infection.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 血管紧张素转换酶2（ACE2）是SARS-CoV-2在气道上皮细胞中的主要进入点。ACE2与SARS-CoV-2蛋白峰的结合触发了病毒与细胞质膜的融合，从而导致病毒RNA基因组传递到宿主中。尽管ACE2在SARS-CoV-2感染中起关键作用，但对ACE2表达（包括对病毒感染的反应）的全面了解仍不清楚。ACE2被认为可以编码五种转录物和一种805个氨基酸的蛋白质。在本研究中，我们确定了在呼吸道上皮细胞（SARS-CoV-2感染的主要部位）中表达的ACE2的新型短异构体。短ACE2会因干扰素刺激和鼻病毒感染而显着上调，但SARS-CoV-2感染则不会。这种短的同工型缺乏SARS-CoV-2尖峰的高亲和力结合位点，并且我们的数据完全符合以下模型：短ACE2不太可能直接导致宿主对SARS-CoV-2感染的易感性。</p>

<h2>Pubmed_ID: 33431884</h2>

<h3>Transcription shapes genome-wide histone acetylation patterns</h3>

<h3>转录形成全基因组组蛋白乙酰化模式 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20543-z</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33431884/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Histone acetylation is a ubiquitous hallmark of transcription, but whether the link between histone acetylation and transcription is causal or consequential has not been addressed. Using immunoblot and chromatin immunoprecipitation-sequencing in S. cerevisiae, here we show that the majority of histone acetylation is dependent on transcription. This dependency is partially explained by the requirement of RNA polymerase II (RNAPII) for the interaction of H4 histone acetyltransferases (HATs) with gene bodies. Our data also confirms the targeting of HATs by transcription activators, but interestingly, promoter-bound HATs are unable to acetylate histones in the absence of transcription. Indeed, HAT occupancy alone poorly predicts histone acetylation genome-wide, suggesting that HAT activity is regulated post-recruitment. Consistent with this, we show that histone acetylation increases at nucleosomes predicted to stall RNAPII, supporting the hypothesis that this modification is dependent on nucleosome disruption during transcription. Collectively, these data show that histone acetylation is a consequence of RNAPII promoting both the recruitment and activity of histone acetyltransferases.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 组蛋白乙酰化是转录的普遍特征，但是组蛋白乙酰化和转录之间的联系是因果关系还是因果关系尚未得到解决。在酿酒酵母中使用免疫印迹和染色质免疫沉淀测序，这里我们显示大多数组蛋白乙酰化依赖于转录。RNA聚合酶II（RNAPII）对H4组蛋白乙酰基转移酶（HATs）与基因体相互作用的要求可以部分解释这种依赖性。我们的数据还证实了转录激活因子对HAT的靶向作用，但有趣的是，在没有转录的情况下，启动子结合的HAT无法乙酰化组蛋白。确实，仅HAT占用率就无法预测整个基因组范围内的组蛋白乙酰化，这表明HAT活性在招聘后受到调节。与此相一致，我们表明组蛋白乙酰化在预计会阻滞RNAPII的核小体上增加，支持这一修饰依赖转录过程中核小体破坏的假说。总的来说，这些数据表明，组蛋白乙酰化是RNAPII促进组蛋白乙酰转移酶募集和活性的结果。</p>

<h2>Pubmed_ID: 33431880</h2>

<h3>Construction and integration of three de novo Japanese human genome assemblies toward a population-specific reference</h3>

<h3>构建和整合三个从头开始的日本人类基因组装配体，以适应特定人群 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20146-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33431880/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The complete human genome sequence is used as a reference for next-generation sequencing analyses. However, some ethnic ancestries are under-represented in the reference genome (e.g., GRCh37) due to its bias toward European and African ancestries. Here, we perform de novo assembly of three Japanese male genomes using > 100× Pacific Biosciences long reads and Bionano Genomics optical maps per sample. We integrate the genomes using the major allele for consensus and anchor the scaffolds using genetic and radiation hybrid maps to reconstruct each chromosome. The resulting genome sequence, JG1, is contiguous, accurate, and carries the Japanese major allele at most loci. We adopt JG1 as the reference for confirmatory exome re-analyses of seven rare-disease Japanese families and find that re-analysis using JG1 reduces total candidate variant calls versus GRCh37 while retaining disease-causing variants. These results suggest that integrating multiple genomes from a single population can aid genome analyses of that population.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 完整的人类基因组序列可用作下一代测序分析的参考。但是，由于其对欧洲和非洲祖先的偏见，某些种族祖先在参考基因组（例如GRCh37）中的代表性不足。在这里，我们使用>100倍的太平洋生物科学长读和每个样品的BionanoGenomics光学图，对三个日本男性基因组进行从头组装。我们使用主要等位基因整合基因组以达成共识，并使用遗传和辐射杂交图谱将支架锚定以重建每个染色体。产生的基因组序列JG1是连续的，准确的，并且最多携带日本主要等位基因。我们采用JG1作为对七个日本罕见病家庭进行确认性外显子组重新分析的参考，发现使用JG1进行的重新分析与GRCh37相比，减少了总候选变异，而保留了致病变异。这些结果表明，整合来自单个种群的多个基因组可以帮助对该种群进行基因组分析。</p>

<h2>Pubmed_ID: 33431825</h2>

<h3>Characterization of a new Leishmania major strain for use in a controlled human infection model</h3>

<h3>新的利什曼原虫主要菌株的表征，用于可控制的人类感染模型 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20569-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33431825/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Leishmaniasis is widely regarded as a vaccine-preventable disease, but the costs required to reach pivotal Phase 3 studies and uncertainty about which candidate vaccines should be progressed into human studies significantly limits progress in vaccine development for this neglected tropical disease. Controlled human infection models (CHIMs) provide a pathway for accelerating vaccine development and to more fully understand disease pathogenesis and correlates of protection. Here, we describe the isolation, characterization and GMP manufacture of a new clinical strain of Leishmania major. Two fresh strains of L. major from Israel were initially compared by genome sequencing, in vivo infectivity and drug sensitivity in mice, and development and transmission competence in sand flies, allowing one to be selected for GMP production. This study addresses a major roadblock in the development of vaccines for leishmaniasis, providing a key resource for CHIM studies of sand fly transmitted cutaneous leishmaniasis.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 利什曼病被广泛认为是一种可预防疫苗的疾病，但是进行关键的3期研究所需的费用以及对于应将哪些候选疫苗进行人体研究的不确定性，大大限制了这种被忽视的热带病的疫苗开发进展。受控的人类感染模型（CHIM）提供了加速疫苗开发并更充分了解疾病发病机理和保护相关性的途径。在这里，我们描述了一种新的利什曼原虫主要临床菌株的分离，鉴定和GMP生产。首先通过基因组测序，小鼠体内感染性和药物敏感性以及沙蝇的发育和传播能力，比较了以色列的两株新鲜的L.majorL.菌株，从而选择其中一株用于生产GMP。这项研究解决了开发利什曼病疫苗的主要障碍，为沙蝇传播的皮肤利什曼病的CHIM研究提供了重要资源。</p>

<h2>Pubmed_ID: 33431815</h2>

<h3>Ultraviolet radiation drives mutations in a subset of mucosal melanomas</h3>

<h3>紫外线驱动黏膜黑色素瘤子集中的突变 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20432-5</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33431815/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/11/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Although identified as the key environmental driver of common cutaneous melanoma, the role of ultraviolet radiation (UVR)-induced DNA damage in mucosal melanoma is poorly defined. We analyze 10 mucosal melanomas of conjunctival origin by whole genome sequencing and our data shows a predominance of UVR-associated single base substitution signature 7 (SBS7) in the majority of the samples. Our data shows mucosal melanomas with SBS7 dominance have similar genomic patterns to cutaneous melanomas and therefore this subset should not be excluded from treatments currently used for common cutaneous melanoma.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 尽管被确定为常见皮肤黑色素瘤的关键环境驱动因素，但对紫外线（UVR）诱导的DNA损伤在粘膜黑色素瘤中的作用尚不清楚。我们通过全基因组测序分析了10个结膜起源的黏膜黑色素瘤，我们的数据显示，在大多数样品中，UVR相关的单碱基取代标记7（SBS7）占优势。我们的数据显示，具有SBS7优势的粘膜黑色素瘤具有与皮肤黑色素瘤相似的基因组模式，因此不应将这一亚群排除在目前用于普通皮肤黑色素瘤的治疗中。</p>

<h2>Pubmed_ID: 33429094</h2>

<h3>Genome-wide Identification of Agronomically Important Genes in Outcrossing Crops using OutcrossSeq</h3>

<h3>使用OutcrossSeq在全系作物中农艺学重要基因的全基因组鉴定 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.molp.2021.01.003</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33429094/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Molecular plant  {'ISSN': '1674-2052', 'Name': 'molecular plant', '2019': '12.084', '2018': '10.812', '2017': '9.326', '2016': '8.827', '2015': '7.142'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Many important crops (e.g., tuber, root and tree crops) have the cross-pollination nature. In these crops, no inbred lines are available for genetic study and breeding due to self-incompatibility, clonal propagation way or long generation time, which makes the identifications of agronomically important genes difficult, particularly in crops with complex autopolyploid genome. Here we developed a method OutcrossSeq for mapping agronomically important loci in outcrossing crops based on whole-genome low-coverage resequencing of a large genetic population, and designed three computation algorithms in OutcrossSeq for different types of outcrossing populations. We applied OutcrossSeq in tuberous root crop (sweet potato, autopolyploidy), tree crop (walnut tree, highly heterozygous diploidy) and hybrid crops (double-cross populations) to generate high-density genotype maps in the outcrossing populations, which enabled precise identifications of genomic loci underlying important agronomic traits. Among them, candidate causative genes were detected based on functional clues. The results demonstrate that OutcrossSeq is robust and powerful for heterozygous species, including polyploidy, to identify agronomically important genes in a cost-efficient way. The OutcrossSeq software and its instruction manual are available for downloading at www.xhhuanglab.cn/tool/OutcrossSeq.html.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 许多重要的农作物（例如块茎，块根和树木农作物）都具有异花授粉的特性。在这些作物中，由于自身不相容，克隆繁殖方式或生成时间长等原因，没有自交系可用于遗传研究和育种，这使农学上重要基因的鉴定变得困难，尤其是在具有复杂的多倍体基因组的作物中。在这里，我们基于大量遗传种群的全基因组低覆盖率重测序技术，开发了一种用于在农作物上绘制农学上重要的基因座的方法OutcrossSeq，并在OutcrossSeq中针对不同类型的异交种群设计了三种计算算法。我们在块茎块根作物（红薯，多倍体），树木作物（胡桃树，高度杂合二倍体）和杂种作物（双杂交种群）中应用了OutcrossSeq，以在异交种群中生成高密度基因型图谱，从而能够准确鉴定重要农艺性状的基因组位点。其中，根据功能线索检测了候选致病基因。结果表明，OutcrossSeq对杂合物种（包括多倍体）具有强大的功能，可以以经济高效的方式鉴定农学上重要的基因。可从www.xhhuanglab.cn/tool/OutcrossSeq.html下载OutcrossSeq软件及其说明手册。</p>

<h2>Pubmed_ID: 33420778</h2>

<h3>Significantly improving the quality of genome assemblies through curation</h3>

<h3>通过管理显着提高基因组装配的质量 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa153</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33420778/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2019': '5.993', '2018': '4.688', '2017': '7.267', '2016': '6.871', '2015': '7.463'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/09/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Genome sequence assemblies provide the basis for our understanding of biology. Generating error-free assemblies is therefore the ultimate, but sadly still unachieved goal of a multitude of research projects. Despite the ever-advancing improvements in data generation, assembly algorithms and pipelines, no automated approach has so far reliably generated near error-free genome assemblies for eukaryotes. Whilst working towards improved datasets and fully automated pipelines, assembly evaluation and curation is actively used to bridge this shortcoming and significantly reduce the number of assembly errors. In addition to this increase in product value, the insights gained from assembly curation are fed back into the automated assembly strategy and contribute to notable improvements in genome assembly quality. We describe our tried and tested approach for assembly curation using gEVAL, the genome evaluation browser. We outline the procedures applied to genome curation using gEVAL and also our recommendations for assembly curation in a gEVAL-independent context to facilitate the uptake of genome curation in the wider community.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 基因组序列装配为我们对生物学的理解提供了基础。因此，生成无错误的程序集是众多研究项目的最终目标，但可悲的是仍未实现。尽管在数据生成，组装算法和流水线方面取得了不断进步的进步，但到目前为止，还没有一种自动化的方法能够可靠地为真核生物生成接近无错误的基因组组装。在致力于改进数据集和全自动管道的同时，积极地使用装配评估和管理来弥补这一缺陷并显着减少装配错误的数量。除了产品价值的增加外，从装配策划中获得的见解还反馈到自动化装配策略中，并有助于显着提高基因组装配质量。我们介绍了使用基因组评估浏览器gEVAL进行装配管理的久经考验的方法。我们概述了使用gEVAL进行基因组管理的程序，以及在不依赖gEVAL的情况下进行装配管理的建议，以促进更广泛社区对基因组管理的吸收。</p>

<h2>Pubmed_ID: 33420081</h2>

<h3>SLE non-coding genetic risk variant determines the epigenetic dysfunction of an immune cell specific enhancer that controls disease-critical microRNA expression</h3>

<h3>SLE非编码遗传风险变异体决定了控制疾病关键性microRNA表达的免疫细胞特异性增强子的表观遗传功能障碍 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20460-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33420081/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Since most variants that impact polygenic disease phenotypes localize to non-coding genomic regions, understanding the consequences of regulatory element variants will advance understanding of human disease mechanisms. Here, we report that the systemic lupus erythematosus (SLE) risk variant rs2431697 as likely causal for SLE through disruption of a regulatory element, modulating miR-146a expression. Using epigenomic analysis, genome-editing and 3D chromatin structure analysis, we show that rs2431697 tags a cell-type dependent distal enhancer specific for miR-146a that physically interacts with the miR-146a promoter. NF-kB binds the disease protective allele in a sequence-specific manner, increasing expression of this immunoregulatory microRNA. Finally, CRISPR activation-based modulation of this enhancer in the PBMCs of SLE patients attenuates type I interferon pathway activation by increasing miR-146a expression. Our work provides a strategy to define non-coding RNA functional regulatory elements using disease-associated variants and provides mechanistic links between autoimmune disease risk genetic variation and disease etiology.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 由于影响多基因疾病表型的大多数变异都位于非编码基因组区域，因此了解调控元件变异的后果将促进人们对人类疾病机制的了解。在这里，我们报告系统性红斑狼疮（SLE）的风险变体rs2431697可能是通过破坏调节元件，调节miR-146a表达而导致SLE的原因。使用表观基因组分析，基因组编辑和3D染色质结构分析，我们显示rs2431697标签了特定于miR-146a的与细胞类型相关的远端增强子，该增强子与miR-146a启动子发生了物理相互作用。NF-kB以序列特异性方式结合疾病保护等位基因，从而增加了这种免疫调节性microRNA的表达。最后，基于CRISPR激活的SLE患者PBMC中这种增强子的调节通过增加miR-146a表达来减弱I型干扰素途径的激活。我们的工作提供了一种使用疾病相关变体定义非编码RNA功能调控元件的策略，并提供了自身免疫性疾病风险遗传变异与疾病病因之间的机制联系。</p>

<h2>Pubmed_ID: 33420075</h2>

<h3>High-resolution single-cell 3D-models of chromatin ensembles during Drosophila embryogenesis</h3>

<h3>果蝇胚胎发生过程中染色质体的高分辨率单细胞3D模型 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20490-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33420075/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Single-cell chromatin studies provide insights into how chromatin structure relates to functions of individual cells. However, balancing high-resolution and genome wide-coverage remains challenging. We describe a computational method for the reconstruction of large 3D-ensembles of single-cell (sc) chromatin conformations from population Hi-C that we apply to study embryogenesis in Drosophila. With minimal assumptions of physical properties and without adjustable parameters, our method generates large ensembles of chromatin conformations via deep-sampling. Our method identifies specific interactions, which constitute 5-6% of Hi-C frequencies, but surprisingly are sufficient to drive chromatin folding, giving rise to the observed Hi-C patterns. Modeled sc-chromatins quantify chromatin heterogeneity, revealing significant changes during embryogenesis. Furthermore, >50% of modeled sc-chromatin maintain topologically associating domains (TADs) in early embryos, when no population TADs are perceptible. Domain boundaries become fixated during development, with strong preference at binding-sites of insulator-complexes upon the midblastula transition. Overall, high-resolution 3D-ensembles of sc-chromatin conformations enable further in-depth interpretation of population Hi-C, improving understanding of the structure-function relationship of genome organization.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 单细胞染色质研究提供了关于染色质结构如何与单个细胞功能相关的见解。然而，在高分辨率和基因组覆盖范围之间取得平衡仍然是挑战。我们描述了一种用于从人口Hi-C重建单细胞（sc）染色质构象的大型3D集成体的计算方法，该方法可用于研究果蝇的胚胎发生。在最小限度的物理特性假设和无可​​调整参数的情况下，我们的方法通过深度采样生成了大量染色质构象。我们的方法确定了特定的相互作用，这些相互作用构成Hi-C频率的5-6％，但令人惊讶地足以驱动染色质折叠，从而产生了观察到的Hi-C模式。建模的sc染色质可量化染色质异质性，揭示胚胎发生过程中的显着变化。此外，当没有群体TAD可以感知时，>50％的建模sc-染色质在早期胚胎中保持拓扑关联域（TAD）。区域边界在发育过程中被固定，在中胚层过渡时强烈偏好绝缘子复合体的结合位点。总体而言，sc-染色质构象的高分辨率3D集成可进一步深入解释Hi-C种群，从而增进了对基因组组织的结构-功能关系的了解。</p>

<h2>Pubmed_ID: 33420073</h2>

<h3>Plasma ctDNA is a tumor tissue surrogate and enables clinical-genomic stratification of metastatic bladder cancer</h3>

<h3>血浆ctDNA是肿瘤组织的替代物，可实现转移性膀胱癌的临床基因组分层 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20493-6</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33420073/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Molecular stratification can improve the management of advanced cancers, but requires relevant tumor samples. Metastatic urothelial carcinoma (mUC) is poised to benefit given a recent expansion of treatment options and its high genomic heterogeneity. We profile minimally-invasive plasma circulating tumor DNA (ctDNA) samples from 104 mUC patients, and compare to same-patient tumor tissue obtained during invasive surgery. Patient ctDNA abundance is independently prognostic for overall survival in patients initiating first-line systemic therapy. Importantly, ctDNA analysis reproduces the somatic driver genome as described from tissue-based cohorts. Furthermore, mutation concordance between ctDNA and matched tumor tissue is 83.4%, enabling benchmarking of proposed clinical biomarkers. While 90% of mutations are identified across serial ctDNA samples, concordance for serial tumor tissue is significantly lower. Overall, our exploratory analysis demonstrates that genomic profiling of ctDNA in mUC is reliable and practical, and mitigates against disease undersampling inherent to studying archival primary tumor foci. We urge the incorporation of cell-free DNA profiling into molecularly-guided clinical trials for mUC.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 分子分层可以改善晚期癌症的治疗，但需要相关的肿瘤样本。鉴于最近治疗方法的扩展及其高基因组异质性，转移性尿路上皮癌（mUC）有望受益。我们分析了来自104名mUC患者的微创血浆循环肿瘤DNA（ctDNA）样本，并与在侵入性手术中获得的同患者肿瘤组织进行了比较。开始一线系统治疗的患者中，患者ctDNA的丰度可独立预测其总体生存率。重要的是，ctDNA分析可重现基于组织的队列中所述的体细胞驱动基因组。此外，ctDNA与匹配的肿瘤组织之间的突变一致性为83.4％，可对拟议的临床生物标记物进行基准测试。尽管90％的突变是在连续ctDNA样品中鉴定出的，但对于连续肿瘤组织的一致性却明显较低。总的来说，我们的探索性分析表明，在mUC中ctDNA的基因组分析是可靠且实用的，并且可以缓解研究档案原发性肿瘤灶所固有的疾病抽样不足。我们敦促将无细胞DNA谱图纳入分子引导的mUC临床试验中。</p>

<h2>Pubmed_ID: 33420025</h2>

<h3>Bayesian genome scale modelling identifies thermal determinants of yeast metabolism</h3>

<h3>贝叶斯基因组规模模型确定了酵母代谢的热决定因素 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20338-2</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33420025/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The molecular basis of how temperature affects cell metabolism has been a long-standing question in biology, where the main obstacles are the lack of high-quality data and methods to associate temperature effects on the function of individual proteins as well as to combine them at a systems level. Here we develop and apply a Bayesian modeling approach to resolve the temperature effects in genome scale metabolic models (GEM). The approach minimizes uncertainties in enzymatic thermal parameters and greatly improves the predictive strength of the GEMs. The resulting temperature constrained yeast GEM uncovers enzymes that limit growth at superoptimal temperatures, and squalene epoxidase (ERG1) is predicted to be the most rate limiting. By replacing this single key enzyme with an ortholog from a thermotolerant yeast strain, we obtain a thermotolerant strain that outgrows the wild type, demonstrating the critical role of sterol metabolism in yeast thermosensitivity. Therefore, apart from identifying thermal determinants of cell metabolism and enabling the design of thermotolerant strains, our Bayesian GEM approach facilitates modelling of complex biological systems in the absence of high-quality data and therefore shows promise for becoming a standard tool for genome scale modeling.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 温度如何影响细胞代谢的分子基础一直是生物学中一个长期存在的问题，其中的主要障碍是缺乏高质量的数据和方法，无法将温度影响与单个蛋白质的功能相关联以及将它们结合在一起。系统级别。在这里，我们开发并应用贝叶斯建模方法来解决温度对基因组规模代谢模型（GEM）的影响。该方法使酶热参数的不确定性最小化，并大大提高了GEM的预测强度。由此产生的温度受限制的酵母GEM会发现限制在最适温度下生长的酶，而角鲨烯环氧酶（ERG1）被认为是最大的速率限制。通过用耐热酵母菌株的直系同源物替换该单一关键酶，我们获得了超过野生型的耐热菌株，证明了甾醇代谢在酵母热敏性中的关键作用。因此，除了确定细胞代谢的热决定因素并设计耐热菌株外，我们的贝叶斯GEM方法还可以在缺乏高质量数据的情况下简化复杂生物系统的建模，因此有望成为基因组规模建模的标准工具。</p>

<h2>Pubmed_ID: 33420019</h2>

<h3>Interaction between SNAI2 and MYOD enhances oncogenesis and suppresses differentiation in Fusion Negative Rhabdomyosarcoma</h3>

<h3>SNAI2和MYOD之间的相互作用增强了融合阴性横纹肌肉瘤的发生和抑制分化。 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20386-8</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33420019/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Rhabdomyosarcoma (RMS) is an aggressive pediatric malignancy of the muscle, that includes Fusion Positive (FP)-RMS harboring PAX3/7-FOXO1 and Fusion Negative (FN)-RMS commonly with RAS pathway mutations. RMS express myogenic master transcription factors MYOD and MYOG yet are unable to terminally differentiate. Here, we report that SNAI2 is highly expressed in FN-RMS, is oncogenic, blocks myogenic differentiation, and promotes growth. MYOD activates SNAI2 transcription via super enhancers with striped 3D contact architecture. Genome wide chromatin binding analysis demonstrates that SNAI2 preferentially binds enhancer elements and competes with MYOD at a subset of myogenic enhancers required for terminal differentiation. SNAI2 also suppresses expression of a muscle differentiation program modulated by MYOG, MEF2, and CDKN1A. Further, RAS/MEK-signaling modulates SNAI2 levels and binding to chromatin, suggesting that the differentiation blockade by oncogenic RAS is mediated in part by SNAI2. Thus, an interplay between SNAI2, MYOD, and RAS prevents myogenic differentiation and promotes tumorigenesis.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 横纹肌肉瘤（RMS）是一种侵袭性的小儿肌肉恶性肿瘤，包括带有PAX3/7-FOXO1的融合阳性（FP）-RMS和通常带有RAS途径突变的融合阴性（FN）-RMS。RMS表达肌原性主转录因子MYOD和MYOG，但无法终末分化。在这里，我们报道SNAI2在FN-RMS中高度表达，具有致癌性，阻断肌源性分化并促进生长。MYOD通过具有条纹3D接触结构的超级增强剂激活SNAI2转录。全基因组染色质结合分析表明，SNAI2优先结合增强子元件，并在终末分化所需的成肌增强子的子集上与MYOD竞争。SNAI2还抑制由MYOG，MEF2和CDKN1A调节的肌肉分化程序的表达。此外，RAS/MEK信号调节SNAI2水平和与染色质的结合，表明致癌性RAS的分化阻滞部分地由SNAI2介导。因此，SNAI2，MYOD和RAS之间的相互作用阻止了肌原性分化并促进了肿瘤的发生。</p>

<h2>Pubmed_ID: 33410470</h2>

<h3>Chromosome-level reference genome of the European wasp spider Argiope bruennichi: a resource for studies on range expansion and evolutionary adaptation</h3>

<h3>欧洲黄蜂蜘蛛Argiope bruennichi的染色体水平参考基因组：用于范围扩展和进化适应研究的资源 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa148</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33410470/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2019': '5.993', '2018': '4.688', '2017': '7.267', '2016': '6.871', '2015': '7.463'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Background: Argiope bruennichi, the European wasp spider, has been investigated intensively as a focal species for studies on sexual selection, chemical communication, and the dynamics of rapid range expansion at a behavioral and genetic level. However, the lack of a reference genome has limited insights into the genetic basis for these phenomena. Therefore, we assembled a high-quality chromosome-level reference genome of the European wasp spider as a tool for more in-depth future studies. Findings: We generated, de novo, a 1.67 Gb genome assembly of A. bruennichi using 21.8× Pacific Biosciences sequencing, polished with 19.8× Illumina paired-end sequencing data, and proximity ligation (Hi-C)-based scaffolding. This resulted in an N50 scaffold size of 124 Mb and an N50 contig size of 288 kb. We found 98.4% of the genome to be contained in 13 scaffolds, fitting the expected number of chromosomes (n = 13). Analyses showed the presence of 91.1% of complete arthropod BUSCOs, indicating a high-quality assembly. Conclusions: We present the first chromosome-level genome assembly in the order Araneae. With this genomic resource, we open the door for more precise and informative studies on evolution and adaptation not only in A. bruennichi but also in arachnids overall, shedding light on questions such as the genomic architecture of traits, whole-genome duplication, and the genomic mechanisms behind silk and venom evolution.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 背景：欧洲黄蜂蜘蛛Argiopebruennichi已作为一种重点物种进行了深入研究，以研究性选择，化学交流以及行为和遗传水平的快速范围扩展的动力学。然而，缺乏参考基因组限制了对这些现象的遗传基础的了解。因此，我们组装了欧洲黄蜂蜘蛛的高质量染色体水平参考基因组，作为进行更深入的未来研究的工具。发现：我们使用21.8xPacificBiosciences测序从头开始产生了1.67Gb布鲁氏杆菌基因组装配体，并用19.8xIllumina配对末端测序数据和基于邻近连接（Hi-C）的支架进行了修饰。这导致N50支架大小为124Mb，N50重叠群大小为288kb。我们发现98.4％的基因组包含在13个支架中，符合预期的染色体数目（n=13）。分析表明，完整节肢动物BUSCO的存在率为91.1％，表明组装质量很高。结论：我们提出了Araneae的第一个染色体水平的基因组装配。利用这种基因组资源，我们不仅可以在布鲁氏菌中而且可以在整个蜘蛛纲动物中对进化和适应性进行更精确，信息更丰富的研究，从而为诸如性状的基因组结构，全基因组复制和遗传等问题提供了亮点。丝绸和毒液进化背后的基因组机制。</p>

<h2>Pubmed_ID: 33408412</h2>

<h3>Genomic basis of geographical adaptation to soil nitrogen in rice</h3>

<h3>水稻对土壤氮素地理适应的基因组基础 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-020-03091-w</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33408412/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2019': '42.779', '2018': '43.07', '2017': '41.577', '2016': '40.137', '2015': '38.138'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/06/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The intensive application of inorganic nitrogen underlies marked increases in crop production, but imposes detrimental effects on ecosystems1,2: it is therefore crucial for future sustainable agriculture to improve the nitrogen-use efficiency of crop plants. Here we report the genetic basis of nitrogen-use efficiency associated with adaptation to local soils in rice (Oryza sativa L.). Using a panel of diverse rice germplasm collected from different ecogeographical regions, we performed a genome-wide association study on the tillering response to nitrogen-the trait that is most closely correlated with nitrogen-use efficiency in rice-and identified OsTCP19 as a modulator of this tillering response through its transcriptional response to nitrogen and its targeting to the tiller-promoting gene DWARF AND LOW-TILLERING (DLT)3,4. A 29-bp insertion and/or deletion in the OsTCP19 promoter confers a differential transcriptional response and variation in the tillering response to nitrogen among rice varieties. The allele of OsTCP19 associated with a high tillering response to nitrogen is prevalent in wild rice populations, but has largely been lost in modern cultivars: this loss correlates with increased local soil nitrogen content, which suggests that it might have contributed to geographical adaptation in rice. Introgression of the allele associated with a high tillering response into modern rice cultivars boosts grain yield and nitrogen-use efficiency under low or moderate levels of nitrogen, which demonstrates substantial potential for rice breeding and the amelioration of negative environment effects by reducing the application of nitrogen to crops.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 大量施用无机氮是作物产量显着增加的基础，但会对生态系统造成不利影响1,2：因此，对于未来的可持续农业而言，提高作物的氮利用效率至关重要。在这里，我们报告了氮利用效率的遗传基础，其与水稻（OryzasativaL.）的当地土壤适应有关。利用从不同生态地理区域收集的一组不同水稻种质，我们对氮的分ing响应进行了全基因组关联研究（氮与水稻的氮利用效率最相关的性状），并确定了OsTCP19作为氮的调节剂。这种分till反应是通过其对氮的转录反应以及针对分er促进基因DWARF和LOW-TILLERING（DLT）3,4的。OsTCP19启动子中29bp的插入和/或缺失赋予水稻品种不同的转录应答和分nitrogen应答对氮的变异。OsTCP19等位基因与高氮分response反应相关，在野生稻种群中普遍存在，但在现代品种中已大量丢失：这种损失与当地土壤氮含量增加相关，这表明它可能有助于水稻的地理适应。在低水平或中等水平的氮水平下，将与高分till反应相关的等位基因渗入提高了水稻产量和氮素利用效率，这表明水稻育种的潜力很大，并且通过减少氮素的施用改善了负面的环境影响庄稼。</p>

<h2>Pubmed_ID: 33408411</h2>

<h3>Platypus and echidna genomes reveal mammalian biology and evolution</h3>

<h3>鸭嘴兽和棘足动物基因组揭示了哺乳动物的生物学和进化 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41586-020-03039-0</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33408411/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature  {'ISSN': '0028-0836', 'Name': 'nature', '2019': '42.779', '2018': '43.07', '2017': '41.577', '2016': '40.137', '2015': '38.138'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/06/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Egg-laying mammals (monotremes) are the only extant mammalian outgroup to therians (marsupial and eutherian animals) and provide key insights into mammalian evolution1,2. Here we generate and analyse reference genomes of the platypus (Ornithorhynchus anatinus) and echidna (Tachyglossus aculeatus), which represent the only two extant monotreme lineages. The nearly complete platypus genome assembly has anchored almost the entire genome onto chromosomes, markedly improving the genome continuity and gene annotation. Together with our echidna sequence, the genomes of the two species allow us to detect the ancestral and lineage-specific genomic changes that shape both monotreme and mammalian evolution. We provide evidence that the monotreme sex chromosome complex originated from an ancestral chromosome ring configuration. The formation of such a unique chromosome complex may have been facilitated by the unusually extensive interactions between the multi-X and multi-Y chromosomes that are shared by the autosomal homologues in humans. Further comparative genomic analyses unravel marked differences between monotremes and therians in haptoglobin genes, lactation genes and chemosensory receptor genes for smell and taste that underlie the ecological adaptation of monotremes.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 产卵的哺乳动物（单足类）是现存的仅次于人类（有袋动物和欧亚动物）的哺乳动物种群，并提供了有关哺乳动物进化的重要见解1,2。在这里，我们生成并分析了鸭嘴兽（Ornithorhynchusanatinus）和针（Tachyglossusaculeatus）的参考基因组，它们代表了仅存的两个极端物种。鸭嘴兽的近乎完整的基因组组装已将几乎整个基因组固定在染色体上，从而显着改善了基因组的连续性和基因注释。连同我们的针序列，这两个物种的基因组使我们能够检测出祖先和谱系特有的基因组变化，这些变化既影响了单体进化又影响了哺乳动物的进化。我们提供的证据表明，最极端的性染色体复合体起源于祖先的染色体环构型。这样的独特染色体复合体的形成可能已经由人类常染色体同源物共有的多X和多Y染色体之间异常广泛的相互作用所促进。进一步的比较基因组分析揭示了单调单反群和泰里安人之间在触觉珠蛋白基因，泌乳基因和化学感应受体基因上气味和味觉的显着差异，这些差异构成了单调单据的生态适应基础。</p>

<h2>Pubmed_ID: 33406261</h2>

<h3>A new duck genome reveals conserved and convergently evolved chromosome architectures of birds and mammals</h3>

<h3>新的鸭基因组揭示了鸟类和哺乳动物的保守和融合进化的染色体结构 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1093/gigascience/giaa142</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33406261/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: GigaScience  {'ISSN': '2047-217x', 'Name': 'gigascience', '2019': '5.993', '2018': '4.688', '2017': '7.267', '2016': '6.871', '2015': '7.463'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/06/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Background: Ducks have a typical avian karyotype that consists of macro- and microchromosomes, but a pair of much less differentiated ZW sex chromosomes compared to chickens. To elucidate the evolution of chromosome architectures between ducks and chickens, and between birds and mammals, we produced a nearly complete chromosomal assembly of a female Pekin duck by combining long-read sequencing and multiplatform scaffolding techniques. Results: A major improvement of genome assembly and annotation quality resulted from the successful resolution of lineage-specific propagated repeats that fragmented the previous Illumina-based assembly. We found that the duck topologically associated domains (TAD) are demarcated by putative binding sites of the insulator protein CTCF, housekeeping genes, or transitions of active/inactive chromatin compartments, indicating conserved mechanisms of spatial chromosome folding with mammals. There are extensive overlaps of TAD boundaries between duck and chicken, and also between the TAD boundaries and chromosome inversion breakpoints. This suggests strong natural selection pressure on maintaining regulatory domain integrity, or vulnerability of TAD boundaries to DNA double-strand breaks. The duck W chromosome retains 2.5-fold more genes relative to chicken. Similar to the independently evolved human Y chromosome, the duck W evolved massive dispersed palindromic structures, and a pattern of sequence divergence with the Z chromosome that reflects stepwise suppression of homologous recombination. Conclusions: Our results provide novel insights into the conserved and convergently evolved chromosome features of birds and mammals, and also importantly add to the genomic resources for poultry studies.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 背景：鸭子具有典型的禽染色体核型，由大染色体和微染色体组成，但是与鸡相比，它们的一对ZW性染色体差异很小。为了阐明鸭和鸡之间以及鸟类和哺乳动物之间的染色体结构演变，我们通过结合长时间阅读的测序和多平台支架技术，生产了雌性北京鸭的近乎完整的染色体组装体。结果：基因组装配和注释质量的重大改善是由于成功解析了沿袭特异性的繁殖重复片段，使先前基于Illumina的组装片段化了。我们发现鸭的拓扑相关结构域（TAD）被隔离蛋白CTCF的推定结合位点，管家基因或有活性/无活性染色质区室的过渡区划出界线，表明保守的空间染色体折叠机制与哺乳动物有关。鸭和鸡之间，以及TAD边界和染色体倒置断裂点之间的TAD边界有大量重叠。这表明在维持调节结构域完整性或TAD边界对DNA双链断裂的脆弱性方面存在强大的自然选择压力。鸭的W染色体保留的基因比鸡多2.5倍。类似于独立进化的人类Y染色体，鸭子W进化出大量分散的回文结构，并且与Z染色体的序列发散模式反映了同源重组的逐步抑制。结论：我们的结果为鸟类和哺乳动物的保守和融合进化的染色体特征提供了新颖的见解，并且还重要地增加了用于家禽研究的基因组资源。</p>

<h2>Pubmed_ID: 33402735</h2>

<h3>Plant genome editing branches out</h3>

<h3>植物基因组编辑分支 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41477-020-00840-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33402735/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature plants  {'ISSN': '2055-026x', 'Name': 'nature plants', '2019': '13.256', '2018': '13.297', '2017': '11.471', '2016': '10.3', '2015': '0'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 没有摘要</p>

<h2>Pubmed_ID: 33402679</h2>

<h3>Sex-dimorphic genetic effects and novel loci for fasting glucose and insulin variability</h3>

<h3>空腹血糖和胰岛素变异性的性别双态遗传效应和新基因座 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-19366-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33402679/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Differences between sexes contribute to variation in the levels of fasting glucose and insulin. Epidemiological studies established a higher prevalence of impaired fasting glucose in men and impaired glucose tolerance in women, however, the genetic component underlying this phenomenon is not established. We assess sex-dimorphic (73,089/50,404 women and 67,506/47,806 men) and sex-combined (151,188/105,056 individuals) fasting glucose/fasting insulin genetic effects via genome-wide association study meta-analyses in individuals of European descent without diabetes. Here we report sex dimorphism in allelic effects on fasting insulin at IRS1 and ZNF12 loci, the latter showing higher RNA expression in whole blood in women compared to men. We also observe sex-homogeneous effects on fasting glucose at seven novel loci. Fasting insulin in women shows stronger genetic correlations than in men with waist-to-hip ratio and anorexia nervosa. Furthermore, waist-to-hip ratio is causally related to insulin resistance in women, but not in men. These results position dissection of metabolic and glycemic health sex dimorphism as a steppingstone for understanding differences in genetic effects between women and men in related phenotypes.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 性别差异会导致空腹血糖和胰岛素水平发生变化。流行病学研究表明，男性空腹血糖受损和女性葡萄糖耐量受损的发生率较高，但是，尚无此现象的遗传成分。我们通过全基因组关联研究荟萃分析评估了欧洲血统无糖尿病个体的性别-二态性（73,089/50,404女性和67,506/47,806男性）和性别结合（151,188/105,056个体）的空腹血糖/空腹胰岛素遗传效应。在这里，我们报道了IRS1和ZNF12基因座对空腹胰岛素的等位基因效应中的性别二态性，后者在女性中的全血中的RNA表达高于男性。我们还观察到七个新基因座对空腹血糖的性别同质影响。与腰臀比和神经性厌食症的男性相比，女性的空腹胰岛素具有更强的遗传相关性。此外，女性的腰臀比与胰岛素抵抗有因果关系，而男性则与胰岛素抵抗无关。这些结果将代谢和血糖健康性二态性的解剖定位为了解相关表型男女遗传效应差异的垫脚石。</p>

<h2>Pubmed_ID: 33398158</h2>

<h3>PAM-less plant genome editing using a CRISPR-SpRY toolbox</h3>

<h3>使用CRISPR-SpRY工具箱进行无PAM的植物基因组编辑 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41477-020-00827-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33398158/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature plants  {'ISSN': '2055-026x', 'Name': 'nature plants', '2019': '13.256', '2018': '13.297', '2017': '11.471', '2016': '10.3', '2015': '0'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The rapid development of the CRISPR-Cas9, -Cas12a and -Cas12b genome editing systems has greatly fuelled basic and translational plant research1-6. DNA targeting by these Cas nucleases is restricted by their preferred protospacer adjacent motifs (PAMs). The PAM requirement for the most popular Streptococcus pyogenes Cas9 (SpCas9) is NGG (N = A, T, C, G)7, limiting its targeting scope to GC-rich regions. Here, we demonstrate genome editing at relaxed PAM sites in rice (a monocot) and the Dahurian larch (a coniferous tree), using an engineered SpRY Cas9 variant8. Highly efficient targeted mutagenesis can be readily achieved by SpRY at relaxed PAM sites in the Dahurian larch protoplasts and in rice transgenic lines through non-homologous end joining (NHEJ). Furthermore, an SpRY-based cytosine base editor was developed and demonstrated by directed evolution of new herbicide resistant OsALS alleles in rice. Similarly, a highly active SpRY adenine base editor was developed based on ABE8e (ref. 9) and SpRY-ABE8e was able to target relaxed PAM sites in rice plants, achieving up to 79% editing efficiency with high product purity. Thus, the SpRY toolbox breaks a PAM restriction barrier in plant genome engineering by enabling DNA editing in a PAM-less fashion. Evidence was also provided for secondary off-target effects by de novo generated single guide RNAs (sgRNAs) due to SpRY-mediated transfer DNA self-editing, which calls for more sophisticated programmes for designing highly specific sgRNAs when implementing the SpRY genome editing toolbox.</p>

<p style="line-height:1.8;margin:20px"> 摘要: CRISPR-Cas9，-Cas12a和-Cas12b基因组编辑系统的迅速发展极大地推动了基础植物和转化植物的研究1-6。这些Cas核酸酶靶向的DNA受其优选的原间隔子相邻基序（PAM）限制。最流行的化脓性链球菌Cas9（SpCas9）的PAM要求为NGG（N=A，T，C，G）7，从而将其靶向范围限制在富含GC的区域。在这里，我们展示了使用工程化的SpRYCas9变异体，在水稻（单子叶植物）和Dahurian落叶松（针叶树）的轻松PAM位点进行基因组编辑。SpRY可通过非同源末端连接（NHEJ）在Dahurian落叶松原生质体和水稻转基因品系的轻松PAM位点轻松实现高效的定向诱变。此外，开发了基于SpRY的胞嘧啶碱基编辑器，并通过水稻中新的除草剂抗性OsALS等位基因的定向进化得到证明。同样，基于ABE8e（参考资料9）开发了一种高度活跃的SpRY腺嘌呤碱基编辑器，SpRY-ABE8e能够靶向水稻植株中的轻松PAM位点，实现了高达79％的编辑效率和较高的产品纯度。因此，SpRY工具箱通过启用无PAM方式的DNA编辑来打破植物基因组工程中的PAM限制障碍。还提供了由于SpRY介导的转移DNA自我编辑而从头产生的单向导RNA（sgRNA）造成的次级脱靶效应的证据，这要求在实施SpRY基因组编辑工具箱时设计更复杂的程序来设计高度特异性的sgRNA。</p>

<h2>Pubmed_ID: 33397980</h2>

<h3>Order and stochasticity in the folding of individual Drosophila genomes</h3>

<h3>果蝇个体基因组折叠的有序性和随机性 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20292-z</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397980/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Mammalian and Drosophila genomes are partitioned into topologically associating domains (TADs). Although this partitioning has been reported to be functionally relevant, it is unclear whether TADs represent true physical units located at the same genomic positions in each cell nucleus or emerge as an average of numerous alternative chromatin folding patterns in a cell population. Here, we use a single-nucleus Hi-C technique to construct high-resolution Hi-C maps in individual Drosophila genomes. These maps demonstrate chromatin compartmentalization at the megabase scale and partitioning of the genome into non-hierarchical TADs at the scale of 100 kb, which closely resembles the TAD profile in the bulk in situ Hi-C data. Over 40% of TAD boundaries are conserved between individual nuclei and possess a high level of active epigenetic marks. Polymer simulations demonstrate that chromatin folding is best described by the random walk model within TADs and is most suitably approximated by a crumpled globule build of Gaussian blobs at longer distances. We observe prominent cell-to-cell variability in the long-range contacts between either active genome loci or between Polycomb-bound regions, suggesting an important contribution of stochastic processes to the formation of the Drosophila 3D genome.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 哺乳动物和果蝇基因组被划分为拓扑关联域（TAD）。尽管已经报道了这种分区在功能上是相关的，但是尚不清楚TAD是代表位于每个细胞核中相同基因组位置的真实物理单元，还是作为细胞群中众多染色质折叠模式的平均值出现。在这里，我们使用单核Hi-C技术在单个果蝇基因组中构建高分辨率Hi-C图。这些图证明了染色质在兆碱基尺度上的区室化，以及基因组以100kb的尺度分为非分级TAD，这与大量原位Hi-C数据中的TAD图谱非常相似。超过40％的TAD边界在各个原子核之间是保守的，并具有高水平的活性表观遗传标记。聚合物模拟表明，染色质折叠最好由TAD内的随机游走模型描述，最适合由距离较远的高斯Blob的皱缩小球构建近似。我们观察到活跃的基因组基因座之间或Polycomb绑定的区域之间的远程接触中突出的细胞间变化，表明随机过程对果蝇3D基因组的形成的重要贡献。</p>

<h2>Pubmed_ID: 33397978</h2>

<h3>The patterns of deleterious mutations during the domestication of soybean</h3>

<h3>大豆驯化过程中有害突变的模式 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20337-3</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397978/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Globally, soybean is a major protein and oil crop. Enhancing our understanding of the soybean domestication and improvement process helps boost genomics-assisted breeding efforts. Here we present a genome-wide variation map of 10.6 million single-nucleotide polymorphisms and 1.4 million indels for 781 soybean individuals which includes 418 domesticated (Glycine max), 345 wild (Glycine soja), and 18 natural hybrid (G. max/G. soja) accessions. We describe the enhanced detection of 183 domestication-selective sweeps and the patterns of putative deleterious mutations during domestication and improvement. This predominantly selfing species shows 7.1% reduction of overall deleterious mutations in domesticated soybean relative to wild soybean and a further 1.4% reduction from landrace to improved accessions. The detected domestication-selective sweeps also show reduced levels of deleterious alleles. Importantly, genotype imputation with this resource increases the mapping resolution of genome-wide association studies for seed protein and oil traits in a soybean diversity panel.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 在全球范围内，大豆是主要的蛋白质和油料作物。增进我们对大豆驯化和改良过程的了解有助于促进基因组学辅助的育种工作。在这里，我们提供了781个大豆个体的1060万个单核苷酸多态性和140万个插入缺失的全基因组变异图，其中包括418头驯化的大豆（大豆），345头野生的大豆（大豆）和18种天然杂交的大豆（大豆）。。soja）加入。我们描述了增强的183驯化选择性扫描的检测和驯化和改善过程中的有害有害突变的模式。与野生大豆相比，这种主要自交种显示驯化大豆的总体有害突变减少了7.1％，从地方品种到改良种质进一步减少了1.4％。检测到的驯化选择性扫描也显示有害等位基因水平降低。重要的是，用这种资源进行基因型估算提高了大豆多样性研究小组对种子蛋白和油性状的全基因组关联研究的作图分辨率。</p>

<h2>Pubmed_ID: 33397967</h2>

<h3>ZNF143 mediates CTCF-bound promoter-enhancer loops required for murine hematopoietic stem and progenitor cell function</h3>

<h3>ZNF143介导鼠造血干细胞和祖细胞功能所需的CTCF结合的启动子-增强子环 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20282-1</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397967/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: CCCTC binding factor (CTCF) is an important factor in the maintenance of chromatin-chromatin interactions, yet the mechanism regulating its binding to chromatin is unknown. We demonstrate that zinc finger protein 143 (ZNF143) is a key regulator for CTCF-bound promoter-enhancer loops. In the murine genome, a large percentage of CTCF and ZNF143 DNA binding motifs are distributed 37 bp apart in the convergent orientation. Furthermore, deletion of ZNF143 leads to loss of CTCF binding on promoter and enhancer regions associated with gene expression changes. CTCF-bound promoter-enhancer loops are also disrupted after excision of ZNF143. ZNF143-CTCF-bound promoter-enhancer loops regulate gene expression patterns essential for maintenance of murine hematopoietic stem and progenitor cell integrity. Our data suggest a common feature of gene regulation is that ZNF143 is a critical factor for CTCF-bound promoter-enhancer loops.</p>

<p style="line-height:1.8;margin:20px"> 摘要: CCCTC结合因子（CTCF）是维持染色质与染色质相互作用的重要因素，但调节其与染色质结合的机制尚不清楚。我们证明锌指蛋白143（ZNF143）是CTCF绑定的启动子-增强子环的关键调节剂。在鼠基因组中，大量的CTCF和ZNF143DNA结合基序在聚合方向上相距37bp。此外，ZNF143的缺失导致与基因表达变化相关的启动子和增强子区域上CTCF结合的丧失。切除ZNF143后，CTCF结合的启动子-增强子环也被破坏。ZNF143-CTCF结合的启动子-增强子环调节维持小鼠造血干和祖细胞完整性必不可少的基因表达模式。我们的数据表明基因调控的一个共同特征是ZNF143是CTCF结合的启动子-增强子环的关键因素。</p>

<h2>Pubmed_ID: 33397927</h2>

<h3>Humanizing the yeast origin recognition complex</h3>

<h3>人源化酵母起源识别复合体 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20277-y</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397927/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The Origin Recognition Complex (ORC) is an evolutionarily conserved six-subunit protein complex that binds specific sites at many locations to coordinately replicate the entire eukaryote genome. Though highly conserved in structure, ORC's selectivity for replication origins has diverged tremendously between yeasts and humans to adapt to vastly different life cycles. In this work, we demonstrate that the selectivity determinant of ORC for DNA binding lies in a 19-amino acid insertion helix in the Orc4 subunit, which is present in yeast but absent in human. Removal of this motif from Orc4 transforms the yeast ORC, which selects origins based on base-specific binding at defined locations, into one whose selectivity is dictated by chromatin landscape and afforded with plasticity, as reported for human. Notably, the altered yeast ORC has acquired an affinity for regions near transcriptional start sites (TSSs), which the human ORC also favors.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 起源识别复合物（ORC）是一种进化保守的六亚基蛋白质复合物，可在许多位置结合特定位点以协调复制整个真核生物基因组。尽管ORC的结构高度保守，但其在复制起点上的选择性在酵母和人类之间却相差甚远，以适应不同的生命周期。在这项工作中，我们证明了ORC对DNA结合的选择性决定因素在于Orc4亚基中的19个氨基酸插入螺旋，该螺旋存在于酵母中，但在人类中不存在。从Orc4去除此基序后，酵母ORC转化为一种酵母，ORC在确定的位置根据碱基特异性结合选择来源，其选择性由染色质分布决定并具有可塑性，正如人类报道的那样。值得注意的是，改变的酵母ORC已获得人类ORC也偏爱的转录起始位点（TSS）附近区域的亲和力。</p>

<h2>Pubmed_ID: 33397914</h2>

<h3>Rapid parallel adaptation despite gene flow in silent crickets</h3>

<h3>尽管无声gene中有基因流，仍可快速并行适应 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20263-4</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397914/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Gene flow is predicted to impede parallel adaptation via de novo mutation, because it can introduce pre-existing adaptive alleles from population to population. We test this using Hawaiian crickets (Teleogryllus oceanicus) in which 'flatwing' males that lack sound-producing wing structures recently arose and spread under selection from an acoustically-orienting parasitoid. Morphometric and genetic comparisons identify distinct flatwing phenotypes in populations on three islands, localized to different loci. Nevertheless, we detect strong, recent and ongoing gene flow among the populations. Using genome scans and gene expression analysis we find that parallel evolution of flatwing on different islands is associated with shared genomic hotspots of adaptation that contain the gene doublesex, but the form of selection differs among islands and corresponds to known flatwing demographics in the wild. We thus show how parallel adaptation can occur on contemporary timescales despite gene flow, indicating that it could be less constrained than previously appreciated.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 预测基因流将阻止从头突变的并行适应，因为它可以在种群之间引入已有的适应性等位基因。我们使用夏威夷（Teleogryllusoceanicus）对此进行了测试，在这些lack中，最近出现了缺乏发声机翼结构的flat翼雄性，并在从定向声学的寄生虫中选择传播。形态计量学和遗传学比较确定了位于三个岛上，定位于不同基因座的种群中不同的扁平表型。尽管如此，我们在人群中发现了强大的，最近的和正在进行的基因流。使用基因组扫描和基因表达分析，我们发现扁wing在不同岛屿上的平行进化与包含基因doublesex的共享基因组适应热点相关，但是选择的形式在各​​个岛屿之间有所不同，并且对应于野生环境中已知的扁wing人口。因此，我们显示了尽管有基因流，并行适应如何在当代时间尺度上发生，这表明与以前的理解相比，它的约束可能更少。</p>

<h2>Pubmed_ID: 33397904</h2>

<h3>Long-read metagenomics using PromethION uncovers oral bacteriophages and their interaction with host bacteria</h3>

<h3>使用PromethION进行的长期宏基因组学研究发现了口腔噬菌体及其与宿主细菌的相互作用 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20199-9</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397904/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Bacteriophages (phages), or bacterial viruses, are very diverse and highly abundant worldwide, including as a part of the human microbiomes. Although a few metagenomic studies have focused on oral phages, they relied on short-read sequencing. Here, we conduct a long-read metagenomic study of human saliva using PromethION. Our analyses, which integrate both PromethION and HiSeq data of >30 Gb per sample with low human DNA contamination, identify hundreds of viral contigs; 0-43.8% and 12.5-56.3% of the confidently predicted phages and prophages, respectively, do not cluster with those reported previously. Our analyses demonstrate enhanced scaffolding, and the ability to place a prophage in its host genomic context and enable its taxonomic classification. Our analyses also identify a Streptococcus phage/prophage group and nine jumbo phages/prophages. 86% of the phage/prophage group and 67% of the jumbo phages/prophages contain remote homologs of antimicrobial resistance genes. Pan-genome analysis of the phages/prophages reveals remarkable diversity, identifying 0.3% and 86.4% of the genes as core and singletons, respectively. Furthermore, our study suggests that oral phages present in human saliva are under selective pressure to escape CRISPR immunity. Our study demonstrates the power of long-read metagenomics utilizing PromethION in uncovering bacteriophages and their interaction with host bacteria.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 噬菌体（噬菌体）或细菌病毒在世界范围内非常多样且高度丰富，包括作为人类微生物群系的一部分。尽管一些宏基因组学研究集中于口服噬菌体，但它们依赖于短读测序。在这里，我们使用PromethION进行了人类唾液的宏基因组学研究。我们的分析结合了PromethION和HiSeq数据（每个样品>30Gb）以及低人类DNA污染，可鉴定出数百个病毒重叠群。可信预测的噬菌体和原噬菌体分别有0-43.8％和12.5-56.3％与先前报道的不相聚。我们的分析表明增强的脚手架，以及将噬菌体放置在其宿主基因组环境中并实现其分类学分类的能力。我们的分析还确定了链球菌噬菌体/噬菌体组和9个巨型噬菌体/噬菌体。86％的噬菌体/噬菌体组和67％的巨型噬菌体/噬菌体包含抗微生物耐药基因的远程同源物。对噬菌体/噬菌体的全基因组分析显示出显着的多样性，分别确定了0.3％和86.4％的基因为核心和单例。此外，我们的研究表明人类唾液中存在的口腔噬菌体处于选择性压力下，以逃避CRISPR免疫力。我们的研究证明了利用PromethION进行的长期宏基因组学研究在发现噬菌体及其与宿主细菌相互作用中的作用。</p>

<h2>Pubmed_ID: 33397900</h2>

<h3>Efficient assembly of nanopore reads via highly accurate and intact error correction</h3>

<h3>通过高度准确和完整的错误校正有效组装纳米孔读数 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1038/s41467-020-20236-7</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397900/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Nature communications  {'ISSN': '2041-1723', 'Name': 'nature communications', '2019': '12.121', '2018': '11.878', '2017': '12.353', '2016': '12.124', '2015': '11.329'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Long nanopore reads are advantageous in de novo genome assembly. However, nanopore reads usually have broad error distribution and high-error-rate subsequences. Existing error correction tools cannot correct nanopore reads efficiently and effectively. Most methods trim high-error-rate subsequences during error correction, which reduces both the length of the reads and contiguity of the final assembly. Here, we develop an error correction, and de novo assembly tool designed to overcome complex errors in nanopore reads. We propose an adaptive read selection and two-step progressive method to quickly correct nanopore reads to high accuracy. We introduce a two-stage assembler to utilize the full length of nanopore reads. Our tool achieves superior performance in both error correction and de novo assembling nanopore reads. It requires only 8122 hours to assemble a 35X coverage human genome and achieves a 2.47-fold improvement in NG50. Furthermore, our assembly of the human WERI cell line shows an NG50 of 22 Mbp. The high-quality assembly of nanopore reads can significantly reduce false positives in structure variation detection.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 长的纳米孔读数在从头基因组组装中是有利的。但是，纳米孔读取通常具有广泛的错误分布和较高的错误率子序列。现有的错误校正工具无法有效地校正纳米孔读数。大多数方法都会在纠错期间修剪高错误率的子序列，从而减少读取的长度和最终装配的连续性。在这里，我们开发了一种纠错和从头组装的工具，旨在克服纳米孔读取中的复杂错误。我们提出了一种自适应阅读选择和两步渐进方法，可以快速正确地校正纳米孔读数。我们介绍了一个两阶段的汇编程序，以利用全长的纳米孔读数。我们的工具在纠错和从头组装纳米孔读数方面均具有卓越的性能。组装35X覆盖率的人类基因组仅需8122个小时，就能将NG50提高2.47倍。此外，我们组装的人类WERI细胞系显示NG50为22Mbp。纳米孔读段的高质量组装可以显着减少结构变异检测中的假阳性。</p>

<h2>Pubmed_ID: 33397809</h2>

<h3>Prediction of Alzheimer's disease-specific phospholipase c gamma-1 SNV by deep learning-based approach for high-throughput screening</h3>

<h3>基于深度学习的高通量筛选方法预测阿尔茨海默氏病特定磷脂酶c gamma-1 SNV </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2011250118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397809/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/19/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Exon splicing triggered by unpredicted genetic mutation can cause translational variations in neurodegenerative disorders. In this study, we discover Alzheimer's disease (AD)-specific single-nucleotide variants (SNVs) and abnormal exon splicing of phospholipase c gamma-1 (PLCγ1) gene, using genome-wide association study (GWAS) and a deep learning-based exon splicing prediction tool. GWAS revealed that the identified single-nucleotide variations were mainly distributed in the H3K27ac-enriched region of PLCγ1 gene body during brain development in an AD mouse model. A deep learning analysis, trained with human genome sequences, predicted 14 splicing sites in human PLCγ1 gene, and one of these completely matched with an SNV in exon 27 of PLCγ1 gene in an AD mouse model. In particular, the SNV in exon 27 of PLCγ1 gene is associated with abnormal splicing during messenger RNA maturation. Taken together, our findings suggest that this approach, which combines in silico and deep learning-based analyses, has potential for identifying the clinical utility of critical SNVs in AD prediction.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 由不可预测的基因突变触发的外显子剪接可导致神经退行性疾病的翻译变异。在这项研究中，我们使用全基因组关联研究（GWAS）和基于深度学习的方法，发现了阿尔茨海默氏病（AD）特异的单核苷酸变异（SNV）和磷脂酶cgamma-1（PLCγ1）基因的异常外显子剪接。外显子剪接预测工具。GWAS揭示，在AD小鼠模型的脑发育过程中，鉴定出的单核苷酸变异主要分布在PLCγ1基因体的H3K27ac富集区域。一项经过深度学习分析的人类基因组序列，预测了人类PLCγ1基因的14个剪接位点，其中一个与ADγ小鼠模型中PLCγ1基因第27外显子的SNV完全匹配。特别是，PLCγ1基因第27外显子中的SNV与信使RNA成熟期间的异常剪接有关。综上所述，我们的发现表明，这种结合了计算机分析和基于深度学习的分析的方法，具有识别关键SNV在AD预测中的临床效用的潜力。</p>

<h2>Pubmed_ID: 33397718</h2>

<h3>ADP-ribose and analogues bound to the deMARylating macrodomain from the bat coronavirus HKU4</h3>

<h3>ADP核糖和类似物与蝙蝠冠状病毒HKU4的deMARylating宏域结合 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2004500118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33397718/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Macrodomains are proteins that recognize and hydrolyze ADP ribose (ADPR) modifications of intracellular proteins. Macrodomains are implicated in viral genome replication and interference with host cell immune responses. They are important to the infectious cycle of Coronaviridae and Togaviridae viruses. We describe crystal structures of the conserved macrodomain from the bat coronavirus (CoV) HKU4 in complex with ligands. The structures reveal a binding cavity that accommodates ADPR and analogs via local structural changes within the pocket. Using a radioactive assay, we present evidence of mono-ADPR (MAR) hydrolase activity. In silico analysis presents further evidence on recognition of the ADPR modification for hydrolysis. Mutational analysis of residues within the binding pocket resulted in diminished enzymatic activity and binding affinity. We conclude that the common structural features observed in the macrodomain in a bat CoV contribute to a conserved function that can be extended to other known macrodomains.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 大域是识别并水解细胞内蛋白的ADP核糖（ADPR）修饰的蛋白。大域涉及病毒基因组复制和对宿主细胞免疫反应的干扰。它们对于冠状病毒和Togaviridae病毒的感染周期很重要。我们描述了从蝙蝠冠状病毒（CoV）HKU4与配体复合的保守大域的晶体结构。这些结构揭示了一个可通过口袋内局部结构变化容纳ADPR和类似物的结合腔。使用放射性测定，我们提出了单ADPR（MAR）水解酶活性的证据。电子计算机分析提供了对ADPR修饰用于水解的认识的进一步证据。结合袋内残基的突变分析导致酶活性和结合亲和力降低。我们得出结论，蝙蝠冠状病毒在宏域中观察到的常见结构特征有助于保守功能，可以扩展到其他已知的宏域。</p>

<h2>Pubmed_ID: 33382968</h2>

<h3>Genome-Scale Identification of SARS-CoV-2 and Pan-coronavirus Host Factor Networks</h3>

<h3>SARS-CoV-2和泛冠状病毒宿主因子网络的基因组规模鉴定。 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.12.006</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33382968/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The coronavirus disease 2019 (COVID-19) pandemic has claimed the lives of over one million people worldwide. The causative agent, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is a member of the Coronaviridae family of viruses that can cause respiratory infections of varying severity. The cellular host factors and pathways co-opted during SARS-CoV-2 and related coronavirus life cycles remain ill defined. To address this gap, we performed genome-scale CRISPR knockout screens during infection by SARS-CoV-2 and three seasonal coronaviruses (HCoV-OC43, HCoV-NL63, and HCoV-229E). These screens uncovered host factors and pathways with pan-coronavirus and virus-specific functional roles, including major dependency on glycosaminoglycan biosynthesis, sterol regulatory element-binding protein (SREBP) signaling, bone morphogenetic protein (BMP) signaling, and glycosylphosphatidylinositol biosynthesis, as well as a requirement for several poorly characterized proteins. We identified an absolute requirement for the VMP1, TMEM41, and TMEM64 (VTT) domain-containing protein transmembrane protein 41B (TMEM41B) for infection by SARS-CoV-2 and three seasonal coronaviruses. This human coronavirus host factor compendium represents a rich resource to develop new therapeutic strategies for acute COVID-19 and potential future coronavirus pandemics.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 2019年冠状病毒病（COVID-19）大流行夺走了全球超过一百万人的生命。病因是严重的急性呼吸系统综合症冠状病毒2（SARS-CoV-2），是冠状病毒科病毒的成员，可导致严重程度不同的呼吸道感染。在SARS-CoV-2和相关冠状病毒生命周期中选择的细胞宿主因子和途径仍然不清楚。为了解决这个差距，我们在SARS-CoV-2和三种季节性冠状病毒（HCoV-OC43，HCoV-NL63和HCoV-229E）感染期间进行了基因组规模的CRISPR基因敲除筛选。这些筛选发现了具有泛冠状病毒和病毒特异性功能作用的宿主因素和途径，包括对糖胺聚糖生物合成，甾醇调节元素结合蛋白（SREBP）信号传导，骨形态发生蛋白（BMP）信号传导和糖基磷脂酰肌醇生物合成的主要依赖性，以及作为一些特征较差的蛋白质的要求。我们确定了对VMP1，TMEM41和TMEM64（VTT）域的蛋白跨膜蛋白41B（TMEM41B）感染SARS-CoV-2和三种季节性冠状病毒的绝对要求。该人类冠状病毒宿主因子纲要为开发针对急性COVID-19和潜在的未来冠状病毒大流行的新治疗策略提供了丰富的资源。</p>

<h2>Pubmed_ID: 33372156</h2>

<h3>UFMylation inhibits the proinflammatory capacity of interferon-γ-activated macrophages</h3>

<h3>UFMylation抑制γ-干扰素激活的巨噬细胞的促炎能力 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2011763118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33372156/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/05/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Macrophages activated with interferon-γ (IFN-γ) in combination with other proinflammatory stimuli, such as lipopolysaccharide or tumor necrosis factor-α (TNF-α), respond with transcriptional and cellular changes that enhance clearance of intracellular pathogens at the risk of damaging tissues. IFN-γ effects must therefore be carefully balanced with inhibitory mechanisms to prevent immunopathology. We performed a genome-wide CRISPR knockout screen in a macrophage cell line to identify negative regulators of IFN-γ responses. We discovered an unexpected role of the ubiquitin-fold modifier (Ufm1) conjugation system (herein UFMylation) in inhibiting responses to IFN-γ and lipopolysaccharide. Enhanced IFN-γ activation in UFMylation-deficient cells resulted in increased transcriptional responses to IFN-γ in a manner dependent on endoplasmic reticulum stress responses involving Ern1 and Xbp1. Furthermore, UFMylation in myeloid cells is required for resistance to influenza infection in mice, indicating that this pathway modulates in vivo responses to infection. These findings provide a genetic roadmap for the regulation of responses to a key mediator of cellular immunity and identify a molecular link between the UFMylation pathway and immune responses.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 干扰素-γ（IFN-γ）结合其他促炎性刺激（例如脂多糖或肿瘤坏死因子-α（TNF-α））活化的巨噬细胞会产生转录和细胞变化，从而增强细胞内病原体的清除率，但有破坏危险组织。因此，必须通过抑制机制仔细平衡IFN-γ的作用，以防止免疫病理学。我们在巨噬细胞系中进行了全基因组CRISPR敲除筛选，以鉴定IFN-γ反应的负调控因子。我们发现了遍在蛋白倍数修饰剂（Ufm1）偶联系统（在本文中为UFMylation）在抑制对IFN-γ和脂多糖的应答中具有意想不到的作用。UFMylation缺陷型细胞中增强的IFN-γ激活以依赖于涉及Ern1和Xbp1的内质网应激反应的方式增加了对IFN-γ的转录反应。此外，髓样细胞中的UFMylation是小鼠抗流感感染所必需的，这表明该途径可调节体内对感染的反应。这些发现为调节对细胞免疫关键介质的应答提供了遗传路线图，并确定了UFMylation途径与免疫应答之间的分子联系。</p>

<h2>Pubmed_ID: 33372147</h2>

<h3>DeepTFactor: A deep learning-based tool for the prediction of transcription factors</h3>

<h3>DeepTFactor：基于深度学习的工具，用于预测转录因子 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1073/pnas.2021171118</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33372147/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Proceedings of the National Academy of Sciences of the United States of America  {'ISSN': '0027-8424', 'Name': 'proceedings of the national academy of sciences of the united states of america', '2019': '9.412', '2018': '9.58', '2017': '9.504', '2016': '9.661', '2015': '9.423'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/12/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: A transcription factor (TF) is a sequence-specific DNA-binding protein that modulates the transcription of a set of particular genes, and thus regulates gene expression in the cell. TFs have commonly been predicted by analyzing sequence homology with the DNA-binding domains of TFs already characterized. Thus, TFs that do not show homologies with the reported ones are difficult to predict. Here we report the development of a deep learning-based tool, DeepTFactor, that predicts whether a protein in question is a TF. DeepTFactor uses a convolutional neural network to extract features of a protein. It showed high performance in predicting TFs of both eukaryotic and prokaryotic origins, resulting in F1 scores of 0.8154 and 0.8000, respectively. Analysis of the gradients of prediction score with respect to input suggested that DeepTFactor detects DNA-binding domains and other latent features for TF prediction. DeepTFactor predicted 332 candidate TFs in Escherichia coli K-12 MG1655. Among them, 84 candidate TFs belong to the y-ome, which is a collection of genes that lack experimental evidence of function. We experimentally validated the results of DeepTFactor prediction by further characterizing genome-wide binding sites of three predicted TFs, YqhC, YiaU, and YahB. Furthermore, we made available the list of 4,674,808 TFs predicted from 73,873,012 protein sequences in 48,346 genomes. DeepTFactor will serve as a useful tool for predicting TFs, which is necessary for understanding the regulatory systems of organisms of interest. We provide DeepTFactor as a stand-alone program, available at https://bitbucket.org/kaistsystemsbiology/deeptfactor.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 转录因子（TF）是序列特异性的DNA结合蛋白，可调节一组特定基因的转录，从而调节细胞中的基因表达。通常通过分析与已经表征的TF的DNA结合结构域的序列同源性来预测TF。因此，难以预测与文献报道的同源性不高的TF。在这里，我们报告了基于深度学习的工具DeepTFactor的开发，该工具可以预测所讨论的蛋白质是否为TF。DeepTFactor使用卷积神经网络来提取蛋白质的特征。它在预测真核和原核来源的TF方面显示出高性能，导致F1分数分别为0.8154和0.8000。对输入的预测分数梯度的分析表明，DeepTFactor检测到TF预测的DNA结合域和其他潜在特征。DeepTFactor在大肠杆菌K-12MG1655中预测了332个候选TF。其中，有84个候选TF属于y-ome，这是缺乏实验功能证据的基因集合。我们通过进一步表征三个预测的TF，YqhC，YiaU和YahB的全基因组结合位点，通过实验验证了DeepTFactor预测的结果。此外，我们提供了从48,346个基因组中的73,873,012个蛋白质序列预测的4,674,808个TF的列表。DeepTFactor将作为预测TF的有用工具，这对于理解目标生物的调控系统是必需的。我们提供DeepTFactor作为独立程序，可从https://bitbucket.org/kaistsystemsbiology/deeptfactor获得。</p>

<h2>Pubmed_ID: 33346062</h2>

<h3>Orphan Crops and their Wild Relatives in the Genomic Era</h3>

<h3>基因组时代的孤儿作物及其近缘种 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.molp.2020.12.013</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33346062/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Molecular plant  {'ISSN': '1674-2052', 'Name': 'molecular plant', '2019': '12.084', '2018': '10.812', '2017': '9.326', '2016': '8.827', '2015': '7.142'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: More than half of the calories consumed by humans are provided by three major cereal crops (rice, maize, and wheat). Orphan crops are usually well adapted to low-input agricultural conditions, and they not only play vital roles in local areas but can also contribute to food and nutritional needs worldwide. Interestingly, many wild relatives of orphan crops are important weeds of major crops. Although orphan crops and their wild relatives have received little attentions from researchers for many years, genomic studies have recently been performed on these plants. Here, we provide an overview of genomic studies on orphan crops, with a focus on orphan cereals and their wild relatives. The genomes of at least 12 orphan cereals and/or their wild relatives have been sequenced. In addition to genomic benefits for orphan crop breeding, we discuss the potential ways for mutual utilization of genomic data from major crops, orphan crops, and their wild relatives (including weeds) and provide perspectives on genetic improvement of both orphan and major crops (including de novo domestication of orphan crops) in the coming genomic era.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 人类消耗的卡路里一半以上来自三种主要的谷物作物（大米，玉米和小麦）。孤儿作物通常非常适合低投入的农业条件，它们不仅在当地起着至关重要的作用，而且还可以满足全世界的粮食和营养需求。有趣的是，许多孤儿的野生近缘种是主要农作物的重要杂草。尽管多年以来孤儿作物及其野生近缘种一直很少受到研究人员的关注，但最近已对这些植物进行了基因组研究。在这里，我们概述了孤儿作物的基因组研究，重点是孤儿谷物及其野生近缘种。已对至少12种孤儿谷物和/或其野生近缘植物的基因组进行了测序。除了对孤儿作物育种的基因组益处外，我们还讨论了相互利用主要作物，孤儿作物及其野生近缘种（包括杂草）的基因组数据的潜在途径，并提供了有关孤儿和主要作物（包括杂草）遗传改良的观点即将到来的基因组时代从头开始对孤儿作物的驯化）。</p>

<h2>Pubmed_ID: 33340689</h2>

<h3>A Clearinghouse for Genome-Edited Crops and Field Testing</h3>

<h3>基因组编辑作物和田间试验的资料交换所 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.molp.2020.12.010</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33340689/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Molecular plant  {'ISSN': '1674-2052', 'Name': 'molecular plant', '2019': '12.084', '2018': '10.812', '2017': '9.326', '2016': '8.827', '2015': '7.142'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: No abstract</p>

<p style="line-height:1.8;margin:20px"> 摘要: 没有摘要</p>

<h2>Pubmed_ID: 33338422</h2>

<h3>A Crisp(r) New Perspective on SARS-CoV-2 Biology</h3>

<h3>SARS-CoV-2生物学的崭新视角 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.12.003</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33338422/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Complementary genome-wide CRISPR-Cas9 screens performed by multiple groups reveal new insights into SARS-CoV-2 biology including aspects of viral entry, translation, replication, egress, and the genes regulating these processes. Comparisons with other coronaviruses enhances our understanding of the cellular life cycle of this medically important family of emerging viruses.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 由多个小组进行的补充全基因组CRISPR-Cas9筛选揭示了SARS-CoV-2生物学的新见解，包括病毒进入，翻译，复制，外出以及调节这些过程的基因等方面。与其他冠状病毒的比较增强了我们对这一医学上重要的新兴病毒家族的细胞生命周期的了解。</p>

<h2>Pubmed_ID: 33338421</h2>

<h3>TMEM41B Is a Pan-flavivirus Host Factor</h3>

<h3>TMEM41B是泛黄病毒宿主因子 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.12.005</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33338421/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Flaviviruses pose a constant threat to human health. These RNA viruses are transmitted by the bite of infected mosquitoes and ticks and regularly cause outbreaks. To identify host factors required for flavivirus infection, we performed full-genome loss of function CRISPR-Cas9 screens. Based on these results, we focused our efforts on characterizing the roles that TMEM41B and VMP1 play in the virus replication cycle. Our mechanistic studies on TMEM41B revealed that all members of the Flaviviridae family that we tested require TMEM41B. We tested 12 additional virus families and found that SARS-CoV-2 of the Coronaviridae also required TMEM41B for infection. Remarkably, single nucleotide polymorphisms present at nearly 20% in East Asian populations reduce flavivirus infection. Based on our mechanistic studies, we propose that TMEM41B is recruited to flavivirus RNA replication complexes to facilitate membrane curvature, which creates a protected environment for viral genome replication.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 黄病毒对人类健康构成持续威胁。这些RNA病毒通过被叮咬的蚊子和tick虫叮咬传播，并经常引起疾病暴发。为了鉴定黄病毒感染所需的宿主因子，我们进行了全基因组功能丧失的CRISPR-Cas9筛选。基于这些结果，我们集中精力确定了TMEM41B和VMP1在病毒复制周期中所起的作用。我们对TMEM41B的机理研究表明，我们测试的黄病毒科的所有成员都需要TMEM41B。我们测试了另外12个病毒家族，发现冠状病毒科的SARS-CoV-2也需要TMEM41B进行感染。值得注意的是，东亚人群中近20％的单核苷酸多态性减少了黄病毒的感染。根据我们的机理研究，我们建议将TMEM41B募集到黄病毒RNA复制复合物中以促进膜弯曲，从而为病毒基因组复制创造一个受保护的环境。</p>

<h2>Pubmed_ID: 33333024</h2>

<h3>Genetic Screens Identify Host Factors for SARS-CoV-2 and Common Cold Coronaviruses</h3>

<h3>遗传筛查确定SARS-CoV-2和普通感冒冠状病毒的宿主因素 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.12.004</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33333024/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The Coronaviridae are a family of viruses that cause disease in humans ranging from mild respiratory infection to potentially lethal acute respiratory distress syndrome. Finding host factors common to multiple coronaviruses could facilitate the development of therapies to combat current and future coronavirus pandemics. Here, we conducted genome-wide CRISPR screens in cells infected by SARS-CoV-2 as well as two seasonally circulating common cold coronaviruses, OC43 and 229E. This approach correctly identified the distinct viral entry factors ACE2 (for SARS-CoV-2), aminopeptidase N (for 229E), and glycosaminoglycans (for OC43). Additionally, we identified phosphatidylinositol phosphate biosynthesis and cholesterol homeostasis as critical host pathways supporting infection by all three coronaviruses. By contrast, the lysosomal protein TMEM106B appeared unique to SARS-CoV-2 infection. Pharmacological inhibition of phosphatidylinositol kinases and cholesterol homeostasis reduced replication of all three coronaviruses. These findings offer important insights for the understanding of the coronavirus life cycle and the development of host-directed therapies.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 冠状病毒科是引起人类疾病的病毒家族，范围从轻度呼吸道感染到可能致命的急性呼吸窘迫综合征。寻找多种冠状病毒共同的宿主因素可以促进抗击目前和未来冠状病毒大流行的疗法的发展。在这里，我们对受SARS-CoV-2感染的细胞以及两种季节性循环的普通感冒冠状病毒OC43和229E进行了全基因组CRISPR筛选。这种方法正确地确定了独特的病毒进入因子ACE2（对于SARS-CoV-2），氨肽酶N（对于229E）和糖胺聚糖（对于OC43）。此外，我们确定磷脂酰肌醇磷酸酯的生物合成和胆固醇稳态是支持所有三种冠状病毒感染的关键宿主途径。相比之下，溶酶体蛋白TMEM106B似乎是SARS-CoV-2感染所特有的。磷脂酰肌醇激酶和胆固醇稳态的药理抑制作用降低了所有三种冠状病毒的复制。这些发现为了解冠状病毒的生命周期和宿主导向疗法的发展提供了重要的见识。</p>

<h2>Pubmed_ID: 33275900</h2>

<h3>Evaluating the Effects of SARS-CoV-2 Spike Mutation D614G on Transmissibility and Pathogenicity</h3>

<h3>评价SARS-CoV-2穗突变D614G对传播性和致病性的影响 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.11.020</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33275900/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Global dispersal and increasing frequency of the SARS-CoV-2 spike protein variant D614G are suggestive of a selective advantage but may also be due to a random founder effect. We investigate the hypothesis for positive selection of spike D614G in the United Kingdom using more than 25,000 whole genome SARS-CoV-2 sequences. Despite the availability of a large dataset, well represented by both spike 614 variants, not all approaches showed a conclusive signal of positive selection. Population genetic analysis indicates that 614G increases in frequency relative to 614D in a manner consistent with a selective advantage. We do not find any indication that patients infected with the spike 614G variant have higher COVID-19 mortality or clinical severity, but 614G is associated with higher viral load and younger age of patients. Significant differences in growth and size of 614G phylogenetic clusters indicate a need for continued study of this variant.</p>

<p style="line-height:1.8;margin:20px"> 摘要: SARS-CoV-2刺突蛋白变体D614G的全球散布和频率增加表明具有选择优势，但也可能是由于随机创始者效应所致。我们调查使用超过25,000个全基因组SARS-CoV-2序列在英国积极选择穗状D614G的假说。尽管可以使用两个峰值614变体很好地表示的大型数据集，但并非所有方法都显示出肯定选择的结论性信号。群体遗传分析表明，与614D相比，614G的频率以与选择优势一致的方式增加。我们没有发现任何迹象表明感染了614G穗突变体的患者具有更高的COVID-19死亡率或临床严重性，但是614G与较高的病毒载量和患者年龄较小相关。614G系统发生簇的生长和大小上的显着差异表明需要继续研究该变体。</p>

<h2>Pubmed_ID: 33172935</h2>

<h3>Transmission of SARS-CoV-2 on mink farms between humans and mink and back to humans</h3>

<h3>SARS-CoV-2在貂场和人与貂之间的传播并传回人类 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1126/science.abe5901</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33172935/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Science (New York, N.Y.)   ||| Not finding IF</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/08/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Animal experiments have shown that nonhuman primates, cats, ferrets, hamsters, rabbits, and bats can be infected by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). In addition, SARS-CoV-2 RNA has been detected in felids, mink, and dogs in the field. Here, we describe an in-depth investigation using whole-genome sequencing of outbreaks on 16 mink farms and the humans living or working on these farms. We conclude that the virus was initially introduced by humans and has since evolved, most likely reflecting widespread circulation among mink in the beginning of the infection period, several weeks before detection. Despite enhanced biosecurity, early warning surveillance, and immediate culling of animals in affected farms, transmission occurred between mink farms in three large transmission clusters with unknown modes of transmission. Of the tested mink farm residents, employees, and/or individuals with whom they had been in contact, 68% had evidence of SARS-CoV-2 infection. Individuals for which whole genomes were available were shown to have been infected with strains with an animal sequence signature, providing evidence of animal-to-human transmission of SARS-CoV-2 within mink farms.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 动物实验表明，非人类灵长类动物，猫，雪貂，仓鼠，兔子和蝙蝠可被严重的急性呼吸系统综合症冠状病毒2（SARS-CoV-2）感染。此外，在野外的猫，貂和狗中也检测到SARS-CoV-2RNA。在这里，我们描述了使用全基因组测序对16个水貂农场以及在这些农场生活或工作的人类进行的深入调查。我们得出的结论是，该病毒最初是由人类引入的，并且已经进化，很可能反映出在感染期开始时（检测前几周）水貂之间的广泛传播。尽管增强了生物安全性，预警监测并在受影响的农场中立即对动物进行了扑杀，但在三个大型传播群中，水貂场之间发生了传播，传播方式未知。在经过测试的水貂农场居民，雇员和/或与他们接触过的个人中，有68％的人有SARS-CoV-2感染的证据。显示可获得整个基因组的个体已感染具有动物序列特征的菌株，从而提供了貂场中SARS-CoV-2在动物间传播的证据。</p>

<h2>Pubmed_ID: 33152519</h2>

<h3>CRISPR-Mediated Engineering across the Central Dogma in Plant Biology for Basic Research and Crop Improvement</h3>

<h3>CRISPR介导的植物生物学中部原理的工程学，用于基础研究和作物改良 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.molp.2020.11.002</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33152519/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Molecular plant  {'ISSN': '1674-2052', 'Name': 'molecular plant', '2019': '12.084', '2018': '10.812', '2017': '9.326', '2016': '8.827', '2015': '7.142'}</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/04/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: The central dogma (CD) of molecular biology is the transfer of genetic information from DNA to RNA to protein. Major CD processes governing genetic flow include the cell cycle, DNA replication, chromosome packaging, epigenetic changes, transcription, posttranscriptional alterations, translation, and posttranslational modifications. The CD processes are tightly regulated in plants to maintain genetic integrity throughout the life cycle and to pass genetic materials to next generation. Engineering of various CD processes involved in gene regulation will accelerate crop improvement to feed the growing world population. CRISPR technology enables programmable editing of CD processes to alter DNA, RNA, or protein, which would have been impossible in the past. Here, an overview of recent advancements in CRISPR tool development and CRISPR-based CD modulations that expedite basic and applied plant research is provided. Furthermore, CRISPR applications in major thriving areas of research, such as gene discovery (allele mining and cryptic gene activation), introgression (de novo domestication and haploid induction), and application of desired traits beneficial to farmers or consumers (biotic/abiotic stress-resilient crops, plant cell factories, and delayed senescence), are described. Finally, the global regulatory policies, challenges, and prospects for CRISPR-mediated crop improvement are discussed.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 分子生物学的中心教条（CD）是遗传信息从DNA到RNA到蛋白质的转移。控制遗传流动的主要CD过程包括细胞周期，DNA复制，染色体包装，表观遗传变化，转录，转录后改变，翻译和翻译后修饰。植物中的CD过程受到严格的调节，以在整个生命周期中维持遗传完整性，并将遗传物质传递给下一代。基因调控涉及的各种CD过程的工程设计将加速作物改良，以养活不断增长的世界人口。CRISPR技术可实现CD过程的可编程编辑，以改变DNA，RNA或蛋白质，而这在过去是不可能的。在此，概述了CRISPR工具开发和基于CRISPR的CD调制的最新进展，这些进展加快了基础和应用植物的研究。此外，CRISPR在主要研究领域的应用，例如基因发现（等位基因挖掘和隐秘基因激活），基因渗入（从头驯化和单倍体诱导），以及对农民或消费者有益的所需性状的应用（生物/非生物胁迫-描述了抗逆性作物，植物细胞工厂和延缓衰老。最后，讨论了CRISPR介导的作物改良的全球监管政策，挑战和前景。</p>

<h2>Pubmed_ID: 33147445</h2>

<h3>Identification of Required Host Factors for SARS-CoV-2 Infection in Human Cells</h3>

<h3>鉴定人类细胞中SARS-CoV-2感染所需的宿主因子 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.10.030</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33147445/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: To better understand host-virus genetic dependencies and find potential therapeutic targets for COVID-19, we performed a genome-scale CRISPR loss-of-function screen to identify host factors required for SARS-CoV-2 viral infection of human alveolar epithelial cells. Top-ranked genes cluster into distinct pathways, including the vacuolar ATPase proton pump, Retromer, and Commander complexes. We validate these gene targets using several orthogonal methods such as CRISPR knockout, RNA interference knockdown, and small-molecule inhibitors. Using single-cell RNA-sequencing, we identify shared transcriptional changes in cholesterol biosynthesis upon loss of top-ranked genes. In addition, given the key role of the ACE2 receptor in the early stages of viral entry, we show that loss of RAB7A reduces viral entry by sequestering the ACE2 receptor inside cells. Overall, this work provides a genome-scale, quantitative resource of the impact of the loss of each host gene on fitness/response to viral infection.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 为了更好地理解宿主病毒的遗传依赖性并找到COVID-19的潜在治疗靶标，我们进行了基因组规模的CRISPR功能丧失筛选，以鉴定人肺泡上皮细胞SARS-CoV-2病毒感染所需的宿主因子。排名靠前的基因聚集到不同的途径中，包括液泡ATPase质子泵，Retromer和Commander复合物。我们使用几种正交方法验证这些基因靶点，例如CRISPR敲除，RNA干扰敲除和小分子抑制剂。使用单细胞RNA测序，我们确定了失去顶级基因后胆固醇生物合成中的共享转录变化。此外，考虑到ACE2受体在病毒进入的早期阶段中的关键作用，我们显示出RAB7A的丢失通过隔离细胞内的ACE2受体而减少了病毒进入。总的来说，这项工作为每个宿主基因的丧失对病毒感染的适应性/反应的影响提供了一个基因组规模的定量资源。</p>

<h2>Pubmed_ID: 33147444</h2>

<h3>Genome-wide CRISPR Screens Reveal Host Factors Critical for SARS-CoV-2 Infection</h3>

<h3>全基因组CRISPR筛选揭示了SARS-CoV-2感染至关重要的宿主因素 </h3>

<p style="margin:20px"> <font face="arial"> <b>DOI: 10.1016/j.cell.2020.10.028</b> </font> </P>

<p style="margin:20px"> <font face="arial" <b>URL: https://pubmed.ncbi.nlm.nih.gov/33147444/</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b>Journal: Cell  None</b> </font> </P>

<p style="margin:20px"> <font face="arial" color="blue"> <b> Date: 01/07/2021</b> </font> </p>

<p style="line-height:1.8;margin:20px"> Abstract: Identification of host genes essential for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection may reveal novel therapeutic targets and inform our understanding of coronavirus disease 2019 (COVID-19) pathogenesis. Here we performed genome-wide CRISPR screens in Vero-E6 cells with SARS-CoV-2, Middle East respiratory syndrome CoV (MERS-CoV), bat CoV HKU5 expressing the SARS-CoV-1 spike, and vesicular stomatitis virus (VSV) expressing the SARS-CoV-2 spike. We identified known SARS-CoV-2 host factors, including the receptor ACE2 and protease Cathepsin L. We additionally discovered pro-viral genes and pathways, including HMGB1 and the SWI/SNF chromatin remodeling complex, that are SARS lineage and pan-coronavirus specific, respectively. We show that HMGB1 regulates ACE2 expression and is critical for entry of SARS-CoV-2, SARS-CoV-1, and NL63. We also show that small-molecule antagonists of identified gene products inhibited SARS-CoV-2 infection in monkey and human cells, demonstrating the conserved role of these genetic hits across species. This identifies potential therapeutic targets for SARS-CoV-2 and reveals SARS lineage-specific and pan-CoV host factors that regulate susceptibility to highly pathogenic CoVs.</p>

<p style="line-height:1.8;margin:20px"> 摘要: 鉴定严重急性呼吸系统综合症冠状病毒2（SARS-CoV-2）感染所必需的宿主基因可能揭示新的治疗靶标，并有助于我们了解2019年冠状病毒病（COVID-19）的发病机理。在这里，我们对SARS-CoV-2，中东呼吸综合征CoV（MERS-CoV），表达SARS-CoV-1尖峰的蝙蝠CoVHKU5和水泡性口炎病毒（VSV）的Vero-E6细胞进行了全基因组CRISPR筛选表达SARS-CoV-2峰值。我们确定了已知的SARS-CoV-2宿主因子，包括受体ACE2和蛋白酶组织蛋白酶L。我们还发现了SARS谱系和泛冠状病毒特有的促病毒基因和途径，包括HMGB1和SWI/SNF染色质重塑复合体。，分别。我们显示，HMGB1调节ACE2表达，对于SARS-CoV-2，SARS-CoV-1和NL63的进入至关重要。我们还显示，已鉴定的基因产物的小分子拮抗剂抑制了猴子和人类细胞中的SARS-CoV-2感染，证明了这些遗传命中物种之间的保守作用。这确定了SARS-CoV-2的潜在治疗靶标，并揭示了SARS谱系特异性和泛CoV宿主因子，这些因子可调节对高致病性CoV的敏感性。</p>

</body>
</html>

